dataset=[
  {
    "page": "940",
    "text": "This page intentionally left blank \n\nCarlson_FM_pi-xiv.indd   4\n\n25/10/21   1:18 PM\n\n"
  },
  {
    "page": "941",
    "text": "Section 1  Basic Considerations in \nInfectious Diseases\n\n119\n\nApproach to the Patient \nwith an Infectious Disease\nNeeraj K. Surana, Dennis L. Kasper\n\n \u25a0 HISTORICAL PERSPECTIVE\nThe origins of the field of infectious diseases are humble. The notion \nthat communicable diseases were due to a miasma (\u201cbad air\u201d) can be \ntraced back to at least the mid-sixteenth century. Not until the work \nof Louis Pasteur and Robert Koch in the late nineteenth century was \nthere  credible  evidence  supporting  the  germ  theory  of  disease\u2014i.e., \nthat  microorganisms  are  the  direct  cause  of  infections.  In  contrast \nto  this  relatively  slow  start,  the  twentieth  century  saw  remarkable \nadvances in the field of infectious diseases, and the etiologic agents of \nnumerous  infectious  diseases  were  soon  identified.  Furthermore,  the \ndiscovery of antibiotics and the advent of vaccines against some of the \nmost  deadly  and  debilitating  infections  greatly  altered  the  landscape \nof  human  health.  Indeed,  the  twentieth  century  saw  the  elimination \nof  smallpox,  one  of  the  great  scourges  in  the  history  of  humanity. \nThese remarkable successes prompted Sir Frank MacFarlane Burnet, a \nnoted immunologist and Nobel laureate, to write in a 1962 publication \nentitled Natural History of Infectious Diseases: \u201cIn many ways one can \nthink of the middle of the twentieth century as the end of one of the \nmost important social revolutions in history, the virtual elimination of \ninfectious disease.\u201d Professor Burnet was not alone in this view. Robert \nPetersdorf, a renowned infectious disease expert and former editor of \nthis textbook, wrote in 1978 that \u201ceven with my great personal loyalties \nto  infectious  diseases,  I  cannot  conceive  a  need  for  309  more  [grad-\nuating  trainees  in  infectious  diseases]  unless  they  spend  their  time \nculturing  each  other.\u201d  Given  the  enormous  growth  of  interest  in  the \nmicrobiome in the past 15 years, Dr. Petersdorf \u2019s statement might have \nbeen ironically clairvoyant, although he could have had no idea what \nwas  in  store  for  humanity,  with  an  onslaught  of  new,  emerging,  and \nreemerging infectious diseases.\n\nClearly,  even  with  all  the  advances  of  the  twentieth  century, \ninfectious  diseases  continue  to  represent  a  formidable  challenge  for \npatients  and  physicians  alike.  Furthermore,  during  the  latter  half  of \nthe century, several chronic diseases were demonstrated to be directly \nor indirectly caused by infectious microbes; perhaps the most notable \nexamples are the associations of Helicobacter pylori with peptic ulcer \ndisease  and  gastric  carcinoma,  human  papillomavirus  with  cervical \ncancer, and hepatitis B and C viruses with liver cancer. In fact, ~16% \nof all malignancies are now known to be associated with an infectious \ncause.  In  addition,  numerous  emerging  and  reemerging  infectious \ndiseases continue to have a dire impact on global health: HIV/AIDS,  \nSARS-CoV-2,  Ebola,  and  Zika  are  but  a  few  examples.  The  fear  of \nweaponizing  pathogens  for  bioterrorism  is  ever  present  and  poses  a \npotentially enormous threat to public health. Moreover, escalating anti-\nmicrobial resistance in clinically relevant microbes (e.g., carbapenem- \nresistant  Enterobacteriaceae  and  Acinetobacter  spp.,  Candida  auris, \ndrug-resistant  Mycobacterium  tuberculosis,  and  vancomycin-resistant \nenterococci) signifies that the administration of antimicrobial agents\u2014\nonce thought to be a panacea\u2014requires appropriate stewardship. For \nall these reasons, infectious diseases continue to exert grim effects on \nindividual patients as well as on international public health. Even with \nall the successes of the past century, physicians must be as thoughtful \nabout  infectious  diseases  now  as  they  were  at  the  beginning  of  the \ntwentieth century.\n\nInfectious Diseases\n\nPART 5\n\n \u25a0 GLOBAL CONSIDERATIONS\nInfectious diseases remain the second leading cause of death worldwide. \nAlthough the rate of infectious disease\u2013related deaths has decreased dra-\nmatically over the past 25 years, there were still 10.3 million such deaths \nin 2017 (Fig. 119-1A). These deaths disproportionately affect children \n<1 year of age, adults older than 70 years, and persons living in low- and \nmiddle-income  countries  (Fig.  119-1B  and  119-1C;  Chap.  474);  in \n2017, ~18% of all deaths worldwide were related to infectious diseases, \nwith a rate as high as ~58% in sub-Saharan Africa.\n\nGiven that infectious diseases are still a major cause of global mor-\ntality,  understanding  the  local  epidemiology  of  disease  is  critically \nimportant  in  evaluating  patients.  Diseases  such  as  HIV/AIDS  have \ndecimated  sub-Saharan  Africa,  with  HIV-infected  adults  represent-\ning  20\u201323%  of  the  total  population  in  countries  like  South  Africa, \nBotswana, and Lesotho. Moreover, drug-resistant tuberculosis is ram-\npant  throughout  the  former  Soviet-bloc  countries,  India,  China,  and \nSouth Africa. The ready availability of this type of information allows \nphysicians to develop appropriate differential diagnoses and treatment \nplans for individual patients. Programs such as the Global Burden of \nDisease seek to quantify human losses (e.g., deaths, disability-adjusted \nlife-years) due to diseases by age, sex, and country over time; these data \nnot only help inform local, national, and international health policy but \ncan also help guide local medical decision-making.\n\nEven though some diseases (e.g., pandemic influenza, Middle East \nrespiratory  syndrome)  are  seemingly  geographically  restricted,  the \nincreasing  ease  of  rapid  worldwide  travel  has  raised  concern  about \ntheir  swift  spread  around  the  globe.  Indeed,  human  migration  has \nhistorically been the source of epidemics: Yersinia pestis spread along \ntrade  routes  in  the  fourteenth  century,  Native  American  populations \nwere  devastated  by  diseases  such  as  smallpox  and  measles  that  were \nimported by European explorers in the fifteenth and sixteenth centu-\nries, military maneuvers helped facilitate the spread of the 1918 influ-\nenza pandemic, and religious pilgrimages (e.g., the Hajj) provide the \nmeans for worldwide dissemination of diseases. The continued effects \nof  global  travel  on  the  spread  of  infectious  diseases  are  perhaps  best \nhighlighted by the SARS-CoV-2 pandemic (Chap. 199). Although this \nvirus was first identified in Wuhan, China, it quickly spread across the \nglobe and brought an abrupt end to virtually all travel and commerce \nthroughout the world, plunging economies into a deep recession, and \nresulting at one point in over half the world\u2019s population living under \nstay-at-home  orders.  Not  only  can  travelers  carry  person-to-person \ntransmitted  infections  (e.g.,  influenza,  HIV)  anywhere  in  the  world, \nbut they can also introduce vector-borne infections to new geographic \nareas (e.g., chikungunya and Zika viruses) and contribute to the world-\nwide spread of multidrug-resistant organisms. The world\u2019s increasing \ninterconnectedness has profound implications not only for the global \neconomy but also for medicine and the spread of infectious diseases.\n\n \u25a0 UNDERSTANDING THE MICROBIOTA\nNormal, healthy humans are colonized with ~50 trillion bacteria as well \nas  countless  viruses,  fungi,  and  archaea;  taken  together,  these  micro-\norganisms  outnumber  human  cells  by  ~10  times  in  the  human  body \n(Chap. 471). The major reservoir of these microbes is the gastrointesti-\nnal tract, but substantial numbers of microbes live in the female genital \ntract, the oral cavity, and the nasopharynx. There is increasing interest in \nthe skin and lungs as sites where microbial colonization might be highly \nrelevant to the biology and disease susceptibility of the host. These com-\nmensal organisms provide the host with myriad benefits, from aiding in \nmetabolism  to  shaping  the  immune  system.  With  regard  to  infectious \ndiseases, the vast majority of infections are caused by organisms that are \npart of the normal microbiota (e.g., Staphylococcus aureus, Streptococcus \npneumoniae,  Pseudomonas  aeruginosa),  with  relatively  few  infections \ndue to organisms that are strictly pathogens (e.g., Neisseria gonorrhoeae, \nrabies virus). Perhaps it is not surprising that a general understanding \nof  the  microbiota  is  essential  in  the  evaluation  of  infectious  diseases. \n\nHPIM21e_Part05_p941-p1796.indd   941\n\n20/01/22   6:34 PM\n\n"
  },
  {
    "page": "942",
    "text": "942\n\n15\n\n14\n\n13\n\n12\n\n11\n\n10\n\n9\n\n)\ns\nn\no\n\ni\nl\nl\ni\n\nm\nn\ni\n(\n\ns\nh\nt\na\ne\nd\n\nf\no\n\nr\ne\nb\nm\nu\nN\n\nA\n\n15000\n\n) 20000\n0\n0\n0\n,\n0\n0\n1\nr\ne\np\n(\nh\nt\na\ne\nd\nf\no\ne\nt\na\nR\n\n10000\n\n2500\n\n5000\n\n350\n\n300\n\n250\n\n200\n\n150\n\n100\n\nR\na\nt\ne\no\nf\nd\ne\na\nt\nh\n(\np\ne\nr\n1\n0\n0\n,\n0\n0\n0\n)\n\n1990\n\n1995 2000 2005 2010 2015 2017\n\nB\n\n0\n\n90+ years\n0\u201327 days\n80\u201389 years\n70\u201379 years\n60\u201369 years\n50\u201359 years\n40\u201349 years\n30\u201339 years\n20\u201329 years\n5\u20139 years\n10\u201319 years\n1\u20134 years\n28\u2013364 days\n\nAge\n\nC\n\n100\n\n200\n\n300\n\n400\n\n500\n\n600\n\n700\n\n800\n\nFIGURE 119-1  Magnitude of infectious disease\u2013related deaths globally. A. The absolute number (blue line; left axis) and rate (red line; right axis) of infectious disease\u2013\nrelated deaths throughout the world since 1990. B. Age-specific rates of infectious disease\u2013related deaths in 2017. In both A and B, the charts depict the mean estimate and \n95% uncertainty intervals. C. A map depicting country-specific data for the rate of total deaths that were attributable to communicable, maternal, neonatal, and nutritional \ndisorders in 2017. (Source: Global Burden of Disease Study, Institute for Health Metrics and Evaluation.)\n\nIndividuals\u2019 microbiotas have a major impact on their susceptibility to \ninfectious  diseases  and  even  their  responses  to  vaccines.  Site-specific \nknowledge  of  the  indigenous  microbiota  may  facilitate  appropriate \ninterpretation of culture results, aid in selection of empirical antimicro-\nbial therapy based on the likely causative agents, and provide additional \nimpetus for rational antibiotic use to minimize the untoward effects of \nthese drugs on the \u201cbeneficial\u201d microbes that inhabit the body.\n\n \u25a0 WHEN TO CONSIDER AN INFECTIOUS ETIOLOGY\nThe title of this chapter may appear to presuppose that the physician \nknows when a patient has an infectious disease. In reality, this chapter \ncan  serve  only  as  a  guide  to  the  evaluation  of  a  patient  in  whom  an \ninfectious  disease  is  a  possibility.  Once  a  specific  diagnosis  is  made, \nthe reader should consult the subsequent chapters that deal with spe-\ncific  microorganisms  in  detail.  The  challenge  for  the  physician  is  to \nrecognize which patients may have an infectious disease as opposed to \nsome other underlying disorder. This task is greatly complicated by the \nfact that infections have an infinite range of presentations, from acute \nlife-threatening  conditions  (e.g.,  meningococcemia)  to  chronic  dis-\neases of varying severity (e.g., H. pylori\u2013associated peptic ulcer disease) \nto no symptoms at all (e.g., latent M. tuberculosis infection). While it \nis impossible to generalize about a presentation that encompasses all \ninfections, common findings in the history, physical examination, and \n\nbasic  laboratory  testing  often  suggest  that  the  patient  either  has  an \ninfectious  disease  or  should  be  more  closely  evaluated  for  one.  This \nchapter focuses on these common findings and how they may direct \nthe ongoing evaluation of the patient.\n\nAPPROACH TO THE PATIENT\nInfectious Disease\n\nSee also Chap. 122. \nHISTORY\nAs  in  all  of  medicine,  a  complete  and  thorough  history  is  para-\nmount  in  the  evaluation  of  a  patient  with  a  possible  infectious \ndisease. The history is critical for developing a focused differential \ndiagnosis and for guiding the physical exam and initial diagnostic \ntesting.  Although  a  detailing  of  all  the  elements  of  a  history  is \nbeyond the scope of this chapter, specific components relevant to \ninfectious  diseases  require  particular  attention.  In  general,  these \naspects focus on two areas: (1) an exposure history that may iden-\ntify  microorganisms  with  which  the  patient  may  have  come  into \ncontact  and  (2)  host-specific  factors  that  may  predispose  to  the \ndevelopment of an infection. \n\nHPIM21e_Part05_p941-p1796.indd   942\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases \n \n \n \n \n \n \n \n \n \n \n \n"
  },
  {
    "page": "943",
    "text": "Exposure  History  \u2022\u2003 History  of  infections  or  exposure  to \ndrug-resistant microbes \nInformation about a patient\u2019s previous \ninfections,  with  the  associated  microbial  susceptibility  profiles,  is \nvery  helpful  in  determining  possible  etiologic  agents.  Specifically, \nknowing  whether  a  patient  has  a  history  of  infection  with  drug- \nresistant organisms (e.g., methicillin-resistant S. aureus, vancomycin- \nresistant  Enterococcus  species,  enteric  organisms  that  produce  an \nextended-spectrum  \u03b2-lactamase  or  carbapenemase)  or  may  have \nbeen exposed to drug-resistant microbes (e.g., during a recent stay \nin a hospital, nursing home, or long-term acute-care facility) may \nalter  the  choice  of  empirical  antibiotics.  For  example,  a  patient \npresenting with sepsis who is known to have a history of invasive \ninfection with a multidrug-resistant isolate of P. aeruginosa should \nbe treated empirically with an antimicrobial regimen that will cover \nthis strain. \nSocial history  Although the social history taken by physicians \nis  often  limited  to  inquiries  about  a  patient\u2019s  alcohol  and  tobacco \nuse,  a  complete  social  history  can  offer  a  number  of  clues  to  the \nunderlying diagnosis. Knowing whether the patient has any high-\nrisk behaviors (e.g., unsafe sexual behaviors, intravenous [IV] drug \nuse),  potential  hobby-associated  exposures  (e.g.,  avid  gardening, \nwith possible Sporothrix schenckii exposure), or occupational expo-\nsures  (e.g.,  increased  risk  for  M.  tuberculosis  exposure  in  funeral \nservice  workers)  can  facilitate  diagnosis.  The  importance  of  the \nsocial history is exemplified by a case in 2009 in which a laboratory \nresearcher  died  of  a  Y.  pestis  infection  acquired  during  his  work; \nalthough this patient had visited both an outpatient clinic and an \nemergency department, his records at both sites failed to include his \noccupation\u2014information that potentially could have led quickly to \nappropriate treatment and infection control measures. \nDietary  habits  Because  certain  pathogens  are  associated  with \nspecific dietary habits, inquiring about a patient\u2019s diet can provide \ninsight into possible exposures. For example, Shiga toxin\u2013producing \nstrains of Escherichia coli, and Toxoplasma gondii are associated with \nthe  consumption  of  raw  or  undercooked  meat;  Salmonella  typh-\nimurium,  Listeria  monocytogenes,  and  Mycobacterium  bovis  with \nunpasteurized milk; Leptospira species, parasites, and enteric bacte-\nria with unpurified water; and Vibrio species, norovirus, helminths, \nand protozoa with raw seafood. \nAnimal exposures  Because animals are often important vectors \nof infectious diseases, patients should be asked about exposures to \nany animals, including contact with their own pets, visits to petting \nzoos,  or  random  encounters  (e.g.,  home  rodent  infestation).  For \nexample, dogs can carry ticks that serve as agents for the transmis-\nsion of several infectious diseases, including Lyme disease, Rocky \nMountain spotted fever, and ehrlichiosis. Cats are associated with \nBartonella  henselae  infection,  reptiles  with  Salmonella  infection, \nrodents with leptospirosis, and rabbits with tularemia (Chap. 141). \nTravel  history  Attention  should  be  paid  to  both  international \nand domestic travel. Fever in a patient who has recently returned \nfrom  abroad  significantly  broadens  the  differential  diagnosis \n(Chap.  124);  even  a  remote  history  of  international  travel  may \nreflect patients\u2019 exposure to infections with pathogens such as M. \ntuberculosis  or  Strongyloides  stercoralis.  Similarly,  domestic  travel \nmay  have  exposed  patients  to  pathogens  that  are  not  normally \nfound in their local environment and therefore may not routinely \nbe considered in the differential diagnosis. For example, a patient \nwho has recently visited California or Martha\u2019s Vineyard may have \nbeen  exposed  to  Coccidioides  immitis  or  Francisella  tularensis, \nrespectively. Beyond simply identifying locations that a patient may \nhave visited, the physician needs to delve deeper to learn what kinds \nof activities and behaviors the patient engaged in during travel (e.g., \nthe types of food and sources of water consumed, freshwater swim-\nming, animal exposures) and whether the patient had the necessary \nimmunizations and/or took the necessary prophylactic medications \nprior to travel; these additional exposures, which the patient may \n\n943\n\nC\nH\nA\nP\nT\nE\nR\n1\n1\n9\n\nA\np\np\nr\no\na\nc\nh\nt\no\nt\nh\ne\nP\na\nt\ni\ne\nn\nt\n\nw\n\ni\nt\nh\na\nn\nI\nn\nf\ne\nc\nt\ni\no\nu\ns\nD\ni\ns\ne\na\ns\ne\n\nnot think to report without specific prompting, are as important as \nexposures during a patient\u2019s routine daily living. \nHost-Specific  Factors  Because  many  opportunistic  infections \n(e.g.,  with  Pneumocystis  jirovecii,  Aspergillus  species,  or  JC  virus) \naffect primarily immunocompromised patients, it is of vital impor-\ntance to determine the immune status of the patient. Defects in the \nimmune system may be due to an underlying disease (e.g., malig-\nnancy, HIV infection, malnutrition), a medication (e.g., chemother-\napy, glucocorticoids, monoclonal antibodies to components of the \nimmune system), a treatment modality (e.g., total body irradiation, \nsplenectomy), or a primary immunodeficiency. The type of infection \nfor which the patient is at increased risk varies with the specific type \nof immune defect. In concert with determining whether a patient is \nimmunocompromised for any reason, the physician should review \nthe  immunization  record  to  ensure  that  the  patient  is  adequately \nprotected against vaccine-preventable diseases (Chap. 123). \n\nPHYSICAL EXAMINATION\nLike  the  history,  a  thorough  physical  examination  is  crucial  in \nevaluating patients with an infectious disease. Some elements of the \nphysical exam (e.g., skin, lymphatics) that are often performed in a \ncursory manner as a result of the ever-increasing pace of medical \npractice  may  help  identify  the  underlying  diagnosis.  Moreover, \nserial exams are critical since new findings may appear as the illness \nprogresses. A description of all the elements of a physical exam is \nbeyond  the  scope  of  this  chapter,  but  the  following  components \nhave particular relevance to infectious diseases. \n\nVital Signs  Given that elevations in temperature are often a hall-\nmark  of  infection,  paying  close  attention  to  the  temperature  may \nbe of value in diagnosing an infectious disease. The idea that 37\u00b0C \n(98.6\u00b0F) is the normal human body temperature dates back to the \nnineteenth  century  and  was  initially  based  on  axillary  measure-\nments.  Rectal  temperatures  more  accurately  reflect  the  core  body \ntemperature and are 0.4\u00b0C (0.7\u00b0F) and 0.8\u00b0C (1.4\u00b0F) higher than oral \nand axillary temperatures, respectively. This idea of a \u201cnormal\u201d body \ntemperature does not take into account the fact that temperatures \ntend to be higher later in the day, in women, and in younger people. \nMoreover,  the  average  body  temperature  seems  to  have  dropped \n~0.03\u00b0C  every  decade  since  the  early  1800s  to  a  new  normal  of \n~36.7\u00b0C. Although the definition of fever varies greatly throughout \nthe medical literature, the most common definition, which is based \non studies defining fever of unknown origin (Chap. 20), uses a core \ntemperature  \u226538.3\u00b0C  (\u2265101\u00b0F).  Although  fever  is  very  commonly \nassociated  with  infection,  it  is  also  documented  in  many  other \ndiseases (Chap. 18). For every 1\u00b0C (1.8\u00b0F) increase in core temper-\nature, the heart rate typically rises by 15\u201320 beats/min. Table 119-1 \nlists infections that are associated with relative bradycardia (Faget\u2019s \nsign), where patients have a lower heart rate than might be expected \nfor  a  given  body  temperature.  Although  this  pulse\u2013temperature \ndissociation  is  not  highly  sensitive  or  specific  for  establishing  a \ndiagnosis, it is potentially useful in low-resource settings given its \nready availability and simplicity. \n\nLymphatics  There  are  ~600  lymph  nodes  throughout  the  body, \nand  infections  are  an  important  cause  of  lymphadenopathy.  A \nphysical  examination  should  include  evaluation  of  lymph  nodes \nin  multiple  regions  (e.g.,  popliteal,  inguinal,  epitrochlear,  axillary, \nmultiple cervical regions), with notation of the location, size (nor-\nmal,  <1  cm),  presence  or  absence  of  tenderness,  and  consistency \n(soft, firm, or rubbery) and of whether the nodes are matted (i.e., \nconnected  and  moving  together).  Nodes  that  are  small  and  firm \ncan also be described as \u201cshotty,\u201d referring to the size and consis-\ntency of buckshot pellets. Of note, palpable epitrochlear nodes are \nalways  pathologic.  Of  patients  presenting  with  lymphadenopathy,   \n75% have localized findings, and the remaining 25% have general-\nized lymphadenopathy (i.e., that involving more than one anatomic \nregion). Localized lymphadenopathy in the head and neck region is \n\nHPIM21e_Part05_p941-p1796.indd   943\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n \n \n \n"
  },
  {
    "page": "944",
    "text": "944\n\nTABLE 119-1  Causes of Relative Bradycardia\nInfectious Causes\n\nIntracellular organisms\n  Gram-negative bacteria\n\n  Tick-borne organisms\n\n  Other\n\nViruses/viral infections\n\nNoninfectious Causes\n\nSalmonella typhi\nFrancisella tularensis\nBrucella spp.\nCoxiella burnetii (Q fever)\nLeptospira interrogans\nLegionella pneumophila\nMycoplasma pneumoniae\nRickettsia spp.\nOrientia tsutsugamushi (scrub typhus)\nBabesia spp.\nCorynebacterium diphtheriae\nPlasmodium spp. (malaria)\nYellow fever virus\nDengue virus\nViral hemorrhagic feversa\nViral myocarditis\n\nDrug fever\nBeta blocker use\nCentral nervous system lesions\nMalignant lymphoma\nFactitious fever\n\naPrimarily early in the course of infection with Marburg or Ebola virus.\n\nfound in 55% of patients, inguinal lymphadenopathy in 14%, and \naxillary lymphadenopathy in 5%. Determining whether the patient \nhas generalized versus localized lymphadenopathy can help narrow \nthe differential diagnosis, as various infections present differently. \nSkin  The fact that many infections have cutaneous manifestations \ngives the skin examination particular importance in the evaluation \nof patients (Chaps. 19, 58, 129, and A1). It is important to perform \na complete skin exam, with attention to both front and back. Spe-\ncific rashes are often extremely helpful in narrowing the differential \ndiagnosis of an infection (Chaps. 19 and A1). In numerous anec-\ndotal instances, patients in the intensive care unit have had \u201cfever \nof unknown origin\u201d that was actually due to unrecognized pressure \nulcers.  Moreover,  close  examination  of  the  distal  extremities  for \nsplinter hemorrhages, Janeway lesions, or Osler\u2019s nodes may yield \nevidence of endocarditis or other causes of septic emboli. \nForeign  Bodies  As  previously  mentioned,  many  infections  are \ncaused by members of the indigenous microbiota. These infections \ntypically  occur  when  these  microbes  escape  their  normal  habitat \nand  enter  a  new  one.  Thus,  maintenance  of  epithelial  barriers \nis  one  of  the  most  important  mechanisms  in  protection  against \ninfection.  However,  hospitalization  of  patients  is  often  associated \nwith breaches of these barriers\u2014e.g., due to placement of IV lines, \nsurgical drains, or tubes (e.g., endotracheal tubes and Foley cathe-\nters)  that  allow  microorganisms  to  localize  in  sites  to  which  they \nnormally would not have access (Chap. 142). Accordingly, knowing \nwhat lines, tubes, and drains are in place is helpful in ascertaining \nwhat body sites might be infected. \nDIAGNOSTIC TESTING\nLaboratory  and  radiologic  testing  has  advanced  greatly  over  the \npast few decades and has become an important component in the \nevaluation of patients. The dramatic increase in the number of sero-\nlogic diagnostics, antigen tests, and molecular diagnostics available \nto the physician has, in fact, revolutionized medical care. However, \nall  of  these  tests  should  be  viewed  as  adjuncts  to  the  history  and \nphysical  examination\u2014not  a  replacement  for  them.  The  selection \n\nof initial tests should be based directly on the patient\u2019s history and \nphysical  exam  findings.  Moreover,  diagnostic  testing  should  gen-\nerally be limited to those conditions that are reasonably likely and \ntreatable, important in terms of public health considerations, and/\nor capable of providing a definitive diagnosis that will consequently \nlimit other testing. \nWhite Blood Cell (WBC) Count  Elevations in the WBC count are \noften  associated  with  infection,  though  many  viral  infections  are \nassociated with leukopenia. It is important to assess the WBC dif-\nferential, given that different classes of microbes are associated with \nvarious leukocyte types. For example, bacteria are associated with \nan increase in polymorphonuclear neutrophils, often with elevated \nlevels  of  earlier  developmental  forms  such  as  bands;  viruses  are \nassociated  with  an  increase  in  lymphocytes;  and  certain  parasites \nare associated with an increase in eosinophils. Table 119-2 lists the \nmajor infectious causes of eosinophilia. \nInflammatory Markers  The erythrocyte sedimentation rate (ESR) \nand  the  C-reactive  protein  (CRP)  level  are  indirect  and  direct \nmeasures  of  the  acute-phase  response,  respectively,  that  can  be \nused to assess a patient\u2019s general level of inflammation. Moreover, \nthese markers can be followed serially over time to monitor disease \nprogress/resolution. It is noteworthy that the ESR changes relatively \nslowly, and its measurement more often than weekly usually is not \nuseful;  in  contrast,  CRP  concentrations  change  rapidly,  and  daily \nmeasurements can be useful in the appropriate context. Although \nthese  markers  are  sensitive  indicators  of  inflammation,  neither  is \nvery specific. An extremely elevated ESR (>100 mm/h) has a 90% \npredictive  value  for  a  serious  underlying  disease  (Table  119-3). \nWork is ongoing to identify other potentially useful inflammatory \nmarkers  (e.g.,  procalcitonin,  serum  amyloid  A  protein);  however, \ntheir clinical utility requires further validation. \nAnalysis of Cerebrospinal Fluid (CSF)  Assessment of CSF is criti-\ncal for patients with suspected meningitis or encephalitis. An open-\ning pressure should always be recorded, and fluid should routinely \nbe sent for cell counts, Gram\u2019s stain and culture, and determination \nof glucose and protein levels. A CSF Gram\u2019s stain typically requires \n>105  bacteria/mL  for  reliable  positivity;  its  specificity  approaches \n100%. Table 119-4 lists the typical CSF profiles for various infec-\ntions.  In  general,  CSF  with  lymphocytic  pleocytosis  and  a  low \nglucose concentration suggests either infection (e.g., with Listeria, \nM.  tuberculosis,  or  a  fungus)  or  a  noninfectious  disorder  (e.g., \nneoplastic  meningitis,  sarcoidosis).  Bacterial  antigen  tests  of  CSF   \n(e.g.,  latex  agglutination  tests  for  Haemophilus  influenzae  type  b, \ngroup  B  Streptococcus,  S.  pneumoniae,  and  Neisseria  meningitidis) \nare  not  recommended  for  screening,  given  that  these  tests  are  no \nmore sensitive than Gram\u2019s stain; however, these assays can be help-\nful  in  presumptively  identifying  organisms  seen  on  Gram\u2019s  stain. \nIn  contrast,  other  antigen  tests  (e.g.,  for  Cryptococcus)  and  some \nCSF  serologic  testing  (e.g.,  for  Treponema  pallidum,  Coccidioides) \nare highly sensitive and are useful for select patients. In addition, \npolymerase  chain  reaction  (PCR)  analysis  of  CSF  is  increasingly \nbeing  used  for  the  diagnosis  of  bacterial  (e.g.,  N.  meningitidis,  S. \npneumoniae,  mycobacteria)  and  viral  (e.g.,  herpes  simplex  virus, \nenterovirus)  infections;  while  these  molecular  tests  permit  rapid \ndiagnosis with a high degree of sensitivity and specificity, they often \ndo not allow determination of antimicrobial resistance profiles. \nCultures  The  mainstays  of  infectious  disease  diagnosis  include \nthe culture of infected tissue (e.g., surgical specimens) or fluid (e.g., \nblood, urine, sputum, pus from a wound). Samples can be sent for \nculture of bacteria (aerobic or anaerobic), fungi, or viruses. Ideally, \nspecimens are collected before the administration of antimicrobial \ntherapy;  in  instances  where  this  order  of  events  is  not  clinically \nfeasible,  microscopic  examination  of  the  specimen  (e.g.,  Gram-\nstained  or  potassium  hydroxide  [KOH]\u2013treated  preparations)  is \nparticularly important. Culture of the organism(s) allows identifi-\ncation of the etiologic agent(s), determination of the antimicrobial \n\nHPIM21e_Part05_p941-p1796.indd   944\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases\u00a0\n\u00a0\n"
  },
  {
    "page": "945",
    "text": "TABLE 119-2  Major Infectious Causes of Eosinophiliaa\nORGAN INVOLVED\nCentral nervous system\n\nORGANISM\nAngiostrongylus\nGnathostoma\nLoa loa\n\nEXPOSURE\nRaw seafood\nRaw poultry and seafood\nInsect bite\n\nGEOGRAPHIC DISTRIBUTION\nAsia\nAsia\nAfrica\n\nOnchocerca\n\nInsect bite\n\nAfrica\n\nEye\n\nLung\n\nLiver\n\nIntestines\n\nBladder\nMuscle\nLymphatics\n\nOther\n\nWorldwide\nTropical\nWorldwide\nAsia\nSouthwestern United States\nAsia\nWorldwide\nAsia\nAfrica, Middle East, Latin \nAmerica\nWorldwide\nAsia\nAsia\nWorldwide\n\nChlamydia trachomatis\nStrongyloides\nToxocara canis/Toxocara cati c\nParagonimus\nCoccidioides immitis\nBrugia malayi\nPneumocystis jirovecii\nSchistosoma japonicum\nSchistosoma mansoni\n\nSexual transmission\nSoil\nDogs, soil\nCrabs and crayfish\nSoil\nInsect bite\nAir\nFreshwater swimming\nFreshwater swimming\n\nFasciola\nClonorchis\nOpisthorchis\nAscarisd\n\nHookworm\nTrichuris\n\nCystoisospora belli\nDientamoeba fragilis\n\nCapillaria\nHeterophyes\nAnisakis\nBaylisascaris procyonise\nHymenolepis nana\nSchistosoma haematobium\nTrichinella\nWuchereria bancrofti d\nBartonella henselae\nRecovery from bacterial or viral \ninfections\nHIV\nCryptococcus neoformans\n\nWorldwide\nTropical\n\nWatercress\nRaw seafood\nRaw seafood\nRaw fruits and vegetables, \ncontaminated water\nSoil\nRaw fruits and vegetables, \ncontaminated water\nContaminated water and food Worldwide\nWorldwide\nUnclear; spread via fecal\u2013oral \nroute\nRaw seafood\nRaw seafood\nRaw seafood\nSoil\nContaminated water, soil\nFreshwater swimming\nPork\nInsect bite\nCats\n\u2014\n\nAsia\nAsia, Middle East\nWorldwide\nNorth America\nWorldwide\nAfrica, Middle East\nWorldwide\nTropical\nWorldwide\n\u2014\n\nDEGREE OF EOSINOPHILIAb\nMild\nModerate to extreme\nModerate (expatriates), mild \n(patients living in endemic areas)\nMild (expatriates), moderate \n(patients living in endemic areas)\nMild\nModerate (acute), mild (chronic)\nModerate to extreme\nModerate (acute), mild (chronic)\nMild (acute), extreme (disseminated)\nMild to moderate\nMild\nModerate (acute), mild (chronic)\nModerate (acute), mild (chronic)\n\nModerate\nMild to moderate\nMild to moderate\nMild to extreme\n\nMild to moderate\nMild\n\nMild\nMild\n\nExtreme\nMild\nMild\nModerate to extreme\nMild\nModerate (acute), mild (chronic)\nModerate to extreme\nModerate to extremef\nMild\nMild\n\n945\n\nC\nH\nA\nP\nT\nE\nR\n1\n1\n9\n\nA\np\np\nr\no\na\nc\nh\nt\no\nt\nh\ne\nP\na\nt\ni\ne\nn\nt\n\nw\n\ni\nt\nh\na\nn\nI\nn\nf\ne\nc\nt\ni\no\nu\ns\nD\ni\ns\ne\na\ns\ne\n\nContaminated bodily fluid\nSoil\n\nWorldwide\nWorldwide\n\nMild\nModerate to extreme (disseminated)\n\naThere are numerous noninfectious causes of eosinophilia, such as atopic disease, DRESS (drug reaction with eosinophilia and systemic symptoms) syndrome, and \npernicious anemia, which can cause mild eosinophilia; drug hypersensitivity and serum sickness, which can cause mild to moderate eosinophilia; collagen vascular \ndisease, which can cause moderate eosinophilia; and malignancy, Churg-Strauss syndrome, and hyper-IgE syndromes, which can cause moderate to extreme eosinophilia. \nbMild: 500\u20131500 cells/\u03bcL; moderate: 1500\u20135000 cells/\u03bcL; extreme: >5000 cells/\u03bcL. cCan also affect the liver and the eyes. dCan also affect the lungs. eCan also affect the eyes \nand the central nervous system. fLevels are typically higher with pulmonary infections.\n\nsusceptibility  profile,  and\u2014when  there  is  concern  about  an   \noutbreak\u2014isolate typing. While cultures are extremely useful in the \nevaluation of patients, determining whether culture results are clin-\nically  meaningful  or  represent  contamination  (e.g.,  a  non-aureus, \nnon-lugdunensis staphylococcal species growing in a blood culture) \ncan sometimes be challenging and requires an understanding of the \npatient\u2019s immune status, exposure history, and microbiota. In some \ncases, serial cultures to demonstrate clearance of the organism may \nbe helpful. \n\nPathogen-Specific  Testing  Numerous  pathogen-specific  tests \n(e.g., serology, antigen testing, PCR testing) are commercially avail-\nable,  and  many  hospitals  now  offer  some  of  these  tests  in-house \nto  facilitate  rapid  turnaround  that  ultimately  enhances  patient \ncare. The reader is directed to relevant chapters on the pathogens \n\nof  interest  for  specific  details.  Some  of  these  tests  (e.g.,  universal \nPCRs)  identify  organisms  that  currently  are  not  easily  cultivable \nand  have  unclear  relationships  to  disease,  thereby  complicating \ndiagnosis. As these tests become more commonplace and the work \nof  the  Human  Microbiome  Project  progresses,  the  relevance  of \nsome  of  these  previously  unrecognized  bacteria  to  human  health \nwill likely become more apparent. \nRadiology \nImaging provides an important adjunct to the physical \nexamination, allowing evaluation for lymphadenopathy in regions \nthat are not externally accessible (e.g., mediastinum, intraabdomi-\nnal sites), assessment of internal organs for evidence of infection, \nand  facilitation  of  image-guided  percutaneous  sampling  of  deep \nspaces. The choice of imaging modality (e.g., CT, MRI, ultrasound, \nnuclear medicine, use of contrast) is best made in consultation with \n\nHPIM21e_Part05_p941-p1796.indd   945\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n \n \n \n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n"
  },
  {
    "page": "946",
    "text": "946\n\nTABLE 119-3  Causes of an Extremely Elevated Erythrocyte \nSedimentation Rate (>100 mm/h)\nETIOLOGIC CATEGORY (% OF CASES)\nInfectious diseases (35\u201340)\n\nInflammatory diseases (15\u201320)\n\nMalignancies (15\u201320)\n\nOther (20\u201335)\n\nSPECIFIC CAUSES\nSubacute bacterial endocarditis\nAbscesses\nOsteomyelitis\nTuberculosis\nUrinary tract infection\nGiant cell arteritis\nRheumatoid arthritis\nSystemic lupus erythematosus\nMultiple myeloma\nLeukemias\nLymphomas\nCarcinomas\nDrug hypersensitivity reactions (drug \nfever)\nIschemic tissue injury/trauma\nRenal diseases\n\na radiologist to ensure that the results will address the physician\u2019s \nspecific concerns. \nTREATMENT\nPhysicians often must balance the need for empirical antibiotic treat-\nment with the patient\u2019s clinical condition. When clinically feasible, it \nis best to obtain relevant samples (e.g., blood, CSF, tissue, purulent \nexudate)  for  culture  prior  to  the  administration  of  antibiotics,  as \nantibiotic treatment often makes subsequent diagnosis more difficult. \nAlthough a general maxim for antibiotic treatment is to use a regi-\nmen with as narrow a spectrum as possible (Chap. 144), empirical \nregimens are necessarily somewhat broad, given that a specific diag-\nnosis has not yet been made. Table 119-5 lists empirical antibiotic \ntreatment regimens for commonly encountered infectious presen-\ntations. These regimens should be narrowed as appropriate once a \nspecific diagnosis is made. In addition to antibiotics, there is some-\ntimes  a  role  for  adjunctive  therapies,  such  as  intravenous  immu-\nnoglobulin G (IVIG) pooled from healthy adults or hyperimmune \n\nglobulin  prepared  from  the  blood  of  individuals  with  high  titers \nof  specific  antibodies  to  select  pathogens  (e.g.,  cytomegalovirus, \nhepatitis  B  virus,  rabies  virus,  vaccinia  virus,  Clostridium  tetani, \nvaricella-zoster virus, Clostridium botulinum toxin). Although the \ndata  suggesting  efficacy  are  limited,  IVIG  is  sometimes  used  for \npatients with suspected staphylococcal or streptococcal toxic shock \nsyndrome. \nINFECTION CONTROL\nWhen evaluating a patient with a suspected infectious disease, the \nphysician must consider what infection control methods are neces-\nsary to prevent transmission of any possible infection to other peo-\nple. In 2007, the U.S. Centers for Disease Control and Prevention \npublished guidelines for isolation precautions that are available for \ndownload  at  www.cdc.gov/hicpac/2007IP/2007isolationPrecautions.\nhtml.  Persons  exposed  to  certain  pathogens  (e.g.,  N.  meningitidis, \nHIV,  Bacillus  anthracis)  should  receive  postexposure  prophylaxis \nto prevent disease acquisition. (See relevant chapters for details on \nspecific pathogens.) \nWHEN TO OBTAIN AN INFECTIOUS DISEASE CONSULT\nAt times, primary physicians need assistance with patient manage-\nment  from  a  diagnostic  and/or  therapeutic  perspective.  Multiple \nstudies  have  demonstrated  that  an  infectious  disease  consult  is \nassociated with improved outcomes, shorter length of hospital stay, \nand decreased costs for patients with various diseases. For example, \nin a prospective cohort study of patients with S. aureus bacteremia, \ninfectious disease consultation was independently associated with a \n56%  reduction  in  28-day  mortality.  In  addition,  infectious  disease \nspecialists  provide  other  services  (e.g.,  infection  control,  antimi-\ncrobial  stewardship,  management  of  outpatient  antibiotic  therapy, \noccupational exposure programs) that have been shown to benefit \npatients.  Whenever  such  assistance  would  be  advantageous  to  a \npatient  with  a  possible  infection,  the  primary  physician  should \nopt for an infectious disease consult. Specific situations that might \nprompt a consult include (1) difficult-to-diagnose patients with pre-\nsumed infections, (2) patients who are not responding to treatment \nas  expected,  (3)  patients  with  a  complicated  medical  history  (e.g., \norgan transplant recipients, patients immunosuppressed due to auto-\nimmune or inflammatory conditions), and (4) patients with \u201cexotic\u201d \ndiseases (i.e., diseases that are not typically seen within the region).\n\nTABLE 119-4  Typical Cerebrospinal Fluid Profiles for Meningitis and Encephalitisa\nFUNGAL \nBACTERIAL \nMENINGITISb\nMENINGITIS\n40\u2013600\n>1000\n\nVIRAL \nMENINGITIS\n25\u2013500\n\nNORMAL\n<5\n\nWBC count (per \n\u03bcL)\nDifferential of WBC 60\u201370% \n\nlymphocytes, \n\u226430% monocytes/\nmacrophages\nNegative\n\nGram\u2019s stain\n\nGlucose (mg/dL)\n\n40\u201385\n\n15\u201345\n50\u2013180\n\nProtein (mg/dL)\nOpening pressure \n(mmH2O)\nCommon causes \u2014\n\nLymphocytes or \nPMNs, depending \non specific \norganism\nRarely positive\n\n\u2193 to normal\n\n150\u2013300\n160\u2013340\n\n\u2191\u2191PMNs (\u226580%)\n\nPredominantly \nlymphocytesc\n\nNegative\n\nNormal\n\n20\u201380\n100\u2013350\n\nPositive (in >60% \nof cases)\n<40\n\n>100\n>300\n\nStreptococcus \npneumoniae, \nNeisseria \nmeningitidis\n\nEnteroviruses\n\nCandida, \nCryptococcus, and \nAspergillus spp.\n\nPARASITIC \nMENINGITIS\n150\u20132000\n\nTUBERCULOUS \nMENINGITIS\n25\u2013100\n\nENCEPHALITIS\n50\u2013500\n\n\u2191\u2191 Eosinophils \n(\u226550%)d\n\nPredominantly \nlymphocytesc\n\nPredominantly \nlymphocytesc\n\nNegative\n\nNormal\n\n50\u2013200\nNormal\n\nAngiostrongylus \ncantonensis, \nGnathostoma \nspinigerum, \nBaylisascaris \nprocyonis\n\nOccasionally \npositivee\n<50 in 75% of \ncases\n100\u2013200\n150\u2013280\n\nMycobacterium \ntuberculosis\n\nNegative\n\nNormal\n\n50\u2013100\nNormal to \u2191\n\nHerpesviruses, \nenteroviruses, \ninfluenza virus, \nrabies virus\n\naNumbers indicate typical results, but actual results may vary. bCerebrospinal fluid characteristics depend greatly on the specific organism. cNeutrophils may predominate \nearly in the disease course. dPatients typically have striking eosinophilia as well. eSensitivity can be increased by examination of a smear of protein coagulum (pellicle) and \nthe use of acid-fast stains.\nAbbreviations: PMNs, polymorphonuclear neutrophils; WBC, white blood cell.\n\nHPIM21e_Part05_p941-p1796.indd   946\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases\u00a0\n"
  },
  {
    "page": "947",
    "text": "TABLE 119-5  Initial Empirical Antibiotic Therapy for Common Infectious Disease Presentationsa\nCLINICAL SYNDROME\nSeptic shock\n\nCOMMON ETIOLOGIES\nStaphylococcus aureus, \nStreptococcus pneumoniae, \nenteric gram-negative bacilli\n\nANTIBIOTIC(S)\nVancomycin, 15 mg/kg q12hb\nplus\nA broad-spectrum antipseudomonal \u03b2-lactam \n(piperacillin-tazobactam, 4.5 g q6h; imipenem,  \n1 g q8h; meropenem, 1 g q8h; or cefepime, 1\u20132 g \nq8\u201312h)\nVancomycin, 15 mg/kg q12hb\nplus\nCeftriaxone, 2 g q12h\n\nMeningitis\n\nS. pneumoniae, Neisseria \nmeningitidis\n\nCNS abscess\n\nStreptococcus spp., \nStaphylococcus spp., \nanaerobes, gram-negative \nbacilli\n\nAcute endocarditis \n(native valve)\n\nS. aureus, Streptococcus \nspp., coagulase-negative \nstaphylococci\n\nPneumonia\n\n Community-acquired, \noutpatient\n\nS. pneumoniae, Mycoplasma \npneumoniae, Haemophilus \ninfluenzae, Chlamydia \npneumoniae\n\nInpatient, non-ICU\n\nAbove plus Legionella spp.\n\nInpatient, ICU\n\nAbove plus S. aureus\n\n Hospital-acquired \npneumoniad\n\nS. pneumoniae, H. influenzae, \nS. aureus, gram-negative bacilli \n(e.g., Pseudomonas aeruginosa, \nKlebsiella pneumoniae, \nAcinetobacter spp.)\n\nComplicated \nintraabdominal \ninfection\n\n Mild to moderate \nseverity\n\nAnaerobes (Bacteroides \nspp., Clostridium spp.), gram-\nnegative bacilli (Escherichia \ncoli), Streptococcus spp.\n\n High-risk patient \nor high degree of \nseverity\n\nSame as above\n\nVancomycin, 15 mg/kg q12hb\nplus\nCeftriaxone, 2 g q12h\nplus\nMetronidazole, 500 mg q8h\nVancomycin, 15 mg/kg q12hb\nplus\nCefepime, 2 g q8h\nNo comorbiditiesh:\nAzithromycin, 500 mg PO \u00d7 1, then 250 mg PO qd \u00d7 \n4 days\nWith comorbiditiesh:\nLevofloxacin, 750 mg PO qd\n\nA respiratory fluoroquinolone (moxifloxacin,  \n400 mg IV/PO qd; gemifloxacin, 320 mg PO qd; or \nlevofloxacin, 750 mg IV/PO qd)\nor\nA \u03b2-lactam (cefotaxime, ceftriaxone, or ampicillin-\nsulbactam) plus azithromycin\nA \u03b2-lactam\nplus\nAzithromycin or a respiratory fluoroquinolone\nAn antipseudomonal \u03b2-lactam (cefepime, 2 g \nq8h; ceftazidime, 2 g q8h; imipenem, 500 mg q6h; \nmeropenem, 1 g q8h; or piperacillin-tazobactam, \n4.5 g q6h)\nplus\nAn antipseudomonal fluoroquinolone (levofloxacin, \n700 mg qd, or ciprofloxacin, 400 mg q8h) or an \naminoglycoside (amikacin, 15\u201320 mg/kg q24hc; \ngentamicin, 5\u20137 mg/kg q24he; or tobramycin,  \n5\u20137 mg/kg q24he)\n\nCefoxitin, 2 g q6h\nor\nA combination of metronidazole (500 mg q8\u201312h) \nplus one of the following: cefazolin (1\u20132 g q8h), \ncefuroxime (1.5 g q8h), ceftriaxone (1\u20132 g q12\u201324h), \ncefotaxime (1\u20132 g q6\u20138h), ciprofloxacin (400 mg \nq12h), levofloxacin (750 mg qd)\nA carbapenem (imipenem, 500 mg q6h; meropenem, \n1 g q8h; doripenem, 500 mg q8h)\nor\nPiperacillin-tazobactam, 3.375 g q6hf\nor\nA combination of metronidazole (500 mg q8h) plus \nan antipseudomonal cephalosporin (cefepime, 2 g \nq8h; ceftazidime, 2 g q8h)\n\nCOMMENTS\nIf a pseudomonal species \nis likely, a second \nantipseudomonal agent \nshould be added.\n\nSEE CHAPTER(S)\n304\n\nDexamethasone  \n(0.15 mg/kg IV q6h for  \n2\u20134 d) should be added for \npatients with suspected \nor proven pneumococcal \nmeningitis, with the first \ndose administered 10\u201320 \nmin before the first dose of \nantibiotics.\n\u2014\n\n\u2014\n\n138 and pathogen-\nspecific chapters\n\n138\n\n128\n\n126 and pathogen-\nspecific chapters\n\nIf MRSA is a consideration, \nadd vancomycin (15 mg/kg \nq8\u201312hb) or linezolid (600 mg \nq12h); daptomycin should \nnot be used in patients with \npneumonia.\n\nIf MRSA is a consideration, \nadd vancomycin (15 mg/kg \nq8\u201312hb) or linezolid (600 mg \nq12h); daptomycin should \nnot be used in patients with \npneumonia.\n\nIf MRSA is a consideration, \nadd vancomycin (15 mg/kg \nq12hb)\n\n132, 177, and \npathogen-specific \nchapters\n\n947\n\nC\nH\nA\nP\nT\nE\nR\n1\n1\n9\n\nA\np\np\nr\no\na\nc\nh\nt\no\nt\nh\ne\nP\na\nt\ni\ne\nn\nt\n\nw\n\ni\nt\nh\na\nn\nI\nn\nf\ne\nc\nt\ni\no\nu\ns\nD\ni\ns\ne\na\ns\ne\n\n(Continued)\n\nHPIM21e_Part05_p941-p1796.indd   947\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n \n \n \n \n \n\u00a0\n\u00a0\n \n\u00a0\n\u00a0\n \n\u00a0\n \n \n \n \n \n \n\u00a0\n\u00a0\n"
  },
  {
    "page": "948",
    "text": "948\n\nTABLE 119-5  Initial Empirical Antibiotic Therapy for Common Infectious Disease Presentationsa\nCLINICAL SYNDROME\n\nANTIBIOTIC(S)\n\nSkin and soft tissue \ninfection\n\nCOMMON ETIOLOGIES\nS. aureus, Streptococcus \npyogenes\n\nDicloxacillin, 250\u2013500 mg PO qid\nor\nCephalexin, 250\u2013500 mg PO qid\nor\nClindamycin, 300\u2013450 mg PO tid\nor\nNafcillin/oxacillin, 1\u20132 g q4h\n\n(Continued)\n\nCOMMENTS\n\nIf MRSA is a consideration, \nclindamycin, vancomycin  \n(15 mg/kg q12hb), linezolid \n(600 mg IV/PO q12h), or \nTMP-SMX (1\u20132 double-\nstrength tablets PO bidg) can \nbe used.\n\nSEE CHAPTER(S)\n\n129 and pathogen-\nspecific chapters\n\naThis table refers to immunocompetent adults with normal renal and hepatic function. All doses listed are for parenteral administration unless indicated otherwise. Local \nantimicrobial susceptibility profiles may influence the choice of antibiotic. Therapy should be tailored once a specific etiologic agent and its susceptibilities are identified. \nbTrough levels for vancomycin should be 15\u201320 \u03bcg/mL. cTrough levels for amikacin should be <4 \u03bcg/mL. dIn patients with late onset (i.e., after \u22655 days of hospitalization) or \nrisk factors for multidrug-resistant organisms. eTrough levels for gentamicin and tobramycin should be <1 \u03bcg/mL. fIf P. aeruginosa is a concern, the dosage may be increased \nto 3.375 g IV q4h or 4.5 g IV q6h. gData on the efficacy of TMP-SMX in skin and soft tissue infections are limited. hComorbidities include chronic heart, lung, liver, or renal \ndisease; diabetes mellitus; alcoholism; malignancy; or asplenia.\nAbbreviations: CNS, central nervous system; ICU, intensive care unit; MRSA, methicillin-resistant S. aureus; TMP-SMX, trimethoprim-sulfamethoxazole.\n\n \u25a0 PERSPECTIVE\nThe study of infectious diseases is really a study of host\u2013microbial inter-\nactions and represents evolution by both the host and the microbe\u2014an \nendless struggle in which microbes have generally been more creative \nand adaptive. Given that nearly one-fifth of deaths worldwide are still \nrelated to infectious diseases, it is clear that the war against infectious \ndiseases has not been won. For example, a cure for HIV infection is still \nlacking, there have been only marginal improvements in the methods \nfor detection and treatment of tuberculosis after more than a half cen-\ntury  of  research,  new  infectious  disease  outbreaks  (e.g.,  viral  hemor-\nrhagic fevers, Zika, SARS-CoV-2) continue to emerge, and the threat of \nmicrobial bioterrorism remains high. The subsequent chapters in Part 5  \ndetail\u2014on  both  a  syndrome  and  a  microbe-by-microbe  basis\u2014the \ncurrent state of medical knowledge about infectious diseases. At their \ncore, all of these chapters carry a similar message: Despite numerous \nadvances  in  the  diagnosis,  treatment,  and  prevention  of  infectious \ndiseases, much work and research are required before anyone can con-\nfidently claim we have achieved \u201cthe virtual elimination of infectious \ndisease.\u201d  In  reality,  this  goal  will  never  be  attained,  given  the  rapid \nadaptability of microbes.\n \u25a0 FURTHER READING\nBartlett JG: Why infectious diseases. Clin Infect Dis 59(Suppl 2):S85, \n\n2014.\n\nBhatraju  PK  et  al:  Covid-19  in  critically  ill  patients  in  the  Seattle \n\nregion: Case series. N Engl J Med 382:2012, 2020.\n\nKhabbaz RF et al: Challenges of infectious diseases in the USA. Lancet \n\n384:53, 2014.\n\nMarston HD et al: Antimicrobial resistance. JAMA 316:1193, 2016.\nMcQuillen  DP,  MacIntyre  AT:  The  value  that  infectious  disease \nphysicians bring to the healthcare system. J Infect Dis 216:S588, 2017.\nVerghese A et al: Inadequacies of physical examination as a cause of \nmedical  errors  and  adverse  events:  A  collection  of  vignettes.  Am  J \nMed 128:1322, 2015.\n\n120 Molecular Mechanisms of \nMicrobial Pathogenesis\nThomas E. Wood, Marcia B. Goldberg\n\nInfectious diseases of humans involve intricate interactions among the \ninfecting  microbe,  human  tissue,  and  the  host  microbiome.  The  co- \nevolution of humans and microbes has led to the presence of numerous \nand varied specific microbial factors that promote the disease process \nand  a  corresponding  wide  range  of  human  cellular  responses,  both \n\nspecific  and  nonspecific,  to  pathogens.  Among  the  microbial  factors \nthat promote disease are those that alter human cells, those that inhibit \nhost immune responses, and those that respond to the microbes that \nconstitute  the  microbiota  and  their  metabolic  products.  The  process \nof infection can be divided into several stages: the encounter with and \nentry of the microbe into the human body (colonization), the attach-\nment of the microbe in its favored niche and microbial avoidance of \nhost defenses (infection), the combined deployment of microbial fac-\ntors that damage human tissue and host inflammatory responses to the \npresence of the microbe (disease), and the release of the pathogen into \nthe environment, where it can infect others (transmission). It is notable \nthat  for  most  microbial  pathogens,  the  host  inflammatory  response \ncontributes substantially to symptoms and to tissue damage. Moreover, \nthe  human  microbiota  (the  collection  of  microbes  that  reside  in  and \non  the  human  body)  modulates,  directly  or  indirectly,  every  stage  of \ninfection  (Chap.  471).  This  chapter  describes  the  best-understood \nmolecular and cellular mechanisms that contribute to human disease \ncaused by bacterial pathogens.\n\nENTRY INTO THE HUMAN HOST\nInfectious diseases occur when either a live pathogen enters the human \nhost or a toxic pathogen product is ingested by the host, with the for-\nmer being much more common than the latter.\n\nMost  bacterial  infections  result  from  entry  of  a  pathogen  into  the \nbody. Entry can occur through a break in the skin into the underlying \nsoft tissues or at the surface of a mucous membrane of the respiratory, \ngastrointestinal,  or  genitourinary  tract;  the  skin  may  also  be  directly \ninfected. Entry at these sites may result in infection of the bloodstream, \nwhich may in turn lead to infection of other organ systems.\n\nEntry  into  the  respiratory  tract  occurs  via  respiratory  droplet \nnuclei (airborne particles 1\u20135 \u03bcm in diameter) or via fomites intro-\nduced on a hand that is contaminated by contact with a contaminated \ninert surface (e.g., a doorknob or a faucet). Infectious droplet nuclei \nare  generated  when  an  individual  with  a  communicable  respira-\ntory  infection  (e.g.,  tuberculosis,  Legionnaires\u2019  disease,  psittacosis, \ninfluenza,  measles,  chickenpox,  aspergillosis,  COVID-19  infection) \nsneezes, coughs, talks, plays a musical instrument by mouth, or sings. \nA cough may generate 3000 particles, whereas a sneeze may generate \nup  to  40,000  particles;  large  particles  may  evaporate  down  to  the \n0.5-  to  12-\u03bcm  range,  and  particles  that  are  1.5  \u03bcm  and  hygroscopic \nincrease  in  size  as  they  pass  through  the  moist  nasal  passages  and \nlower respiratory tree.\n\nEntry into the gastrointestinal tract occurs via ingestion of contam-\ninated  food  or  water  or  via  person-to-person  contact,  typically  with \ntransfer  from  a  contaminated  hand  into  the  mouth.  Pathogens  for \nwhich the infectious inoculum is large (e.g., 108 organisms for epidemic \nspread of Vibrio cholerae, 105 organisms for Salmonella enterica serovar \nTyphimurium) are generally acquired via contaminated food or water, \nwhereas  pathogens  for  which  the  infectious  inoculum  is  small  (e.g., \n101 to 102 organisms for Shigella spp.) are more commonly acquired by \nperson-to-person spread.\n\nHPIM21e_Part05_p941-p1796.indd   948\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "949",
    "text": "Entry  into  the  genitourinary  tract  generally  takes  place  either  via \ncolonization  of  the  urethral  meatus  or  vaginal  introitus  with  fecal \norganisms followed by ascension of the organisms into the bladder or \nkidneys  or  via  instrumentation.  Pyelonephritis  can  also  result  from \nseeding from the bloodstream.\n\nESTABLISHMENT OF INFECTION\n \u25a0 NICHE\nLive Pathogens  Within the human host, many bacterial pathogens \ndisplay tissue tropism; the sites of infection within the human body are \npathogen-specific and restricted, such that even adjacent tissues may \nbe  uninvolved.  For  example,  group  A  streptococci  cause  pharyngitis \nand soft tissue infection, but rarely pneumonia. Cholera is an infection \nof the small intestine, but not the stomach or the colon, where Shigella \nspp.  cause  disease  only  in  the  rectosigmoid.  Therefore,  to  establish \ninfection, the pathogen must access its niche and then remain within \nthat  niche  (Table  120-1).  In  the  respiratory  tree,  the  sites  at  which \npathogens initially settle can be determined by mode of spread. Droplet \nnuclei reach the bronchial tree or alveoli, whereas, after contaminated \nhands  touch  the  face,  fomites  reach  the  pharynx  or  nasal  passages. \nPathogens move through the gastrointestinal tract via normal intestinal \nmotility.\n\nTissue  association  and  invasion  are  dictated  by  the  interaction  of \nthe  bacterium  with  host  factors,  commonly  glycan-decorated  recep-\ntors  and/or  the  associated  extracellular  matrix.  The  environmental \nconditions  of  the  niche  trigger  the  expression  of  virulence  factors \n\nDISEASE\nAnthrax\n\nBACTERIUM\nBacillus anthracis\n\nTABLE 120-1  Bacterial Pathogens, Diseases, and Niches\nMOST COMMON \nTROPISM\nSkin, respiratory tract, \nsmall intestine\nRespiratory tract\nSystemic\nSystemic\nSystemic\n\nBordetella pertussis\nBorrelia burgdorferi\nBrucella abortus\nBurkholderia \npseudomallei\nChlamydia trachomatis\n\nPertussis\nLyme disease\nBrucellosis\nMelioidosis\n\nEyes, venereal\n\nColon\nPharynx\n\nSystemic\nColon\n\nStomach\nRespiratory tract\nSystemic, central \nnervous system\nRespiratory tract\n\nUrogenital tract\nRespiratory tract\n\nSystemic\n\nGastrointestinal tract\n\nColon, rectum\nMultiple sites\nSoft tissue\nSmall intestine\nSystemic\n\nClostridioides difficile\nCorynebacterium \ndiphtheriae\nCoxiella burnetii\nEnterohemorrhagic \nEscherichia coli\nHelicobacter pylori\nLegionella pneumophila\nListeria monocytogenes\n\nMycobacterium \ntuberculosis\nNeisseria gonorrhoeae\nPseudomonas \naeruginosa\nSalmonella enterica \nserovar Typhi\nSalmonella enterica \nserovar Typhimurium\nShigella spp.\nStaphylococcus aureus\nGroup A Streptococcus\nVibrio cholerae\nYersinia pestis\n\nVarious chlamydioses, \nincluding trachoma\nColitis\nDiphtheria\n\nQ fever\n\nGastritis, gastric ulcers\nLegionnaires\u2019 disease\nListeriosis\n\nTuberculosis\n\nGonorrhea\n\nTyphoid fever\n\nDysentery, shigellosis\n\nCholera\nPlague\n\n949\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n0\n\nM\no\nl\ne\nc\nu\nl\na\nr\n\nM\ne\nc\nh\na\nn\ni\ns\n\nm\n\ns\no\nf\n\nM\ni\nc\nr\no\nb\ni\na\nl\n\nP\na\nt\nh\no\ng\ne\nn\ne\ns\ni\ns\n\nrequired  for  the  establishment  of  infection;  for  example,  bile  salts  in \nthe gut stimulate the expression of V. cholerae adhesins and toxins and \nthe  germination  of  Clostridioides  difficile  spores.  Bacteria  commonly \nmanipulate  their  particular  niche  environment  in  ways  that  facilitate \ninfection.  For  example,  the  gastroduodenal  pathogen  Helicobacter \npylori  produces  urease,  which  converts  urea  into  ammonia,  thereby \nincreasing  the  pH  of  the  acidic  stomach  environment  and  creating  a \nmore  hospitable  environment  for  its  survival.  This  conversion  also \nconcomitantly alters the physiology of the gastric epithelium.\n\nPreformed  Toxins  A  small  number  of  diseases  are  caused  by \ningestion  in  food  of  preformed  bacterial  toxins.  The  most  common \namong  these  are  Staphylococcus  aureus  enterotoxins,  which  can  be \npresent in prepared foods such as dairy, meat, eggs, salads, and pro-\nduce, and the Bacillus cereus emetic enterotoxin, which is most often \nfound in rice or other starchy food that has been improperly refriger-\nated.  Less  common  but  also  important  is  botulinum  toxin  produced \nby Clostridium botulinum. Preformed toxins cause disease in the small \nintestine (nausea and vomiting) and may cause systemic symptoms.\n\nAlthough the pathogens may be killed when food is cooked, their \ntoxins  are  heat  stable.  S.  aureus  enterotoxins  can  be  strongly  emetic. \nB. cereus produces the heat-stable peptide toxin cereulide, which acts \nas  a  potassium  ionophore  and  induces  emesis.  In  botulism,  a  flaccid \nparalysis caused by botulinum toxin, uptake of the toxin blocks neu-\nrotransmitter release in motor neurons, inhibiting the central nervous \nsystem and resulting in potentially fatal respiratory failure.\n \u25a0 ATTACHMENT\nAttachment of bacteria to host tissue surfaces is a prerequisite for the \npathogen\u2019s  establishment  of  an  infection  and  is  mediated  by  specific \nreceptor\u2013ligand  interactions.  In  this  context,  the  tissue  specificity  of \nthe host cell surface receptor repertoire is a critical factor in delimiting \na pathogen\u2019s niche(s). These physical associations additionally facilitate \nthe  pathogen\u2019s  avoidance  of  host  clearance  mechanisms  (see  \u201cAvoid-\nance  of  Innate  Immune  Responses,\u201d  below)  and  may  contribute  to \nformation of biofilms by the pathogen (see \u201cBiofilms,\u201d below). Because \nadhesion to cellular receptors often triggers cellular signal transduction \nand innate immune signaling, therapeutic blocking of this interaction \nmay in some circumstances exacerbate infection.\n\nAdhesins  Bacterial pathogens have evolved a wide range of strate-\ngies by which to attach to the diverse host cell structures they encoun-\nter. For many bacterial pathogens, ligands or adhesins for specific host \nreceptors  are  known.  Adhesins  comprise  a  wide  variety  of  surface \nstructures, including single proteins, carbohydrates, glycoproteins, lip-\nids, lipoproteins, and multiprotein filamentous complexes that extend \nseveral micrometers from the bacterial surface, each anchored in the \nouter-surface  cell  envelope.  Most  bacteria  produce  multiple  adhesins \nwith varying specificity, enabling the pathogen to interact with multi-\nple receptors, including those on several distinct cell and tissue types \nencountered  during  the  process  of  infection.  These  interactions  are \noften  partially  redundant,  are  serologically  variable,  and  contribute \nadditively or synergistically with other binding interactions.\n\nCommon classes of adhesins are pili (also known as fimbriae), flag-\nella, and autotransporter proteins (Table 120-2). Pili are hairlike exten-\nsions consisting of a polymer of the major pilin subunit capped with \nminor pilins that provide the adherence function of the structure. Pili \nare classified by type and are produced by many gram-negative bacte-\nria and a smaller number of gram-positive bacteria. To date, efforts to \nprevent infection with pilus-based vaccines have been unsuccessful.\n\nTypes of pili include type I, type P, and type IV. Type I pili frequently \nfunction  at  mucosal  surfaces.  For  example,  they  mediate  the  close \nassociation  of  uropathogenic  Escherichia  coli  (UPEC)  with  bladder \nepithelial  cells  and  the  ability  of  this  pathogen  to  persist,  causing \nrelapsing urinary tract infections. UPEC also produces type P pili and \nafimbrial adhesins. These adhesins bind sugar moieties on host surface \nglycoproteins,  with  varied  specificity  depending  on  the  adhesin.  The \nminor  pilin  lectins  at  the  tip  of  the  pili  generally  bind  D-mannose \nglycans,  albeit  with  strain  specificity.  For  example,  the  type  I  pilus \nadhesin  FimH  of  intestinal  E.  coli  strains  often  preferentially  binds \n\nHPIM21e_Part05_p941-p1796.indd   949\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n\u00a0\n\u00a0\n\u00a0\n\u00a0\n\u00a0\n"
  },
  {
    "page": "950",
    "text": "or distributed around the bacterial surface (peritrichous). Flagella are \nevolutionarily related to type III secretion systems (T3SSs) (see \u201cRep-\nlicative  Niche\u201d  and  \u201cSurvival  in  the  Vacuole,\u201d  below)  and  have  been \nshown in some instances to be responsible for the secretion of bacterial \ntoxins. Flagella may also act as adhesins, binding to mucins of mucosal \nsurfaces in the case of the gastrointestinal pathogens enteropathogenic \nE. coli (EPEC) and enterohemorrhagic E. coli (EHEC).\n\nAutotransporter proteins comprise a subdivision of type V secretion \nsystems  (T5SSs),  which  are  prevalent  among  gram-negative  bacteria \n(Fig.  120-1).  Extended  adhesive  projections  anchored  in  the  bacte-\nrial  outer  membrane,  autotransporter  proteins  are  pivotal  virulence \ndeterminants for several human pathogens, including the filamentous \nhemagglutinin of Bordetella pertussis (the etiologic agent of whooping \ncough)  and  the  IcsA  adhesin  and  intracellular  motility  determinant \nof  the  Shigella  spp.  responsible  for  dysenteric  diseases.  In  addition, \nthe autotransporter proteins intimin and invasin are required for the \nintimate association of adhering and effacing pathogens, such as EPEC, \nand invasion by Yersinia spp., respectively. Besides these main classes of \nadhesins, other bacterial surface proteins are also involved in adhesion, \nincluding the Opa family of membrane proteins of Neisseria spp. and \nthe  gram-positive  cell  wall\u2013anchored  microbial  surface  component \nrecognizing adhesive matrix molecules (MSCRAMMs) of S. aureus and \nvarious enterococci.\n\nReceptors  Carbohydrates (glycans) on the surface of and secreted \nby human cells play major roles in the adherence of bacterial patho-\ngens.  The  surfaces  of  human  cells  are  coated  with  glycoproteins  and \nglycolipids, whereas the extracellular matrix (ECM) scaffold of tissues \nis mainly composed of secreted proteoglycans. Most mucosal surfaces \nare covered with a layer of mucus, which consists primarily of mucins, \na family of proteins that are heavily glycosylated. Among the human \nenzymes  involved  in  glycan  decoration  of  secreted  proteins  is  FUT2, \nfor which ~20% of the human population harbors two nonfunctional \nalleles;  the  importance  of  human  glycans  in  infection  is  highlighted \nby  the  observation  that  individuals  who  lack  functional  FUT2  dis-\nplay increased susceptibility to certain bacterial infections, decreased \nsusceptibility  to  certain  viral  infections,  and  altered  susceptibility  to \nchronic  noninfectious  inflammatory  diseases.  These  data  are  con-\nfounded by a relative increase in secretion of sialylated glycans in indi-\nviduals lacking functional FUT2. Glycans participate in many adhesive \n\n950\n\nTABLE 120-2  Classes of Bacterial Adhesion Proteins and Their Host \nReceptors\nADHESIN\nType I pili\n\nEXAMPLE\nFim protein, uropathogenic \nEscherichia coli\n\nRECEPTOR\nTerminal mannose of \nuroplakin N-glycan in \nurinary epithelial cells\n\nType P pili\nType IV pili\n\nMSCRAMM\n\nOpa\n\nFlagellum\n\nAutotransporter\n\nAutotransporter\n\nCD64, CR3, I domain\u2013\ncontaining integrins\n\u03b2-Neurexin\n\nPap protein, uropathogenic E. coli Galactose disaccharides\nTfp protein, Neisseria \ngonorrhoeae\nSdrC protein, Staphylococcus \naureus\nOpa protein, Neisseria \nmeningitidis\nFliC protein, Pseudomonas \naeruginosa\nInvasin, Yersinia \npseudotuberculosis\nAg85, Mycobacterium \ntuberculosis\n\nAsialo-GM1 ganglioside\n\n\u03b21-Integrins\n\nFibronectin\n\nCEACAMsa\n\naCarcinoembryonic antigen\u2013related cell adhesion molecules.\n\noligomannose,  whereas  the  FimH  of  UPEC  strains  commonly  binds \nmonomannose.  Thus,  the  same  pilus  structures  in  different  bacterial \nstrains  can  dictate  adherence  to  distinct  tissues.  Type  IV  pili  (Tfp) \nare widespread among gram-negative bacteria, and similar structures \nexist  among  gram-positive  bacteria.  These  are  evolutionarily  related \nto  the  type  II  secretion  system  (T2SS)  and,  in  addition  to  mediating \nadherence, allow the uptake of DNA into bacteria and the motility of \nbacteria on surfaces. The Tfp of Neisseria spp. and V. cholerae mediate \naggregation of individual bacteria into microcolonies, which promotes \ncolonization.\n\nFlagella are polymeric helical filaments that propel bacteria through \nliquid environments by rotating about their long axis. Because flagella \nconfer  the  ability  to  swim  toward  a  target  surface,  often  following  a \nchemotactic gradient where chemical sensing influences the direction \nof  motility,  they  are  vital  virulence  factors  of  many  pathogenic  bac-\nteria. Flagella can be localized to one end of the bacterial cell (polar) \n\nEffector proteins\n\nHost\n\nHost\ncytosol\n\nPlasma\nmembrane\n\nBacterium\n\nOuter\nmembrane\n\nPeptidoglycan\n\nInner\nmembrane\n\nBacterial\ncytoplasm\n\nType III\nsecretion system\n\nType IV\nsecretion system\n\nType V\nsecretion system\n\nType VI\nsecretion system\n\nFIGURE 120-1  Major bacterial secretion systems involved in pathogenesis. Schematic of the types III, IV, V, and VI secretion systems (T3SS, T4SS, T5SS, and T6SS, \nrespectively) of gram-negative bacteria. The T3SS, T4SS, and T6SS deliver bacterial effector proteins into host cells, whereas the T5SS participates in adhesion to the \nsurface of cells. The architecture of the extracellular portion of the T4SS and how it translocates effector proteins remain poorly defined. Colored shapes with hooks in the \nhost plasma membrane represent host membrane proteins that, it is thought, may participate in these processes.\n\nHPIM21e_Part05_p941-p1796.indd   950\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "951",
    "text": "951\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n0\n\nM\no\nl\ne\nc\nu\nl\na\nr\n\nM\ne\nc\nh\na\nn\ni\ns\n\nm\n\ns\no\nf\n\nM\ni\nc\nr\no\nb\ni\na\nl\n\nP\na\nt\nh\no\ng\ne\nn\ne\ns\ni\ns\n\ninteractions and serve as receptors for certain bacterial toxins. Certain \nbacteria  enzymatically  alter  host  glycans  in  a  manner  that  enables \nimproved access to the epithelial surface, as in the case of V. cholerae \nand the oral pathogen Tannerella forsythia.\n\nThe  GM1  ganglioside,  a  glycolipid,  is  the  receptor  for  cholera \ntoxin  (CTX),  Pseudomonas  aeruginosa  flagella,  and  the  Clostridium \nperfringens \u03b1 toxin, whereas terminal sialic acid residues on GM1 play \nan important role in tropism of the gastric pathogen H. pylori through \ninteractions  with  the  bacterial  membrane  proteins  BabA  and  SabA. \nHeparan  sulfate  proteoglycans  and  other  glycosaminoglycan-conjugated \nproteins are commonly associated with the basolateral membranes of \nepithelial  layers  and  act  as  ligands  for  the  chlamydial  OmcB  protein \nand  the  E.  coli  cytotoxic  necrotizing  factor  toxin;  these  interactions \npromote initial bacterial adherence in proximity to the plasma mem-\nbrane and lead to engagement of additional surface molecules.\n\nIn  mammals,  the  ECM  and  integrin  proteins  are  ubiquitous.  The \nECM  consists  of  laminin,  vimentin,  and  type  IV  collagen,  which \ninteract via fibronectin with integrin receptors in the plasma membrane. \nBecause of direct interaction with plasma membrane\u2013embedded inte-\ngrins,  ECM  alterations  can  result  in  signal  transduction  that  directly \ninfluences  immune  cell  behavior.  Moreover,  many  bacterial  patho-\ngens  engage  integrins  and  the  functionally  related  carcinoembryonic \nantigen\u2013related  cell  adhesion  molecules  (CEACAMs)  as  host  cell \nreceptors and as triggers for their internalization into human cells (see \n\u201cMechanisms of Microbial Entry into Cells,\u201d below). The receptor for \nadherent-invasive  E.  coli,  a  pathogenic  type  of  E.  coli,  is  CEACAM6, \nwhose  levels  are  increased  on  epithelial  cells  in  inflammatory  bowel \ndisease; adherent-invasive E. coli adhere at increased levels in Crohn\u2019s \ndisease,  and  sites  of  excessive  bacterial  adherence  display  increased \ninflammation.\n\nAmong the many other host cell surface proteins that serve as recep-\ntors for bacterial adherence is the cystic fibrosis transmembrane conduc-\ntance regulator (CFTR), a chloride channel involved in the maintenance \nof adequate hydration of mucosal surfaces. Mutation of the CFTR gene \ngives  rise  to  the  hereditary  disease  cystic  fibrosis.  These  patients  are \nhypersusceptible to respiratory infection because the cilia are unable to \nclear viscous mucus from the bronchial epithelial surface. The remark-\nably high frequency of cystic fibrosis (>2.5%) in Caucasian populations \nhas been attributed to relative resistance to Salmonella enterica serovar \nTyphi infection, since S. Typhi adheres to CFTR via type IV pili. It has \nalso  been  reported  that  epithelial  cells  utilize  CFTR  during  internal-\nization-mediated clearance of the extracellular pathogen P. aeruginosa, \npotentially contributing to the decreased ability of cystic fibrosis patients \nto clear infections by this pathogen.\n\n \u25a0 REPLICATIVE NICHE\nOnce  bacterial  pathogens  find  their  niche  and  associate  with  target \nhost  cells,  they  must  replicate  to  persist.  Some  pathogens  remain \npredominantly  associated  with  the  surface  of  epithelial  cells  (e.g., \nBordetella, Pseudomonas, Vibrio, and Clostridium spp.), whereas others \npredominantly  enter  into  cells  (e.g.,  Salmonella,  Shigella,  Francisella, \nand  Listeria  spp.).  Phagocytic  cells  actively  engulf  pathogens  for \ndestruction; thus extracellular bacteria must avoid or inhibit this pro-\ncess to remain associated with the cell surface. Many pathogens deliver \ncytoskeleton-disrupting proteins that perturb the phagocytic process, \nwhereas  others  create  structures  that  impede  phagocytosis;  examples \ninclude biofilm formation by P. aeruginosa and cell chaining by strepto-\ncocci. In contrast, nonphagocytic cells, such as epithelial cells, may also \ninternalize bacteria in processes that can be triggered by either the host \ncell (as a clearance mechanism) or the pathogen (for tissue invasion). A \nsubpopulation of any infecting pathogen will be phagocytosed by mac-\nrophages,  neutrophils,  and/or  dendritic  cells.  Many  pathogens  have \nevolved mechanisms to survive within phagocytes; for example, during \ncolonization  of  the  nasopharynx,  Streptococcus  pneumoniae  survives \nwithin vacuoles in dendritic cells and alveolar or splenic macrophages.\n\nExtracellular  Pathogens  Extracellular  pathogens  replicate  at \nthe  surface  of  human  cells.  Many  secrete  enzymes  that  may  liberate \nnutrients from extracellular factors and fashion a hospitable niche. For \n\nexample, V. cholerae, an extracellular pathogen of the small intestine, \nsecretes the mucinase vibriolysin that degrades the mucus barrier over-\nlying intestinal epithelial cells, giving the bacterium access to the cell \nsurface. H. pylori, an extracellular pathogen of the stomach, secretes a \nurease that converts urea into carbon dioxide and ammonia, thereby \nbuffering the acidic pH of the stomach. Yersinia spp., which associates \nwith the surface of leukocytes, delivers into these naturally phagocytic \ncells secreted bacterial proteins that inhibit phagocytosis. Once phago-\ncytosed,  V.  cholerae  can  inhibit  further  phagocytosis  of  extracellular \nbacteria.\n\nEPEC and EHEC efface the microvilli of the brush border of intesti-\nnal epithelial cells and adhere tightly to the surface of these cells. These \nbacteria induce formation of actin-rich pedestals at the plasma mem-\nbrane, which may help prevent bacterial internalization by the epithe-\nlial cells. Epithelial polarity is important for tissue function, influencing \nion flux, barrier integrity, and protein sorting. Many pathogens subvert \nepithelial  polarity  to  redistribute  membrane  complexes  in  a  manner \nthat  facilitates  tissue  penetration  and  dissemination  and  avoids  inter-\nnalization  and  subsequent  cell-autonomous  immune  mechanisms  (see \n\u201cAvoidance of Innate Immune Responses,\u201d below). For certain pathogens \nthat  do  not  efface  microvilli  (e.g.,  Listeria  monocytogenes  and  Shigella \nspp.),  the  basolateral  compartment  of  the  plasma  membrane\u2014rather \nthan  the  mucus-covered  microvillus  apical  surface\u2014is  the  preferential \nsite of epithelial cell invasion. Tight junctions mediate intimate associ-\nations  between  neighboring  epithelial  cells,  thereby  maintaining  tissue \nbarrier function; the targeted dissolution of tight junctions by pathogens \nfacilitates bacterial penetration into the tissue.\nBIOFILMS  Some  extracellular  pathogens,  including  S.  aureus  and  P. \naeruginosa,  establish  chronic  infections  through  the  production  of \nextracellular polymeric matrices called biofilms, which encase the bac-\nteria. Biofilms commonly develop where tissue integrity has been com-\npromised, such as in burn wounds. The biofilm matrix is composed of \nextracellular polysaccharides and DNA, to which the bacteria adhere. \nThe  biofilm  mass  protects  the  bacterial  residents  from  phagocytosis \nwhile impairing the diffusion and efficacy of antibodies and adminis-\ntered antibiotics. Therefore, the bacteria avoid elimination and persist; \nmeanwhile,  ongoing  recognition  of  virulence  factors  by  the  immune \nsystem  may  lead  to  massive  inflammation  and  local  tissue  damage, \nexacerbating the infection.\n\nMechanisms of Microbial Entry Into Cells  Essentially all bac-\nterial pathogens that are predominantly intracellular during infection \npossess  mechanisms  for  inducing  internalization  into  human  cells. \nEven pathogens that survive for prolonged periods in normally phago-\ncytic  cells  such  as  macrophages  (Salmonella  spp.,  L.  monocytogenes) \npossess mechanisms for inducing internalization into these cells.\n\nTRIGGER  MECHANISM  Bacterial  systems  that  deliver  proteins  into \nhuman  cells  are  critical  to  many  aspects  of  pathogenesis  (Fig.  120-1). \nOne of these systems is the type III secretion system, a specialized system \nthat delivers proteins from the bacterial cytoplasm directly into the cyto-\nsol of eukaryotic cells. Evolutionarily conserved T3SSs are found in many \ngram-negative bacterial pathogens. In all cases, delivery of the proteins \nthat are injected into the cell (effector proteins) has dramatic effects on \nthe cell; for many human pathogens, these effects include the induction \nof bacterial uptake into the cell and the alteration of other cellular pro-\ncesses in ways that promote infection.\n\nThe T3SS forms an apparatus on the bacterial surface that resembles \na needle and syringe, with a hollow tube down its long axis (Fig. 120-1). \nThe base of the organelle spans the two bacterial membranes, forming \na conduit from the bacterial cytoplasm, where proteins are synthesized. \nAnchored to the organelle base and protruding from the bacterial surface \nis a long needle-like structure. Upon contact of the needle tip with the \nplasma membrane of the eukaryotic cell, two proteins secreted through \nthe apparatus form a pore in the plasma membrane. The tip of the needle \ndocks onto the extracellular face of the pore, thereby forming a continu-\nous conduit between the bacterial cytoplasm and the host cell cytosol. It \nis through this conduit that bacterial effector proteins are then delivered \ninto the cell.\n\nHPIM21e_Part05_p941-p1796.indd   951\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n"
  },
  {
    "page": "952",
    "text": "952\n\nBacterial\neffector\nproteins\n\nPathogen-\ncontaining\nvacuoles\n\nTrigger\n\nZipper\n\nFIGURE  120-2  Common  mechanisms  of  bacterial  invasion.  The  mechanisms  for  internalization  of  bacteria  into \nnonphagocytic cells are typically classified as trigger or zipper mechanisms. As examples of the trigger mechanism, \nShigella  and  Salmonella  spp.  use  their  T3SSs  to  deliver  into  host  cells  effector  proteins  that  manipulate  the \ncytoskeleton  in  ways  leading  to  the  formation  of  cytoskeleton-supported  membrane  ruffles.  These  membrane \nruffles extend and surround the pathogen, with consequent endocytosis of the bacterium. As examples of the zipper \nmechanism, bacterial membrane proteins of Yersinia and Listeria spp. induce clustering of host receptors. Clustering \nand subsequent intracellular signaling result in the uptake of the bacterium in a tightly opposed vacuole.\n\nand  the  intracellular  survival  of  the  intra-\ncellular pathogens S. enterica serovar Typh-\nimurium,  Chlamydia  spp.,  and  Shigella  spp.  \nS.  enterica  serovar  Typhimurium  uses  dis-\ntinct T3SSs for the invasion and subsequent \nmaintenance  of  the  Salmonella-containing \nvacuole  (SCV),  whereas  Chlamydia  and \nShigella  each  encode  only  one  such  sys-\ntem.  The  effector  proteins  delivered  are \nthe  major  determinants  of  the  lifestyles  of \nthese  pathogens;  Shigella  quickly  escapes \nthe  vacuole  in  a  T3SS-dependent  manner, \nwhereas Chlamydia remains in the vacuolar \ncompartment (the inclusion) and uses effec-\ntor  proteins  to  hijack  cellular  trafficking \nand perturb innate immune responses. The \nSCV  and  the  Chlamydia  inclusion  exhibit \ndistinct  traits:  the  SCV  is  associated  with \nthin membranous tubules that aid nutrient \nacquisition, whereas the inclusion is local-\nized  at  the  microtubule-organizing  center, \nwhich  is  thought  to  facilitate  recruitment \nof vesicles.\n\nFor  those  pathogens  that  use  a  T3SS  to  induce  entry  into  human \ncells  (Salmonella,  Shigella,  and  Chlamydia  spp.),  among  the  first \neffector  proteins  secreted  into  the  cell  are  several  that  activate  poly-\nmerization of cellular actin immediately beneath the point where the \nbacterium is docked on the plasma membrane. Actin polymerization \npushes outwardly against the plasma membrane, generating large ruf-\nfles of the membrane that engulf the bacterium and take it up into a \nmembrane-bound vacuole (Fig. 120-2).  Shigella spp. then escape the \nvacuole  and  reside  in  the  host  cell  cytosol,  whereas  Salmonella  and \nChlamydia spp. predominantly reside within the vacuole.\nZIPPER  MECHANISM  Several  invasive  pathogens  that  do  not  utilize  a \nT3SS  for  internalization  enter  cells  using  a  zipper-like  mechanism.  In \nthese instances, bacterial surface molecules engage with and stimulate the \nclustering of host cell receptors, which form close associations with the \nbacterium in a processive fashion until the plasma membrane is tightly \nopposed to and surrounding the bacterium (Fig. 120-2). For example, the \nYersinia pseudotuberculosis invasion molecules YadA and invasin bind to \n\u03b21-integrins; this binding stimulates phosphoinositide 3-kinase and pro-\ntein kinase B (Akt) signaling pathways, leading to zipper-mediated entry \nof  Y.  pseudotuberculosis  into  otherwise  nonphagocytic  cells.  In  another \nexample, L. monocytogenes internalins InlA and InlB engage E-cadherin \nand the Met receptor, activating \u03b2-catenin signaling and zipper-mediated \nentry of L. monocytogenes into otherwise nonphagocytic cells. Subpopu-\nlations  of  bacteria  classically  considered  extracellular  may  also  promote \ntheir own uptake in a similar fashion, with the lectin LecA of P. aeruginosa \nand fibronectin-binding protein of S. aureus performing similar functions.\n\nSURVIVAL IN THE VACUOLE\nAfter  uptake  into  cells,  the  majority  of  invasive  bacterial  pathogens \nremain  in  a  vacuole.  Uptake  vacuoles  normally  enter  the  endosomal \npathway,  acquiring  host  proteins  that  promote  vesicle  maturation, \nacidification,  and  degradation  of  contents.  Many  bacterial  pathogens \ncontained in these endosomes subvert this intracellular trafficking in \nways that prevent endosomal maturation and block lysosomal degrada-\ntion of the pathogen. A distinct subset of bacteria actively damage vac-\nuolar membrane integrity, disrupting the vacuole and thereby escaping \ninto the cytosol, where they replicate. For bacteria that remain in the \nvacuole,  various  bacterial  effector  proteins  are  delivered  by  secretion \nsystems either into the vacuolar space or across the vacuolar membrane \ninto  the  cytosol  (Fig.  120-1),  where  these  proteins  manipulate  host \nprocesses to the benefit of the pathogen.\n \u25a0 TYPE III SECRETION SYSTEMS\nT3SSs (Fig. 120-1) are versatile virulence systems, helping bacteria such \nas P.  aeruginosa remain extracellular, but promoting both the uptake \n\n \u25a0 TYPE IV SECRETION SYSTEMS\nFor other pathogens, type IV secretion systems (T4SSs, Figure 120-1)\u2014\nconceptually  similar,  evolutionarily  distinct  effector  protein  delivery \nsystems\u2014are  key  to  the  ability  to  survive  in  cellular  phagocytic  vac-\nuoles.  These  T4SSs  are  similar  to  the  T3SSs  described  above  in  that \nthey form a multiprotein apparatus that contains a continuous channel \nbetween the bacterial cytoplasm and the human cell cytosol. However, \nT4SSs  are  functionally  more  diverse  than  T3SSs,  in  that  (1)  they  are \npresent among both gram-negative and gram-positive bacteria as well \nas some archaea, (2) a large subset of T4SSs transport DNA in a pro-\ncess called conjugation, and (3) some T4SSs deliver proteins, typically \ntoxins, into bacterial cells rather than eukaryotic cells. Not surprisingly, \nT4SSs also display great structural diversity.\n\nFor  the  human  pathogens  Legionella  pneumophila  (the  etiologic \nagent  of  Legionnaires\u2019  disease)  and  Coxiella  burnetii  (the  etiologic \nagent of Q fever), bacterial effector proteins delivered across the vacu-\nolar membrane into the cell cytosol by a T4SS alter maturation of the \nvacuole  in  a  manner  that  makes  it  hospitable  for  bacterial  survival; \nthe resulting vacuole is known as a Legionella-containing or Coxiella- \ncontaining vacuole (LCV or CCV, respectively). The delivered effector \nproteins block fusion of lysosomes with the vacuole, thereby prevent-\ning  bacterial  degradation  by  lysosomal  enzymes;  manipulate  host \nvesicular  trafficking  pathways;  and  remodel  intracellular  membranes  to \nalter the lipid and protein content of the vacuolar membrane. For exam-\nple, the L. pneumophila phospholipase VipD, a T4SS substrate, reduces \nthe  levels  of  phosphatidylinositol  3-phosphate  on  the  LCV,  thereby \npreventing  recruitment  of  the  host  protein  Rab5  GTPase,  which  is \ninvolved in endosomal maturation. The mature LCV membrane con-\ntains  many  features  that  resemble  cellular  endoplasmic  reticulum.  In \nthe case of C. burnetii, the vacuole displays more lysosome-like char-\nacteristics, including a relatively low pH, which induces the activation \nof  T4SS  effector  delivery.  The  delivered  bacterial  effectors  stimulate \nefficient  vesicle  recruitment  from  endosomal  and  autophagosomal  \n(see \u201cAutophagy,\u201d below) networks, massively increasing the membrane \nof  the  CCV  until  it  occupies  the  majority  of  the  cytosol.  For  both  \nL.  pneumophila  and  C.  burnetii,  the  mature  bacterium-containing \nvacuoles are hospitable to bacterial replication and long-term survival.\n\n \u25a0 OTHER SECRETION SYSTEMS\nFrancisella tularensis, the etiologic agent of the zoonotic infection tular-\nemia,  is  a  gram-negative  facultative  intracellular  bacterium  that  dis-\nplays a tropism for macrophages. F. tularensis resides in a phagosome, \nthe maturation of which it delays by remodeling the lipid content of the \nvacuolar membrane through the action of its type VI secretion system \n(T6SS; Fig. 120-1). F. tularensis then avoids destruction by lysosomal \n\nHPIM21e_Part05_p941-p1796.indd   952\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "953",
    "text": "953\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n0\n\nM\no\nl\ne\nc\nu\nl\na\nr\n\nM\ne\nc\nh\na\nn\ni\ns\n\nm\n\ns\no\nf\n\nM\ni\nc\nr\no\nb\ni\na\nl\n\nP\na\nt\nh\no\ng\ne\nn\ne\ns\ni\ns\n\nenzymes  by  escaping  into  the  cytosol,  where  the  bacteria  replicate. \nAnother pathogen that escapes into the cytosol of macrophages is the \nrespiratory  pathogen  Mycobacterium  tuberculosis.  Here,  the  ESX-1 \ntype VII secretion system (T7SS) is required for lysis of the phagoso-\nmal  membrane.  For  both  of  these  pathogens,  few  effectors  of  their \ncognizant secretion system have been characterized and the process of \nphagosomal membrane lysis remains poorly understood.\n\nSURVIVAL IN THE CYTOSOL\nAll invasive bacteria initially enter cells into a vacuole (nonphagocytic \ncells)  or  a  phagosome  (phagocytic  cells),  yet,  for  some  pathogens, \nresidence in the vacuole or phagosome is transient. L. monocytogenes, \nShigella  spp.,  spotted  fever  group  Rickettsia  spp.  (the  etiologic  agents \nof  Rocky  Mountain  spotted  fever,  rickettsialpox,  and  other  spotted \nfever  diseases),  and  Burkholderia  pseudomallei  (the  etiologic  agent \nof  melioidosis)  lyse  the  vacuole  rapidly  after  invasion.  Infections \ncaused  by  these  agents  are  characterized  by  their  cytosolic  lifestyles, \nactin-based motility, and cell-to-cell spread. The gram-positive bacte-\nrium L. monocytogenes produces a pore-forming toxin, listeriolysin O, \nand phospholipases that mediate escape into the cytosol, whereas the \nmechanism of vacuolar escape of Shigella spp. is still uncertain except \nfor  a  requirement  for  its  T3SS.  Once  in  the  cytosol,  bacteria  readily \naccess nutrients such as glucose and amino acids but must also evade \ncell-autonomous  immune  mechanisms,  including  autophagy  (see \n\u201cAutophagy,\u201d below). L. monocytogenes, Shigella spp., B. pseudomallei, \nand  spotted  fever  group  Rickettsia  spp.  display  actin-based  motility, \nwhere expression of a cell-surface protein (ActA in L. monocytogenes, \nthe autotransporter IcsA in Shigella spp., the autotransporter BimA in \nB. pseudomallei, and RickA and Sca2 in spotted fever group Rickettsia \nspp.) recruits host factors that polymerize actin. Actin polymerization \noccurs at the end of the bacterium and propels the microbe through \nthe cell cytosol. This subversion of the host cytoskeleton aids bacterial \nsurvival in the cytosol and promotes bacterial dissemination through \nepithelial  tissues.  In  the  case  of  Shigella,  Listeria,  and  Rickettsia,  \nactin-based motility pushes the bacteria into the host cell membranes \nto form protrusions that are subsequently taken up by the neighboring \ncell, resolving into a double-membrane vacuole containing the bacte-\nrium. This vacuole is lysed once more, releasing the bacteria into the \ncytosol, wherein they begin another round of replication. The modes \nof actin-based motility exhibit many similarities despite being utilized \nby  evolutionarily  diverse  bacteria,  indicating  the  importance  of  this \nvirulence  mechanism  for  cytosolic  pathogens.  Burkholderia  spp.,  on \nthe other hand, induce formation of multinucleated giant cells. Here, \nthe bacteria trigger the infected host cell to fuse its membranes with \nadjacent  cells  in  a  T6SS-dependent  manner,  forming  a  contiguous \ncytoplasm.\n\n \u25a0 NUTRIENT ACQUISITION\nRegardless  of  the  site  of  bacterial  infection,  microbes  must  acquire \nnutrients to replicate. For both extracellular and vacuolar pathogens, \na  global  defensive  strategy  used  by  the  host  is  the  maintenance  of \nnutrient-poor  conditions.  Also  impacting  extracellular  bacteria  is \nthe  microbiota,  which  contributes  to  nutritional  immunity  through \ncompetition  for  resources  (Chap.  471).  Bacterial  pathogens  possess \nan  arsenal  of  nutrient  acquisition  mechanisms  to  overcome  these \nchallenges.  For  example,  upon  oxidative  stress,  to  maximize  nutrient \nuptake, several pathogens secrete metal ion-binding proteins through \nT4SSs (Fig. 120-1) and upregulate expression of cell surface receptors. \nMany bacteria manipulate the host to subvert the processes restricting \nnutrient  availability,  with  intracellular  bacteria  commonly  delivering \nfactors  that  inhibit  host  mRNA  translation  and  thus  increase  the \navailable pool of amino acids. Pathogens within vacuoles have evolved \nmethods of redirecting host nutrient transport to these vacuoles.\n\n \u25a0 CELLULAR TRAFFICKING\nBecause  the  vacuolar  environment  is  poor  in  nutrients,  bacteria \nthat  survive  in  vacuoles  have  evolved  intricate  strategies  for  the \nmanipulation  of  host  processes  to  meet  their  metabolic  needs.  Once \ninternalized,  vacuolar  pathogens  modulate  endosomal  trafficking  to \n\nprevent degradation by lysosomal compartments. Whereas cytoskele-\ntal rearrangements are regulated by small GTPases of the Rho family, \nthe transport of vesicles through the cell cytosol is controlled by Rab \nand Arf GTPases. The co-evolution of vacuole-residing bacteria with \ntheir eukaryotic hosts has promoted the acquisition of eukaryotic-like \ndomains  in  many  secreted  bacterial  effector  proteins.  Many  of  these \neffector proteins mimic host cellular trafficking proteins such as gua-\nnine nucleotide exchange factors (GEFs), which regulate GTPase activ-\nities, and soluble N-ethylmaleimide-sensitive factor attachment protein \nreceptors  (SNAREs),  which  control  vesicle  fusion  with  vacuolar  and \nother membranes. These bacterial effectors cause the accumulation of \nhost  protein  and  lipid  markers  on  the  vacuolar  membranes,  thereby \ncausing  the  vacuoles  to  resemble  host  compartments.  Consequently, \nhost  vesicles  are  redirected  from  the  endoplasmic  reticulum,  Golgi \napparatus, and secretory pathways to the pathogen-containing vacuole, \nthus  delivering  nutrients  to  the  vacuole.  Chlamydia  spp.,  which  are \nobligate intracellular bacteria, acquire host-derived lipids in the bacte-\nrial membranes. Dissection of the modes of action of bacterial effec-\ntors  has  revealed  novel  biochemical  modifications  of  host  trafficking \nregulators. For example, the L. pneumophila T4SS secretes two effector \nproteins  that  inactivate  the  cellular  GTPase  Rab1:  AnkX  inactivates \nRab1 by transferring to it the phospholipid phosphatidylcholine, and \nSidM modifies Rab1 by adding an adenosine monophosphate molecule \nto it. This bacterium delivers more than 300 effector proteins through \nits T4SS; some of these proteins (meta-effectors) modulate the activity \nof other effectors rather than host factors. The coordinated activity of \nthe numerous effector proteins provides precise spatiotemporal control \nof host processes.\n\nAVOIDANCE OF INNATE IMMUNE \nRESPONSES\nFor the host to respond to and clear an infection, it must first be capa-\nble of sensing it. The route of infection and properties of the pathogen \ncontribute  to  determining  the  nature  of  the  immune  response.  The \nhuman immune system is generally considered to consist of two arms: \nthe innate immune system, initiated by germline-encoded sensors, and \nthe adaptive immune system, where lymphocytes are clonally selected \nagainst  specific  antigens.  Initial  interactions  of  pathogens  are  with \nspecific  host  cell  types,  influenced  by  bacterial  tropism;  the  result-\ning  release  of  danger  signals  and  cytokines  from  the  innate  immune \nresponse shapes the later adaptive immune response.\n\nEpithelia (e.g., in the intestinal and respiratory tracts) are commonly \nthe  first  tissues  that  a  pathogen  encounters.  The  mucosal  surfaces \nof  these  epithelia  are  covered  in  a  gel-like  mucus  layer  that  contains \nsecretory  immunoglobulins,  antimicrobial  peptides,  and  commensal \nbacteria, which offer the first line of defense that infecting pathogens \nmust overcome (Chap. 471). In the gastrointestinal tract, this barrier \ncan  be  overcome  through  hijacking  of  microfold  cells  (M  cells)  in \nthe  follicle-associated  epithelia.  M  cells  sample  antigens,  including \nintact  pathogenic  bacteria,  from  the  gut  lumen  and  deliver  them  to \nthe underlying gut-associated lymphoid tissue in a process known as \ntranscytosis.  Within  the  gut-associated  lymphoid  tissue,  professional \nphagocytic  immune  cells,  such  as  dendritic  cells,  macrophages,  and \nneutrophils,  engulf  transcytosed  material  for  destruction.  Lysosomal \ndegradation of pathogenic bacteria in macrophages and dendritic cells \ncan then lead to antigen presentation, where the hydrolytic products \nare  processed  for  display  by  the  major  histocompatibility  complexes \n(MHCs). These antigen-presenting cells migrate through the lymphatic \nsystem to lymph nodes, where the adaptive immune response is stimu-\nlated through activation of B-cell and T-cell clones.\n\n \u25a0 COMPLEMENT\nOnce bacteria have crossed epithelial barriers into the lamina propria, \nthey  encounter  components  of  humoral  immunity.  The  complement \nsystem  is  a  complex  of  plasma  and  tissue-resident  proteins  that  can \nundergo activation from their pre-protein or zymogen forms to either \nlabel  pathogens  for  phagocytosis  (opsonization)  or  lyse  them  directly \n(Fig.  120-3).  Complement  activation  commences  with  detection  of \nthe pathogen through the binding of circulating antibodies, which are \n\nHPIM21e_Part05_p941-p1796.indd   953\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n"
  },
  {
    "page": "954",
    "text": "954\n\nC1 complex\n\nClassical pathway\n\nPseudomonas\naeruginosa\n\nAlternative pathway\n\nImmunoglobulins\n\nLectin pathway\n\nC3 convertase\n\nStreptococcus\npyogenes\n\nC5 convertase\n\nC3 cleavage\n\nNeisseria\ngonorrhoeae\n\nOpsonization\n\nFrancisella\ntularensis\n\nMembrane\nattack\ncomplex\n\nBorrelia\nburgdorferi\n\nPhagocytosis\n\nLysis\n\nFIGURE 120-3  Overview of the complement system in bacterial infection. The classical and lectin pathways of complement activation are initiated by the binding of the \nC1 complex to antibodies bound to bacteria or to lectins binding carbohydrate moieties on the bacterial surface, respectively. A cascade of proteolytic cleavage generates \nthe C3 convertase complex on the bacterial surface, which, upon activation, leads to C5 convertase formation. In the alternative pathway, spontaneous C3 cleavage forms \nalternative C3 products that lead to bacterial opsonization and to amplification of C3 convertase activation. The three complement pathways converge at the point of C3 \nconvertase activation. In a manner similar to that seen for bacteria opsonized with circulating immunoglobulin, opsonized bacteria bind to surface receptors of phagocytes, \ntriggering their engulfment for destruction and stimulation of the adaptive immune system. The C5 convertase, generated by conventional C3 convertase activation, recruits \nfurther complement proteins to form the membrane attack complex that directly lyses bacteria. Many bacterial pathogens inhibit specific points in the complement cascade, \na selection of which are shown with blunt-ended arrows; in contrast, others, such as Francisella spp., promote their own opsonization and subsequent phagocytosis, \nenabling them to reach their intracellular replicative niche.\n\nrecognized by the C1 complex (classical pathway), or of lectins (lectin \npathway)  to  surface  carbohydrates.  These  pathways  converge  at  the \ndeposition of the C3 convertase complex on the bacterium. The alternative \npathway is the third route of complement activation, whereby an alter-\nnative  C3  convertase  complex  is  formed  either  spontaneously  or  by \nthe action of plasma factors, promoting amplification of the cascade. \nActivation  of  further  complement  components  results  in  chemokine \nproduction,  bacterial  opsonization,  or  assembly  of  the  membrane \nattack complex pore on the bacterial surface, which lyses the pathogen. \nBacterial virulence factors have been described that inhibit each step \nof the complement pathway, highlighting the importance of this anti-\nmicrobial system to host immunity. The various strategies of bacterial \ninterference include prevention of initial recognition by assembly of a \npolysaccharide  capsule,  modification  of  bacterial  lipopolysaccharide \n(LPS), degradation of complement components, and masking of sur-\nfaces with host proteins. Streptococci employ numerous mechanisms \nfor  complement  evasion  and  are  thus  a  formidable  foe.  The  strepto-\ncoccal proteases ScpA and SpeB degrade many complement proteins, \nwhereas the bacterial surface-exposed M proteins bind host proteins, \nincluding  the  complement  inhibitor  C4BP,  thereby  preventing  C3b \n\ndeposition.  The  thick  peptidoglycan  layer  of  gram-positive  bacteria \nprovides  modest  resistance  to  membrane  attack  complex\u2013mediated \nlysis.  Furthermore,  streptococci  produce  a  hyaluronic  capsule  that \nshields the bacterial surface from complement recognition. In compar-\nison, Borrelia burgdorferi, the etiologic agent of Lyme disease, encodes \na CD59-like protein that inhibits the completion of membrane attack \ncomplex  assembly.  Whereas  many  pathogens  inhibit  opsonization, \nthereby  avoiding  phagocytosis,  other  bacteria,  including  Francisella \nand Yersinia spp., promote this process, leading to uptake into the pha-\ngosomal compartment, wherein they replicate and prevent lysosomal \nfusion.\n\n \u25a0 LYSOSOMES\nLysosomes  are  vesicular  organelles  of  the  endosomal  system  that \ncontain  dozens  of  hydrolytic  and  antimicrobial  enzymes  in  a  low \npH  environment.  One  such  enzyme  is  lysozyme,  which  hydrolyzes \nthe  polymeric  glycan  chains  in  the  bacterial  cell  wall  peptidoglycan, \nresulting in bacterial lysis. In addition, release of peptidoglycan frag-\nments  stimulates  other  components  of  the  innate  immune  system. \nLysozyme  is  highly  effective  against  gram-positive  bacteria;  however, \n\nHPIM21e_Part05_p941-p1796.indd   954\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "955",
    "text": "955\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n0\n\nM\no\nl\ne\nc\nu\nl\na\nr\n\nM\ne\nc\nh\na\nn\ni\ns\n\nm\n\ns\no\nf\n\nM\ni\nc\nr\no\nb\ni\na\nl\n\nP\na\nt\nh\no\ng\ne\nn\ne\ns\ni\ns\n\nin gram-negative bacteria, because the peptidoglycan layer lies between \nthe  outer  and  inner  membranes,  it  is  inaccessible  to  lysozyme. \nGram-positive bacteria can decrease their susceptibility to lysozyme by \npeptidoglycan modification, such as decreased acetylation of the sugars \nin the glycan strand.\n\n \u25a0 ANTIMICROBIAL PEPTIDES\nAntimicrobial  peptides,  such  as  the  positively  charged  defensins  and \ncathelicidins,  are  concentrated  in  degradative  compartments,  where \nthey bind the anionic surface of bacteria and perturb the integrity of \nthe  LPS  of  the  gram-negative  outer  membrane  and  the  wall  teichoic \nacids in the gram-positive cell wall. Many bacterial pathogens modify \nthese surfaces to reduce the net negative charge, thereby decreasing the \nbinding  of  negatively  charged  antimicrobial  peptides.  In  addition,  S. \naureus and the respiratory pathogen Burkholderia cenocepacia secrete \nproteases that cleave antimicrobial peptides; this action facilitates bac-\nterial survival at mucosal surfaces and within phagosomes.\n\n \u25a0 OXIDATIVE BURST\nPathogens are exposed to reactive oxygen species and reactive nitrogen \nspecies produced by cells responding to infection. A well-characterized \nexample is the neutrophil oxidative burst, where the membrane com-\nplex  nicotinamide  adenine  dinucleotide  phosphate  oxidase  (NADPH \noxidase) produces superoxide radicals through the transfer of electrons \nto molecular oxygen. NADPH oxidase activation at the site of infection \ngenerates  reactive  oxygen  species  in  the  phagosome  of  neutrophils \nand/or  in  the  extracellular  space.  Bacterial  detoxification  enzymes, \nincluding superoxide dismutases, peroxiredoxins, and catalases, enable \nbacteria to survive this oxidative burst. Many bacteria can also block \nthe  activity  of  NADPH  oxidase;  several  pathogens  prevent  assembly \nof  the  NADPH  oxidase  complex  by  T3SS  effector  protein\u2013mediated \npost-translational modification of small GTPases. The importance of \nNADPH  oxidase  in  immune  defense  is  highlighted  clinically  by  the \nmarked  susceptibility  to  bacterial  and  fungal  infections  in  patients \nwith chronic granulomatous disease, which results from nonfunctional \nalleles for essential subunits of this complex.\n\n \u25a0 NEUTROPHIL EXTRACELLULAR TRAPS\nThe oxidative burst also stimulates the release of the pro-inflammatory \ntumor  necrosis  factor  \u03b1,  azurophilic  granules,  and  neutrophil  extra-\ncellular  traps  (NETs).  NETs  are  an  extruded  mesh  of  decondensed \nchromatin  that  are  thought  to  ensnare  pathogens,  thereby  restricting \ntheir  spread  and  destroying  them.  Streptococci  encode  deoxyribo-\nnucleases  (DNases)  that  enable  pathogens  to  evade  NETs.  S.  aureus \nnuclease  degrades  NETs;  the  nucleotide  products  of  this  degradation \nare then converted into deoxyadenosine, which functions as a potent \nmacrophage  toxin.  This  process  showcases  how  bacterial  pathogens \nmay utilize antimicrobial responses to their own benefit.\n\n \u25a0 PATTERN RECOGNITION RECEPTORS AND THEIR \nEVASION\nThe first line of detection of pathogens by the innate immune system \nis via recognition of molecular patterns that are indicative of the pres-\nence  of  pathogens  and  that  distinguish  self  from  nonself.  These  are \nthe pathogen- and damage-associated molecular patterns (PAMPs and \nDAMPs,  respectively).  Recognition  of  these  patterns  enables  cells  to \nestablish that an infection is occurring and to determine the nature of \nthe pathogen, as dictated by the specific PAMP detected. Receptors for \nthe molecular patterns include Toll-like receptors (TLRs) and pattern \nrecognition  receptors  (PRRs).  TLRs,  transmembrane  proteins  at  the \ncell surface or in the endosomal compartment, surveil the extracellular \nmilieu,  whereas  other  PRRs  monitor  threats  within  the  cell  cytosol. \nOn  phagocytic  cells,  TLRs  may  employ  scavenger  receptors  as  co- \nstimulatory surface receptors. TLRs recognize conserved molecules that \ninclude  LPS,  peptidoglycan,  flagella,  and  nucleic  acids  (Table  120-3). \nStimulation of these receptors by ligand binding triggers assembly of \nsupramolecular organizing centers (SMOCs), large protein complexes \nthat are signaling platforms for all-or-nothing signal transduction via \n\nTABLE 120-3  Pattern Recognition Receptors of the Innate Immune \nSystem and Their Ligands\nPATTERN RECOGNITION RECEPTOR\nTLR2 (with TLR1 or TLR6)\nTLR4\nTLR5\nTLR9\nNLRP1\nNLRP3\nNLRP6\nNAIPs/NLRC4\nSTING\nNOD1 and NOD2\n\nLIGAND OR MODE OF ACTIVATION\nLipoproteins\nLPS\nFlagellin\nCpG DNA\nEnzymatic cleavage\nIonic flux, mitochondrial damage\nLipoteichoic acid\nFlagellin/T3SS\nCyclic dinucleotides\nPeptidoglycan\n\nubiquitination  and  phosphorylation  of  target  proteins.  Ultimately, \nsignaling  results  in  translocation  of  transcription  factors,  including \nnuclear  factor-\u03baB  (NF-\u03baB),  AP-1,  and  interferon-regulatory  factors \n(IRFs), into the nucleus, where they stimulate expression of cytokines, \nchemokines,  and  other  immunity-related  genes  (Fig. 120-4).  Among \nthe  signaling  pathways  activated  upon  recognition  of  certain  intra-\ncellular  bacterial  pathogens  is  the  cyclic  guanosine  monophosphate\u2013\nadenosine  monophosphate  synthase  (cGAS)  stimulator  of  interferon \ngenes (STING) pathway, which was previously thought to be relevant \nonly to recognition of intracellular viruses. In response to recognition \nof  microbial  nucleic  acid,  cGAS\u2013STING  activates  the  translocation \nof  interferon  regulatory  factor  3  (IRF3)  and  NF-\u03baB  into  the  nucleus, \nwhereupon they activate a set of interferon-regulated pro-inflammatory \ngenes.  Cytokine  release  alerts  bystander  cells  and  recruits  immune \ncells from the circulation, creating a heightened immune state in the \nlocal area. The nature of the stimulus influences the type of immune \nresponse. Cytosolic ligands tend to stimulate stronger pro-inflammatory \nresponses than extracellular ligands, a difference reflecting the higher \nthreat  perceived.  Moreover,  detection  of  cytosolic  peptidoglycan, \nwhich is derived from bacteria, elicits a response distinct from that to \ndsRNA, which is derived from viruses.\n\nPathogens have evolved a plethora of strategies to avoid detection by \nthe innate immune system. A common strategy is to mask PAMPs. \nFor example, LPS on the surface of gram-negative bacteria contains a \nlipid A component that usually harbors six acyl chains. LPS that is hexa- \nacylated is recognized by TLR4 at the cell surface. The etiologic agent \nof plague, Yersinia pestis, produces hexa-acylated LPS in the flea vector, \nbut, upon the organism\u2019s transmission to human hosts, the temperature \nshift to 37\u00b0C induces acyltransferase expression that results in production \ninstead of tetra-acylated LPS, which does not activate the TLR4 receptor. \nTetra-acylated LPS is also synthesized by the etiologic agent of tularemia, \nF. tularensis; the periodontal pathogen Porphyromonas gingivalis; and the \ngastric pathogen H. pylori. The flagellum of H. pylori is also essential for \nvirulence,  enabling  the  bacterium  to  reach  the  gastric  epithelium  after \ningestion. Flagellin, the monomeric protein subunit of the flagellum, \nis  an  extracellular  ligand  for  TLR5,  but  H.  pylori  bypasses  flagellin \nrecognition  since  its  flagellin  epitope  is  divergent  from  the  TLR5 \nrecognition motif. DNA in intracellular membrane compartments of \nthe endosomal system is recognized by TLR9, which binds to CpG-\ncontaining unmethylated DNA. Because CpG motifs are usually meth-\nylated in mammalian DNA, the methylation state provides a method \nof  discriminating  self  from  nonself.  Group  A  streptococci  secrete \nthe  Sda1  DNase,  which  degrades  CpG  DNA,  therefore  reducing  the \nimmunogenicity of lysed bacteria.\n\nInhibition of NF-\u03baB Signaling  Upon binding to bacterial pep-\ntidoglycan, the cytosolic PRRs NOD1 and NOD2 activate NF-\u03baB sig-\nnaling, thereby priming cell immune responses. Polymorphisms in the \nNOD1  and  NOD2  genes  confer  susceptibility  to  inflammatory  bowel \ndisease, an action that highlights the role of cytosolic sensing of bac-\nterial cell wall components in immune homeostasis. The co-evolution \n\nHPIM21e_Part05_p941-p1796.indd   955\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n"
  },
  {
    "page": "956",
    "text": "956\n\nNucleic acids\n\nPeptidoglycan\n\nLPS\n\nFlagellin\n\nExtracellular PAMPs\n\nTLRs\n\nI\u03baB\u03b1 degradation\n\nGBPs\n\nInnate immunity\nmechanisms\n\nDegradation\n\nUbiquitination\n\nA\n\nImmune GTPases\n\nNutrient\ndepletion\n\nPhagophore\nformation\n\nAutophagosome\nformation\n\nNF-\u03baB\n\nNucleus\n\nSMOC\nassembly\n\nK+\n\nVacuole\nrupture\n\nCytosolic\nPAMPs / DAMPs\n\nInflammasome\nactivation\n\nPro-inflammatory\ngene expression\n\nGasdermin D\npore formation\n\nCaspase-1\nactivation\n\nCytokine\nrelease\n\nIL-1\u03b2\n\nIL-1\u03b2\n\nPyroptosis\n\nDegradation\n\nLysosome\n\nB\n\nAutophagy\n\nC\n\nInflammasomes\n\nFIGURE 120-4  Overview of innate immune recognition and response to bacterial pathogens. A. Immune GTPases. Proteins of the GBP family associate with the bacterial \nsurface,  LPS  shed  from  the  bacterial  surface,  or  pathogen-containing  vacuoles,  disrupting  the  integrity  of  the  pathogen  and  leading  to  lysis.  Labeling  of  intracellular \npathogens with immune GTPases may instead stimulate ubiquitination of the bacterium and its destruction by cell-autonomous immune mechanisms such as autophagy. B. \nAutophagy. Recognition of bacterial invasion leads to host-mediated sequestration of the pathogen in an autophagosome and destruction of the pathogen upon fusion of \nthe autophagosome with lysosomes. Bacteria may be directly labeled by ubiquitination, damaged vacuoles are recognized by the binding of galectins to exposed glycans, \nand/or the cellular autophagy may be induced in response to sensing starvation. C. Pattern recognition receptors and innate immune signaling. Conserved PAMPs, DAMPs, \nand danger signals are recognized by all cells, triggering innate immune signaling pathways and pro-inflammatory gene expression. TLRs surveil the cell surface and \nendosomal interior for PAMPs that include LPS, nucleic acids, and flagellin. Cytosolic PRRs, including NLRs and ALRs, recognize similar ligands as well as distinct stimuli, \nincluding dysregulation of ion homeostasis. TLR activation promotes SMOC assembly and NF-\u03baB signaling. Under resting conditions, NF-\u03baB is sequestered in the cytosol \nby I\u03baB\u03b1, but TLR activation leads to I\u03baB\u03b1 phosphorylation, ubiquitination, and degradation, with the consequent release of NF-\u03baB. Resultant translocation of NF-\u03baB into \nthe nucleus leads to pro-inflammatory gene expression and the production and release of cytokines. Stimulation of cytosolic PRRs leads to assembly of SMOCs known \nas  inflammasomes,  resulting  in  activation  of  caspases.  A  well-characterized  example  is  the  activation  of  caspase-1  by  canonical  and  noncanonical  inflammasomes, \nsubsequent activation of the membrane pore protein GSDMD, maturation and release of pro-inflammatory cytokines IL-1\u03b2 and IL-18, and pyroptotic death of the cell.\n\nof bacterial pathogens with the host has led to the acquisition in many \nbacterial  virulence  factors  of  eukaryotic-like  domains  that  enable \nthe  pathogen  to  hijack  host  signal  transduction.  Ubiquitination,  a \npost-translational  modification  of  eukaryotic  proteins  that  regulates \nmany  cellular  processes,  is  a  common  target  among  known  bacterial \nmechanisms of host signaling inhibition. In ubiquitination, the small \nubiquitin protein is covalently linked to a target protein, and additional \nubiquitin  molecules  are  added  on;  the  consequence  is  branched  or \nlinear chains that signal the fate of the target. Ubiquitination-mediated \ntarget fates are degradation by the proteasome (most common), target-\ning  to  particular  vesicular  compartments,  or  signaling.  The  enzymes \nresponsible for labeling targets with ubiquitin are E3 ubiquitin ligases, \nwhich transfer ubiquitin from an E2 ubiquitin\u2013conjugating enzyme to \nthe target protein.\n\nA  plethora  of  bacterial  effector  proteins  specifically  target  com-\nponents  of  the  NF-\u03baB  pathway  in  ways  that  maintain  repression  of \nNF-\u03baB-dependent pro-inflammatory genes. In doing so, these patho-\ngens  counteract  activation  of  innate  immune  responses  triggered  by \ntheir conserved PAMPs. The translocation of NF-\u03baB into the nucleus \nrequires  the  successive  phosphorylation,  ubiquitination,  and  degra-\ndation  of  I\u03baB\u03b1  by  the  I\u03baB  kinase  (IKK)  complex,  which  in  resting \ncells functions to inhibit NF-\u03baB; the dissociation of the IKK complex \nfrom  NF-\u03baB  enables  the  latter  to  translocate  into  the  nucleus,  where \nit  functions  as  a  transcriptional  factor  (Fig.  120-4C).  The  T3SS  of \nShigella spp. delivers into cells several E3 ubiquitin ligases that target \n\ncomponents of the NF-\u03baB pathway for degradation. The intracellular \npathogens Chlamydia trachomatis and B. pseudomallei secrete bacterial \ndeubiquitinases  through  T3SS  and  T2SS,  respectively,  which  cleave \nthe  ubiquitin  moiety  from  I\u03baB\u03b1,  thereby  preventing  its  degradation, \nmaintaining  inhibited  NF-\u03baB  in  the  cytosol,  and  repressing  pro- \ninflammatory gene expression.\n\nOther  enzymatic  activities  of  bacterial  effectors  can  also  inhibit \nNF-\u03baB  activation  by  ubiquitin-dependent  processes.  L.  pneumophila \ndelivers,  via  its  T4SS,  the  MavC  transglutaminase,  which  performs \nnoncanonical ubiquitination of an E2 enzyme, preventing activation of \nthe IKK complex; the Shigella spp. T3SS cryptic kinase OspG binds and \ninhibits cellular E2 ubiquitin\u2013conjugating enzymes involved in NF-\u03baB \nactivation; and NleE and OspZ from EPEC and Shigella spp., respec-\ntively, are cysteine methyltransferases that inhibit IKK activation.\n\nAnother cellular pathway of pro-inflammatory signaling stimulated \nby  PRRs  is  a  phosphorylation  cascade  involving  mitogen-activated \nprotein kinases (MAPKs) that leads to activation of the transcription \nfactor AP-1. The potent Bacillus anthracis exotoxin lethal factor exhib-\nits metalloprotease activity that cleaves the N-terminal region of most \nMAPKs,  thereby  ablating  the  kinase  cascade.  The  homologous  T3SS \neffectors  YopJ,  AvrA,  and  VopA  of  Yersinia  spp.,  S.  enterica  serovar \nTyphimurium, and Vibrio parahaemolyticus, respectively, acetylate key \nserine residues in MAPKs, which blocks their phosphorylation. Finally, \nOspF of Shigella spp. and the ortholog SpvC from S. enterica serovar \nTyphimurium  are  phosphothreonine  lyases  that  irreversibly  cleave \n\nHPIM21e_Part05_p941-p1796.indd   956\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "957",
    "text": "957\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n0\n\nM\no\nl\ne\nc\nu\nl\na\nr\n\nM\ne\nc\nh\na\nn\ni\ns\n\nm\n\ns\no\nf\n\nM\ni\nc\nr\no\nb\ni\na\nl\n\nP\na\nt\nh\no\ng\ne\nn\ne\ns\ni\ns\n\nphosphorylated  threonine  residues  from  MAPKs,  thereby  preventing \ntheir  activities.  S.  enterica  serovar  Typhimurium  also  targets  NF-\u03baB \ndirectly with several T3SS metalloproteases. The broad range of effec-\ntor protein mechanisms and the large number of distinct host targets \nunderscore the need for bacterial pathogens to subvert innate immune \nsignaling in order to establish a successful infection (Fig. 120-4).\n\nImmune GTPases  For many infectious processes, the initial interac-\ntion of the pathogen with an immune cell primes an enhanced subsequent \nimmune  response  by  augmenting  innate  immune  mechanisms\u2014for \nexample,  through  upregulation  of  immunity-related  gene  expression \nand/or  recruitment  of  neutrophils  to  the  site  of  infection.  A  second \nstimulus  then  activates  a  more  robust  innate  immune  response  that \nis directly geared toward clearing the pathogen or, if the threat is too \ngreat,  leads  to  suicide  of  the  infected  cell.  One  set  of  molecules  that \nprimes  cellular  responses  is  the  immune  GTPases,  of  which  the  best \ncharacterized is the guanylate binding protein (GBP) family. Produc-\ntion  of  these  cytosolic  or  membrane-associated  sensors  is  induced \nby  interferon  signaling.  GBPs  target  pathogen-containing  vacuoles, \nbacterial  membranes,  or  released  fragments  of  bacterial  membranes, \nsuch  as  LPS.  Analogous  to  scavenger  receptors,  GBPs  promote  the \nrelease  of  PAMPs  and  DAMPs  from  these  membrane  structures  and \npresent the PAMP or DAMP to PRRs. For example, some GBPs extract \nLPS  molecules  from  the  surface  of  cytosolic  gram-negative  bacterial \npathogens, and this extraction leads to activation of the noncanonical \ninflammasome (see below). Recruitment of GBPs to Shigella spp. pre-\nvents bacterial actin-based motility and promotes labeling of bacteria \nwith ubiquitin, which targets them for degradation by the proteasome. \nIn this case, the shigellae are able to interfere with this cellular response \nby  secreting  an  E3  ubiquitin  ligase  effector  that  targets  the  GBPs  for \ndegradation.\n\nInflammasomes  Whereas  the  extracellular  environment  is  sur-\nveilled  for  signs  of  infection  by  TLRs,  the  cytosol  is  monitored  by \nother PRRs, including members of the NOD- and AIM2-like receptor \n(NLR and ALR, respectively) families, which form SMOCs known as \ninflammasomes.  Upon  binding  to  ligand  (for  example,  cytosolic  wall \nteichoic  acids  and  dsDNA),  these  PRRs  oligomerize,  often  with  the \nASC scaffold protein. Oligomerization is associated with recruitment \nof pro-caspases, zymogens that are activated within the complexes by \nauto-cleavage.  The  cleaved  forms  of  pro-inflammatory  caspases  are \nthemselves  proteases  that  cleave  zymogen  substrates.  For  example, \ncaspase-1 cleaves the substrate gasdermin D (GSDMD), enabling the \nfree amino-terminal fragment of GSDMD to assemble into a pore in \nthe plasma membrane. Caspase-1 also cleaves and activates pro-IL-1\u03b2 \nand pro-IL-18, releasing the mature cytokines IL-1\u03b2 and IL-18, which \nare  secreted  from  the  cell  through  the  GSDMD  pore,  and  enabling \ncytokine signaling (Fig. 120-4). The formation of GSDMD pores also \nleads to lytic death of the cell, a process designated pyroptosis (inflam-\nmatory  cell  death).  Pyroptosis  is  triggered  by  either  of  two  types  of \ninflammasomes, the canonical caspase-1 inflammasome and the non-\ncanonical inflammasome.\n\nThe  PRRs  of  the  best  characterized  canonical  inflammasomes \nare  NLRP1,  NLRP3,  and  NAIP-NLRC4.  The  NLRP3  inflammasome \nappears  to  be  activated  upon  sensing  of  homeostatic  dysregulation, \nincluding  ion  flux  and  mitochondrial  damage.  Thus,  it  may  be  acti-\nvated by pore-forming toxins such as the phenol-soluble modulins of \nS. aureus and \u03b1-hemolysin of UPEC. NAIP-NLRC4 senses the needle \nprotein of the T3SS and the related flagellin protein. Many pathogens \nhave evolved mechanisms to overcome activation of the NAIP-NLRC4 \ninflammasome, including downregulation of flagella expression by L. \npneumophila upon entry into cells as well as activation of pro-survival \nAkt  signaling  by  the  S.  enterica  serovar  Typhimurium  T3SS  effector \nSopB,  which  reduces  nlrc4  expression  during  intracellular  infection. \nIn addition, caspases of the noncanonical inflammasome are directly \ninhibited by the binding of the T3SS effector proteins OspC3 and NleF \nfrom  Shigella  spp.  and  EHEC,  respectively.  Overall,  the  inhibition  of \ninflammasomes  by  bacterial  pathogens  enables  pathogens  to  avoid \nelimination by the innate immune system.\n\nInflammasomes  have  been  described  as  organelles  that  assess  the \nlevel  of  threat  posed  by  pathogens,  since  they  not  only  recognize \nPAMPs and DAMPs but also act as sentinels for dysregulation of host \nprocesses. Both the NOD1 and pyrin inflammasomes sense cytoskele-\ntal perturbations by monitoring modification of small GTPases. Pyrin \nsenses  a  range  of  covalent  modifications  of  Rho  GTPases,  including \nglucosylation  by  C.  difficile  TcdB,  adenylylation  by  C.  botulinum  C3 \ntoxin, and deamidation by B. cenocepacia T6SS effector TecA. Mean-\nwhile, recent work suggests that the NLRP1 inflammasome may act as \nbait for cleavage by microbial proteases or ubiquitin-targeted degrada-\ntion, processes that stimulate its activation and subsequent pyroptosis. \nOnce more, the importance of inflammasome signaling is underscored \nby a plethora of polymorphisms that exist in inflammasome-associated \ngenes  and  that  may  give  rise  to  autoinflammatory  disorders  or  may \nconfer high susceptibility to microbial infection.\n\nInhibition  of  Nonpyroptotic  Cell  Death  An  outcome  of \ninflammasome  activation  in  macrophages,  and  occasionally  in  non-\nphagocytic  cells,  is  pyroptosis.  This  inflammatory  form  of  cell  death \nboth destroys the pathogen residing within the cell and alerts bystander \ncells of the threat. Several other types of regulated (programmed) cell \ndeath occur, of which inflammatory necroptosis and noninflammatory \napoptosis  are  best  characterized.  In  necroptosis,  activation  of  any  of \na  variety  of  innate  immune  signaling  pathways  (interferon-\u03b3,  TLRs) \nleads to kinase-dependent assembly of a plasma membrane pore com-\nplex that induces cell lysis and release of DAMPs into the extracellular \nspace,  events  that  in  turn  induce  an  intense  inflammatory  response. \nNecroptosis has been described in bacterial pneumonia and sepsis.\n\nIn  apoptosis,  cells  condense  and  form  membranous  blebs  without \nexposing  or  releasing  cytosolic  contents  into  the  surrounding  milieu \nand  thus  without  inducing  inflammation  in  adjacent  tissues.  The \napoptotic  blebs  and  dead  cells  are  disposed  of  by  macrophages  in  a \nprocess known as efferocytosis. Apoptosis is not instigated solely upon \ninfection; it plays a key role in development and also occurs normally \nin many epithelial tissues, such as the intestine, where, during normal \ncell turnover, epithelial cells migrate to the tips of intestinal villi, from \nwhich they extrude and detach.\n\nApoptosis can be initiated by intrinsic or extrinsic cellular stimuli, \nresulting  in  activation  of  apoptotic  caspases  that  demolish  cellular \ncontents.  The  intrinsic  pathway  occurs  through  dissolution  of  the \nmitochondrial  membrane  potential,  which  commonly  occurs  during \ninfection  because  of  cellular  stress  or  the  action  of  bacterial  toxins. \nThe extrinsic pathway can be stimulated by interactions of cell-surface \ndeath receptors with pro-inflammatory ligands like TNF-\u03b1.\n\nThe  gastrointestinal  pathogens  S.  enterica  serovar  Typhimurium, \nYersinia enterocolitica, and pathogenic E. coli inhibit signal transduc-\ntion from cell-surface death receptors, thereby blocking cell death. The \ncell\u2019s sensing of inhibition of extrinsic apoptotic signaling leads to stim-\nulation of alternative inflammatory cell death pathways; this scenario \nhighlights  the  importance  of  functional  redundancy  of  critical  host \nimmune responses. These pathogens can block signaling by modifying \narginine residues of death receptor SMOCs with sugar moieties or by \ndirect cleavage of SMOC components (Fig. 120-4).\n\nThe intracellular vacuole-residing Chlamydia spp. activate survival \npathways through phosphoinositide 3-kinase (PI3K) and Wnt/\u03b2-catenin \nsignaling,  while  simultaneously  inhibiting  apoptosis.  T4SS  effectors \nof  vacuole-residing  L.  pneumophila  and  C.  burnetii  inhibit  apoptosis \nthrough  the  sequestration  of  pro-apoptotic  host  factors.  Cyclomod-\nulins,  a  group  of  bacterial  effector  proteins  that  arrest  the  cell  cycle, \nare  delivered  into  cells  by  the  extracellular  attaching  and  effacing \npathogens EPEC and EHEC. The EPEC and EHEC cyclomodulin Cif \ndeamidates  ubiquitin  and  related  proteins,  inactivating  the  host  E3 \nligases involved in cell cycle progression and thereby reducing epithe-\nlial turnover. The recognition by the cell of this disruption of important \nhost cell processes triggers cell death pathways. Thus, co-evolution of \nbacterial pathogens with their hosts results in a tug-of-war.\n\nAutophagy  Once  detection  of  a  pathogen  has  been  established \nand cytokine secretion has been triggered, nonimmune cells are able \n\nHPIM21e_Part05_p941-p1796.indd   957\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n"
  },
  {
    "page": "958",
    "text": "958\n\nto  deploy  cell-autonomous  immunity  mechanisms  to  neutralize  the \nthreat.  In  addition  to  assembly  of  the  NADPH  oxidase  complex  and \nproduction  of  immune  GTPases,  the  responses  include  upregulation \nof autophagy. Autophagy disposes of damaged organelles and recycles \nlong-lived protein complexes through generation of double-membrane \ncompartments  (autophagosomes)  around  the  cargo  and  delivery  of \nthe  engulfed  cargo  to  the  lysosome,  wherein  the  cargo  is  degraded \nto  amino  acids  and  other  constituents.  Autophagy  can  isolate  and \ndispose  of  cytosolic  pathogens.  Recognition  of  the  pathogen  can  be \ndirect or can occur through the sensing of DAMPs. Here, host galec-\ntins, polysaccharide-binding molecules, participate in the recognition \nof  pathogens  and  induce  ubiquitination  of  the  bacterial  surface  and \nrecruitment of the host autophagy protein LC3, which triggers engulf-\nment in autophagosomes.\n\nThe  survival  of  intracellular  bacterial  pathogens  is  dependent  on \ntheir ability to inhibit the autophagy machinery. Common mechanisms \nutilized by bacteria to interfere with autophagy include evasion of ubiq-\nuitination by inhibition of host E3 ligases, blocking of ubiquitination \nsites  through  decoration  of  surface-exposed  virulence  factors  with \nhost proteins, or alteration of host pathways that regulate autophagy. \nThe cytosolic pathogen L. monocytogenes induces actin-based motility \nand cell-to-cell spread through recruitment of the host factor Arp2/3, \nand  Arp2/3  recruitment  helps  mask  Listeria  surface  proteins  from \nubiquitination. An important metabolic trigger of autophagy is intra-\ncellular  depletion  of  amino  acids,  which  occurs  upon  replication  of \nmetabolically  active  intracellular  pathogens.  The  master  regulator  of \ncell metabolism is mammalian target of rapamycin (mTOR); mTOR is \ninactivated by intracellular starvation conditions, and its inactivation \nderepresses  autophagy.  Invasive  pathogens,  including  M.  tuberculosis, \nS.  enterica  serovar  Typhimurium,  and  Shigella  spp.,  have  evolved \nstrategies  to  re-activate  mTOR  signaling  during  infection,  thereby \nevading autophagy. S. enterica serovar Typhimurium also secretes two \neffectors, SseF and SseG, that block the activation of a kinase required \nfor autophagosome generation. Because of the ubiquity of autophagy \nand  other  cell-autonomous  immunity  pathways  in  the  body,  bacte-\nrial pathogens have evolved mechanisms to overcome these key host \ndefenses regardless of cellular tropism.\n\nDysregulated or excessive immune responses are detrimental to the \nhost,  as  in  sepsis  or  severe  COVID-19  infection,  causing  morbidity \nand  sometimes  death.  Too  weak  an  immune  response  may  lead  to \npathogen-induced morbidity and mortality. A cellular mechanism that \npromotes  appropriate  immune  responses  is  innate  immune  training, \nwhich  enables  cells  to  be  primed  at  the  transcriptional  level  through \nepigenetic  modifications.  Here,  epigenetic  alterations  in  chromatin \nstructure cause stable changes in gene expression profiles. Epigenetic \nregulation consists of post-translational modification of histones, with \nalteration  of  their  phosphorylation,  acetylation,  and/or  methylation \nstates\u2014changes  that  influence  chromatin  structure  and  access  of \ntranscription  factors  to  regulatory  elements  in  the  DNA.  Mounting \nevidence  indicates  that  the  human  microbiota  promotes  this  innate \nimmune  training,  modifying  the  methylation  state  of  promoters  of \ngenes involved in metabolism and immunity and allowing the host to \nrespond to infection in a well-tuned manner. Not surprisingly, patho-\ngens  also  regulate  host  responses  at  the  epigenetic  level.  B.  anthracis \nand  L.  pneumophila  deliver  into  cells  histone  methyltransferases  that \ncontrol  inflammation  and  ribosome  activity.  L.  monocytogenes  liste-\nriolysin O, the P. aeruginosa T3SS pore, and other plasma membrane \npore-forming complexes exert epigenetic modulation through altera-\ntions  in  ion  homeostasis,  thereby  manipulating  the  gene  expression \nprofiles of infected host cells.\n\n \u25a0 INHIBITION OF ADAPTIVE IMMUNE RESPONSES\nEarly  interactions  of  pathogens  with  host  cells  trigger  the  innate \nimmune  response,  releasing  cytokines  that  recruit  additional  anti-\ngen-presenting  immune  cells  to  the  site  of  infection.  To  establish \nchronic  infection,  bacterial  pathogens  need  not  only  to  suppress  the \ninnate  immune  system  but  also  to  avoid  elimination  by  the  adaptive \nimmune  response.  The  adaptive  immune  system  comprises  clonally \nexpanded lineages of B and T lymphocytes that have been activated by \n\nantigen-presenting cells, the best characterized of which are dendritic \ncells  and  macrophages.  B  cells  are  required  for  humoral  immunity; \ngenetic diseases that affect B-cell function often manifest as the inabil-\nity to produce adequate antibody titers to clear extracellular bacterial \ninfections. T cells generally mediate cell-mediated immunity, helping \nthe  host  clear  infected  cells.  The  activation  of  T  cells  occurs  by  the \npresentation  of  processed  antigen  on  MHC  molecules  of  antigen- \npresenting cells. The activation of T cells is controlled by the specificity \nof MHC molecules and of their receptors. The large variety of receptors \npossessed  by  T  cells  enables  collective  recognition  of  a  wide  variety \nof  antigens.  Mutations  that  ablate  the  development  of  the  adaptive \nimmune system result in severe combined immunodeficiency, render-\ning the individual extremely vulnerable to infection.\n\nM.  tuberculosis  is  notorious  for  its  ability  to  establish  latent  infec-\ntions, avoiding elimination by the immune system for decades. Once \nphagocytosed, the pathogen induces secretion of immunosuppressive \ncytokines,  including  IL-6,  IL-10,  and  transforming  growth  factor  \u03b2. \nThe  result  is  inhibition  of  interferon-\u03b3-dependent  gene  expression, \nwith downregulation of MHC class II and other immune-stimulatory \nmolecules,  which  inhibits  induction  of  helper  CD4+  T  lymphocytes. \nSimilarly, B. pertussis induces dendritic cells to produce IL-10, which \nskews  T-cell  maturation  into  regulatory  T  cells,  dampening  the \nimmune response. The S. enterica serovar Typhimurium T3SS effector \nprotein  SteD  depletes  MHC  class  II  molecules  from  the  surface  of \ndendritic cells by the activation of the E3 ubiquitin ligase MARCH8. \nMARCH8 ubiquitinates MHC class II, interfering with its trafficking to \nthe cell surface and thereby decreasing interaction with and subsequent \nactivation of T lymphocytes.\n\nIn addition to preventing antigen-presenting cells from stimulating \nlymphocytes, pathogens may also directly alter B-cell and T-cell activ-\nity. The Yersinia T3SS effector YopH, a protein tyrosine phosphatase, \ndephosphorylates  B-cell  and  T-cell  receptors  and  thus  prevents  both \nsignal  transduction  upon  stimulation  by  antigen-presenting  cells \nand  lymphocyte  activation.  Staphylococcal  toxic  shock  syndrome \nis  induced  by  the  production  of  superantigens  by  S.  aureus.  These \nextremely inflammatory exotoxins are potent activators of T cells, stim-\nulating exuberant and at times fatal cytokine production. Rather than \nbinding  the  specificity-determining  variable  regions  of  MHC  mole-\ncules and the T-cell receptor, superantigens bind invariable regions and \nare  therefore  able  to  nonspecifically  activate  vast  numbers  of  T  cells, \nwith a consequent cytokine storm.\n\n \u25a0 BACTERIAL CYTOTOXINS\nMany bacterial toxins are cytotoxins\u2014toxins that trigger host cell death. \nThe best-characterized group of bacterial cytotoxins are the AB toxins, \nwhich  are  defined  by  the  presence  of  an  active  (A)  subunit,  which \nmediates the enzymatic activity of the toxin, and a binding (B) subunit, \nwhich mediates binding to the cellular receptor. This family of toxins \nincludes Shiga toxins of Shigella spp. and pathogenic E. coli, \u03b9 toxin of C. \nperfringens, diphtheria toxin of Corynebacterium diphtheriae, pertussis \ntoxin of B. pertussis, and CTX of V. cholerae. In general, upon binding \nto cell surface receptors, the toxin is taken up into endosomes, where-\nupon  the  A-subunit  is  translocated  across  the  endosome  membrane \nand released into the cytosol, where its toxic enzymatic activity is stim-\nulated. CTX and pertussis toxin exhibit ADP-ribosyl transferase activ-\nity,  targeting  G-protein  regulators  of  adenylate  cyclases  in  a  manner \nthat increases cellular cyclic AMP concentrations and thereby perturb-\ning ion homeostasis and apoptosis. Increased intracellular cyclic AMP \ninduces  chloride  secretion  via  CFTR  and  inhibits  sodium  chloride \nabsorption;  the  results  are  massive  fluid  secretion  into  the  lumen  of \nthe small intestine and the diarrheal symptoms of cholera. Many other \nbacterial toxins that are not AB family toxins, including C. perfringens \n\u03b9 toxin and P. aeruginosa exotoxin A, display ADP-ribosyl transferase \nactivity that targets elongation factor-2, thereby inhibiting host trans-\nlation. In contrast, the S. enterica serovar Typhimurium protein SpvB \nand  C.  difficile  binary  toxins  target  actin,  inhibiting  normal  cellular \ncytoskeletal rearrangements. Despite the range of enzymatic activities \ndisplayed  by  the  bacterial  toxins  described  above,  the  effects  on  the \nhost  cell  generally  fall  into  the  few  broad  categories  of  cytoskeletal \n\nHPIM21e_Part05_p941-p1796.indd   958\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "959",
    "text": "959\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n0\n\nM\no\nl\ne\nc\nu\nl\na\nr\n\nM\ne\nc\nh\na\nn\ni\ns\n\nm\n\ns\no\nf\n\nM\ni\nc\nr\no\nb\ni\na\nl\n\nP\na\nt\nh\no\ng\ne\nn\ne\ns\ni\ns\n\nmanipulation,  inhibition  of  innate  immune  response  signaling,  and \nhijacking of cellular trafficking.\n\nPore-forming toxins are prevalent virulence factors of extracellular \npathogens. They include the cholesterol-dependent cytolysins inter-\nmedilysin of S. intermedius, \u03b8 toxin of C. perfringens, anthrolysin O of \nB. anthracis, and listeriolysin O of L. monocytogenes. S. aureus produces \nseveral  cytolysins,  including  the  Panton-Valentine  leucocidin;  \u03b1-,  \u03b2-, \n\u03b3-,  and  \u03b4-hemolysins;  and  phenol-soluble  modulins.  Following  their \nrelease by bacteria, leukocidins and \u03b1- and \u03b3-hemolysins form oligo-\nmeric complexes in host plasma membranes, with consequent lysis of \nthe host cell. These pore-forming toxins have different cell-type spec-\nificities that are thought to be driven by binding of pore components \nwith specific cell surface receptors. Phenol-soluble modulins are small \npeptides that, as a result of their amphipathic nature, appear to directly \ninsert  into  cell  membranes,  such  that  no  receptor  is  thought  to  be \nnecessary. Lytic toxins protect S. aureus from phagocytosis, yet recent \nwork has uncovered a host defense mechanism that counters cytolysins \nthrough the release of vesicles (termed exosomes), which act as decoys \nby removing cytolysins from the local environment.\n\n \u25a0 TISSUE DAMAGE AND PATHOGEN \nDISSEMINATION\nMuch of the pathology associated with bacterial infection results from \npro-inflammatory  immune  responses.  Infected  cells  may  continually \nsignal in ways that alert the immune system, even though, as described \nabove,  many  bacteria  avoid  elimination  by  cell-autonomous  and \nimmune  cell\u2013mediated  mechanisms.  In  the  intestine,  tight  junctions \nmediate  intimate  associations  between  epithelial  cells  that  maintain \ntissue  barrier  function,  linking  the  cytoskeletal  networks  of  adjacent \ncells through intimate association of protein complexes across the cell \nmembranes.  Many  intestinal  pathogens  perturb  the  integrity  of  the \ngut  epithelium  either  by  manipulating  cell  polarity  or  by  disrupting \nintercellular  junctions.  C.  perfringens,  V.  cholerae,  pathogenic  E.  coli, \nShigella  spp.,  and  S.  enterica  serovar  Typhi  all  produce  toxins  in  the \ngastrointestinal  tract  that  disrupt  tight  junctions,  consequently  dis-\nrupting the barrier function of the tissue and facilitating access of the \npathogen to deeper tissue. The RtxA multifunctional repeats-in-toxin \n(MARTX) toxin of V. cholerae causes cell rounding and barrier failure \nthrough  actin-crosslinking  activity,  yet  avoids  eliciting  substantial \nimmune responses by simultaneously inactivating phospholipases and \nRho GTPases. The serine protease autotransporters (SPATEs) of Shig-\nella spp. and some pathogenic E. coli are typically secreted into the gut \nlumen or mucus layer, whereupon they cleave components of epithelial \njunctions and mucins in ways that facilitate tissue penetration. Tissue \ndamage  permits  access  to  the  underlying  mucosal  layers,  the  lym-\nphatics, and the bloodstream; for some pathogens, this access enables \nseeding of other organs.\n\nTransmission  to  New  Hosts  The  host  represents  a  replicative \nniche  for  bacterial  pathogens,  in  which  they  multiply  and  are  trans-\nmitted  to  new  hosts.  The  mode  of  transmission  is  typically  aligned \nwith  the  mode  of  entry.  For  example,  for  respiratory  pathogens, \ncoughing induced by tissue damage in the lung aerosolizes the patho-\ngen, enabling inhalation by and colonization of a new host. Similarly, \ngastrointestinal  pathogens  elicit  diarrhea  and  are  transmitted  via  the \ndirect  fecal\u2013oral  route  or  via  contamination  of  crops  or  food  with \nwaste from an infected individual. The understanding of the spread of \ninfectious diseases permits the institution of basic hygiene procedures \nthat greatly diminish transmission rates\u2014for example, hand washing, \ndecontamination  of  communal  surfaces,  and  adoption  of  social  dis-\ntancing measures.\n\nBacteriophages  and  Pathogen  Lifestyle  The  reservoir  of  V. \ncholerae  is  aquatic  environments.  Disease  is  acquired  through  the \ningestion of contaminated seawater or seafood. In regions where chol-\nera is endemic, the disease displays seasonal peaks. This seasonality is \nassociated with blooms of bacteria-targeting viruses (bacteriophages), \nwhich  infect  V.  cholerae  organisms,  replicate,  and  lyse  and  kill  the \nbacterial  host.  Bacterial  lysis  releases  viral  particles  into  the  aquatic \n\nenvironment,  whereupon  they  can  infect  other  V.  cholerae.  Conse-\nquently, the bacteriophages regulate the abundance of pathogens, with \nramifications for the epidemiology of the disease. Whether bacterio-\nphages  contribute  to  cyclical  control  of  other  pathogens  is  currently \nunknown.\n \u25a0 COMPETITION WITH COMMENSAL MICROBES\nIn recent years, it has become clear that the host microbiota constitutes \nan ecosystem that contributes to host defense against infecting patho-\ngens. This community of commensal microorganisms (bacteria, fungi, \nviruses) is essential for the metabolic homeostasis and immune devel-\nopment of the host. Furthermore, the presence of commensal organ-\nisms on external surfaces (e.g., the skin), in the nasopharynx, and in \nthe intestinal tract provides colonization resistance against pathogens. \nThe gut microbiota is among the densest ecosystems known. Conse-\nquently, pathogens are greatly outnumbered by flora, and, within each \nniche,  pathogens  must  compete  with  commensals  for  nutrients.  The \nimportance of the microbiota is underscored by C. difficile infection, \nwherein antibiotic treatment, which eradicates components of the gut \nmicrobiota,  enables  C.  difficile  spores  to  germinate,  with  consequent \ntoxin production and disease. It is notable that an effective treatment \nfor  C.  difficile  colitis  is  repopulation  of  the  microbiota  through  fecal \ntransplantation.\n\nMany commensal organisms possess antibacterial systems, includ-\ning  bacteriocins  and  T6SSs.  One  activity  of  T6SSs  is  the  delivery  of \ntoxins directly into neighboring bacteria, resulting in lysis or growth \ninhibition  of  the  target.  T6SSs  are  present  in  both  pathogenic  and \ncommensal  bacteria,  and  T6SS-mediated  killing  contributes  both  to \nthe establishment of a stable gut microbial community and to coloni-\nzation by invading pathogens, including Shigella sonnei and V. cholerae. \nIn  cystic  fibrosis  patients,  upregulation  of  a  T6SS  of  P.  aeruginosa \noccurs; this alteration suggests a role for this system in the persistence \nof chronic P. aeruginosa infection in the cystic fibrosis lung, which is a \npolymicrobial environment.\n \u25a0 SUMMARY\nBacterial pathogens display myriad mechanisms of colonization, adhe-\nsion,  invasion,  dissemination,  and  manipulation  of  host  pathways. \nInfectious diseases are the result when pathogens successfully establish \nthemselves within the host, and symptoms are usually the result of the \nensuing fights between the pathogens and the immune system. The sheer \ndiversity of virulence determinants highlights the success of the host in \ncombating infection. Further elucidation of how bacteria cause infection \nwill better our understanding of human biology and pathogens and will \nprovide new opportunities for successful therapeutic intervention.\n \u25a0 FURTHER READING\nCosta TRD et al: Secretion systems in gram-negative bacteria: struc-\n\ntural and mechanistic insights. Nat Rev Microbiol 13:343, 2015.\n\nEvavold  CL,  Kagan  JC:  Inflammasomes:  threat-assessment  organ-\n\nelles of the innate immune system. Immunity 348:682, 2019.\n\nFitzgerald  KA,  Kagan  JC:  Toll-like  receptors  and  the  control  of \n\nimmunity. Cell 180:1044, 2020.\n\nGalluzzi L et al: Molecular mechanisms of cell death: Recommenda-\ntions of the Nomenclature Committee on Cell Death 2018. Cell Death \nDiffer 25:486, 2018.\n\nLamason RL, Welch MD: Actin-based motility and cell-to-cell spread \n\nof bacterial pathogens. Curr Opin Microbiol 35:48, 2017.\n\nRibet D, Cossart P: How bacterial pathogens colonize their hosts and \n\ninvade deeper tissues. Microbes Infect 17:173, 2015.\n\nSantos  JC,  Broz  P:  Sensing  of  invading  pathogens  by  GBPs:  At  the \ncrossroads between cell-autonomous and innate immunity. J Leukoc \nBiol 104:729, 2018.\n\nStones  DH,  Krachler  AM:  Against  the  tide:  the  role  of  bacterial \nadhesion in host colonization. Biochem Soc Trans 44:1571, 2016.\nTsolis RM, B\u00e4umler AJ: Gastrointestinal host\u2013pathogen interaction \nin the age of microbiome research. Curr Opin Microbiol 53:78, 2020.\nWu  YW,  Li  F:  Bacterial  interaction  with  host  autophagy.  Virulence \n\n10:352, 2019.\n\nHPIM21e_Part05_p941-p1796.indd   959\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n"
  },
  {
    "page": "960",
    "text": "960\n\n121 Microbial Genomics and \nInfectious Disease\nRoby P. Bhattacharyya, Yonatan H. Grad, \nDeborah T. Hung\n\nJust as microscopy opened up the worlds of microbiology by providing a \ntool with which to visualize microorganisms, technological advances in \ngenomics now provide microbiologists with powerful new methods to \ncharacterize the genetic map that underlies all microbes with unprece-\ndented  resolution,  thereby  illuminating  their  complex  and  dynamic \ninteractions with each other, the environment, and human health. The \nfield of infectious disease genomics encompasses a vast frontier of active \nresearch that is transforming the clinical practice of infectious diseases. \nWhile genetics has long played a key role in elucidating the process of \ninfection and impacting clinical infectious diseases, the ability to extend \nour  thinking  and  approaches  beyond  the  study  of  single  genes  to  an \nexamination of the sequence, structure, and function of entire genomes \nallows  us  to  identify  new  possibilities  for  research  and  opportunities \nto change clinical practice. From the development of diagnostics with \nunprecedented sensitivity, specificity, and speed to the design of novel \npublic  health  interventions,  technical  and  statistical  genomic  innova-\ntions are reshaping our understanding of the influence of the microbial \nworld on human health and providing us with new tools to diagnose, \ntrack, and combat infection. In this chapter, we explore the application \nof  genomics  methods  to  microbial  pathogens  and  the  infections  they \ncause.  We  discuss  innovations  that  are  driving  the  development  of \ndiagnostic approaches as well as the discovery of new pathogens, pro-\nviding  insight  into  novel  therapeutic  approaches  and  paradigms,  and \nadvancing methods in infectious disease epidemiology and the study of \npathogen evolution that can inform infection control measures, public \nhealth responses to outbreaks, and vaccine development. We draw on \nexamples in current practice and from the recent scientific literature as \nsignposts that point toward ways in which the insights from pathogen \ngenomics may influence infectious diseases in the short and long terms, \nand  we  highlight  their  applications  to  SARS-CoV-2  and  COVID-19 \npandemic. Table 121-1 provides definitions for a selection of important \nterms used in genomics.\n\nMICROBIAL DIAGNOSTICS\nThe basic goals of a clinical microbiology laboratory are to establish the \npresence of a pathogen in a clinical sample, to identify the pathogen, \nand, when possible, to provide other information that can help guide \nclinical management and even affect prognosis, such as antibiotic sus-\nceptibility profiles or the presence of virulence factors. To date, clinical \nmicrobiology laboratories have largely approached these goals pheno-\ntypically by growth-based assays and biochemical testing. Bacteria, for \ninstance, are algorithmically grouped into species by their characteristic \nmicroscopic appearance, nutrient requirements for growth, and ability \nto catalyze certain reactions. Antibiotic susceptibility is determined in \nmost cases by assessing bacterial growth in the presence of antibiotic.\n\nWith  the  sequencing  revolution  paving  the  way  to  easy  access  of \ncomplete  pathogen  genomes  (Fig.  121-1),  we  are  now  able  to  more \nsystematically understand the genetic basis for these observable pheno-\ntypes. Compared with traditional growth-based methods for bacterial \ndiagnostics that dominate the clinical microbiology laboratory, nucleic \nacid\u2013based diagnostics that build on this genomic information prom-\nise improved speed, sensitivity, specificity, and breadth of information. \nBridging  clinical  and  research  laboratories,  adaptations  of  genomic \ntechnologies have begun to deliver on this promise (Table 121-2).\n\n \u25a0 HISTORICAL LIMITATIONS AND PROGRESS \nTHROUGH GENETIC APPROACHES\nThe  molecular  diagnostics  revolution  in  the  clinical  microbiology \nlaboratory is well under way, born of necessity in the effort to identify \n\nTABLE 121-1  Glossary of Selected Terms in Genomics\nTERM\nContig\n\nDEFINITION\nA DNA sequence representing a continuous fragment of a \ngenome, assembled from overlapping sequences; relevant \nfor de novo assembly of sequence data that do not align to \npreviously sequenced genomes\nThe entire set of heritable genetic material within an \norganism\nThe transfer of genes between organisms through \nmechanisms other than by clonal descent, such as through \ntransformation, conjugation, or transduction\nAnalysis of genetic material from multiple species directly \nfrom primary samples without requiring prior culture steps\nA collection of DNA oligonucleotides (\u201coligos\u201d) spatially \narranged on a solid surface and used to detect or quantify \nsequences in a sample of interest that are complementary \n(and therefore bind) to one or more of the arrayed oligos\nAn analytic framework to test statistical associations \nbetween microbial genotypes and phenotypes of interest, \nsuch as antibiotic resistance and virulence\nDNA elements that can move within a genome and can \nbe transferred between genomes through horizontal gene \ntransfer (e.g., plasmids, bacteriophages, and transposons)\nA method for typing organisms based on DNA sequence \nfragments from a prespecified set of genes\nHigh-throughput sequencing using a parallelized \nsequencing process that produces millions of sequences \nconcurrently, far beyond the capacity of prior dye-\nterminator methods\nA biochemical assay that evaluates for the presence of a \nparticular string of nucleic acids through amplification by \none of several methods, including polymerase and ligase \nchain reactions\nA type of NAAT used to amplify a specific region of DNA \nby means of specific oligonucleotide primers and a DNA \npolymerase\nPoint mutations, the number of which in different microbial \nisolates is a measure of their genetic distance from one \nanother\nThe catalog of the full set of messenger RNA (mRNA) \ntranscripts from a cell or organism, which are typically \nmeasured by microarray or by next-generation sequencing \nof complementary DNA (cDNA) via a process called \nRNA-Seq\nA process that determines the full DNA sequence of an \norganism\u2019s genome; has been greatly facilitated by next-\ngeneration sequencing technology\n\nGenome\n\nHorizontal gene \ntransfer\n\nMetagenomics\n\nMicroarray\n\nMicrobial genome-\nwide association \nstudy (GWAS)\nMobile genetic \nelements\n\nMultilocus \nsequence typing\nNext-generation \nsequencing\n\nNucleic acid \namplification test \n(NAAT)\n\nPolymerase chain \nreaction (PCR)\n\nSingle-nucleotide \npolymorphism \n(SNP)\nTranscriptome\n\nWhole genome \nsequencing (WGS)\n\nand  characterize  microbes  that  are  refractory  to  traditional  culture \nmethods. Historically, diagnosis of many so-called unculturable patho-\ngens  has  relied  largely  on  serology  and  antigen  detection.  However, \nthese  methods  provide  only  limited  clinical  information  because  of \ntheir suboptimal sensitivity and specificity, the long delays that dimin-\nish  their  utility  for  real-time  patient  management,  and  the  inability \nto  further  characterize  pathogens  beyond  identifying  past  exposure. \nNewer  tests  to  detect  pathogens  based  on  nucleic  acid  content  have \nalready  offered  improvements  in  the  select  cases  in  which  they  have \nbeen applied.\n\nUnlike direct pathogen detection, serologic diagnosis\u2014measurement \nof  the  host\u2019s  response  to  pathogen  exposure\u2014can  typically  be  made \nonly  in  retrospect,  requiring  both  acute-  and  convalescent-phase \nserum  samples.  For  chronic  infections,  distinguishing  active  from \nlatent infection or identifying repeat exposure from serology alone can \nbe  difficult  or  impossible,  depending  on  the  syndrome.  In  addition, \nserologic  diagnosis  is  variably  sensitive,  depending  on  the  organism \nand  the  patient\u2019s  immune  status.  For  instance,  tuberculosis  is  noto-\nriously  difficult  to  identify  by  serologic  methods;  tuberculin  skin \ntesting using purified protein derivative (PPD) is especially insensitive \n\nHPIM21e_Part05_p941-p1796.indd   960\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "961",
    "text": "961\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n1\n\nM\ni\nc\nr\no\nb\ni\na\nl\n\nG\ne\nn\no\nm\ni\nc\ns\na\nn\nd\nI\nn\nf\ne\nc\nt\ni\no\nu\ns\nD\ni\ns\ne\na\ns\ne\n\ns\nt\nc\ne\no\nr\np\n\nj\n\ne\nm\no\nn\ne\ng\n\nf\no\n\nr\ne\nb\nm\nu\nN\n\n200,000\n\n150,000\n\n100,000\n\n50,000\n\n0\n\n1998\n\n1999\n\n2000\n\n2001\n\n2002\n\n2003\n\n2004\n\n2005\n\n2006\n\n2007\n\n2008\n\n2009\n\n2010\n\n2011\n\n2012\n\n2013\n\n2014\n\n2015\n\nYear\n\nFIGURE 121-1  Bacterial genome sequencing projects submitted to the Genomes Online Database, a \nmanually curated repository for genome and metagenome sequencing data, from 1998 to 2019. (Data \ncompiled  from  https://gold.jgi.doe.gov/statistics,  accessed  June  27,  2020.  See  S  Mukherjee  et  al: \nGenomes OnLine Database (GOLD) v.7: Updates and new features. Nucleic Acids Res 47:D1, 2019.)\n\nsensitivity  in  immunodeficient  hosts.  Neither  PPD \ntesting nor IGRAs can distinguish latent from active \ninfection.  Serologic  Lyme  disease  diagnostics  suffer \nsimilar limitations: in patients from endemic regions, \nthe presence of IgG antibodies to Borrelia burgdorferi \nmay reflect prior exposure rather than active disease, \nwhile  IgM  antibodies  are  imperfectly  sensitive  and \nspecific (50% and 80%, respectively, in early disease). \nThe complicated nature of these tests, particularly in \nview  of  the  nonspecific  symptoms  that  may  accom-\npany  Lyme  disease,  has  had  substantial  implications \nfor public perception of Lyme disease and antibiotic \nmisuse in endemic areas. Similarly, syphilis, a chronic \ninfection  caused  by  Treponema  pallidum,  is  notori-\nously  difficult  to  stage  by  serology  alone,  requiring \nmultiple  different  nontreponemal  and  treponemal \ntests  (e.g.,  rapid  protein  reagin  and  fluorescent  tre-\nponemal antibody, respectively) in conjunction with \nclinical  suspicion.  Complementing  serology,  antigen \ndetection  can  improve  sensitivity  and  specificity  in \nselect  cases,  but  has  been  validated  only  for  a  lim-\nited  set  of  infections.  Typically,  structural  elements \nof  pathogens  are  detected,  including  components  of \nviral envelopes (e.g., hepatitis B surface antigen, HIV \np24 antigen), cell surface markers in certain bacteria \n(e.g., Streptococcus pneumoniae, Legionella pneumophila serotype 1) or \nfungi  (e.g.,  Cryptococcus,  Histoplasma),  and  less  specific  fungal  cell-\nwall components such as galactomannan and \u03b2-glucan (e.g., Aspergillus \nand some dimorphic fungi).\n\n2018\n\n2017\n\n2019\n\n2016\n\nin  active  disease  and  possibly  cross-reactive  with  vaccines  or  other \nmycobacteria.  Even  the  newer  interferon  \u03b3  release  assays  (IGRAs), \nwhich  measure  cytokine  release  from  T  lymphocytes  in  response  to \nMycobacterium  tuberculosis\u2013specific  antigens  in  vitro,  have  limited \n\nTABLE 121-2  Selected Clinical Applications of Infectious Disease Genomics\nAPPLICATION\n\nNOTES/EXAMPLES\n\nTECHNOLOGY\n\nOrganism Identification\n\nViral detection\n\nPCR, RT-PCR\n\nTB detection\nPathogen detection\n\nPCR\nPCR, RT-PCR, NAAT\n\nBacterial detection\n\n16S ribosomal gene \nsequencing\n\nPathogen Discovery\n\nBacterial pathogens\n\nSequencing, \nmetagenomic assembly\n\nViral pathogens\n\nMicroarray, sequencing\n\nAntibiotic Resistance\n\nMRSA detection\nVRE detection\nMDR-TB detection\n\nCarbapenemase \ndetection\nHIV resistance detection\n\nPCR\nPCR\nPCR, NAAT\n\nPCR\n\nTargeted sequencing\n\nEpidemiology\n\nOutbreak and epidemic \ntracking\n\nSequencing\n\nEvolution and spread of \npathogens\n\nSequencing\n\nIdentification of HIV, HBV, HCV, respiratory viruses including SARS-CoV-2 and influenza, and others for \ndiagnosis and response to therapy\nAmplification of the rpoB gene for species-specific identification of Mycobacterium tuberculosis\nMultiplexed identification of dozens of viruses, bacteria, yeasts, and parasites from a variety of clinical \nspecimens\nTargeted amplification and sequencing of regions of the 16S rRNA gene for identification of suspected bacterial \ninfections undiagnosed by conventional methods\n\nUnbiased \u201cshotgun\u201d sequencing of isolated nucleic acid from patient samples to identify associated pathogens; \nproofs-of-concept: new Bradyrhizobium species associated with cord colitis; Escherichia coli O104:H4 from \n2011 diarrheal outbreak in Germany; Leptospira species from one patient\u2019s cerebrospinal fluid; research use \nonly at this time\nHybridization of clinical samples to microarrays from phylogenetically diverse known viruses identified the first \nSARS coronavirus and others. Direct sequencing has identified SARS-CoV-2, West Nile virus, and MERS-CoV, \namong others. Use is primarily in research.\n\nDetection of the mecA gene, the genotypic cause of methicillin resistance in Staphylococcus aureus\nDetection of the vanA or vanB gene, the main genotypic causes of vancomycin resistance in Enterococcus\nDetection of polymorphisms in the rpoB gene from M. tuberculosis, which account for 95% of rifampin \nresistance. Other probes available for inhA and katG genes can detect up to 85% of isoniazid resistance.\nDetection of genes encoding one of several types of enzymes (KPC, NDM, OXA-48, IMP, VIM) that hydrolyze \ncarbapenems, accounting for much but not all carbapenemase resistance in Enterobacteriaceae\nTargeted sequencing of specific genes with known resistance-conferring mutations; now the standard of care \nprior to initial therapy in the United States and Europe\n\nApplication to tracking outbreaks and epidemics on local and international scales, including spread of \ncarbapenemase-producing Klebsiella, S. aureus, M. tuberculosis, E. coli, Vibrio cholerae, Ebola virus, Zika virus, \nand influenza virus\nSequencing collections of pathogens to shed light on pathogen dissemination, virulence factors, and antibiotic \nresistance determinants; innumerable examples, including V. cholerae, influenza virus, Ebola virus, and Zika \nvirus\n\nAbbreviations: HBV, hepatitis B virus; HCV, hepatitis C virus; MDR, multidrug-resistant; MERS, Middle East respiratory syndrome; MRSA, methicillin-resistant S. aureus; \nNAAT, nucleic acid amplification test; PCR, polymerase chain reaction; RT, reverse transcriptase; SARS, severe acute respiratory syndrome; TB, tuberculosis; VRE, \nvancomycin-resistant enterococci.\n\nHPIM21e_Part05_p941-p1796.indd   961\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n \n \n \n"
  },
  {
    "page": "962",
    "text": "962\n\nGiven  the  impracticality  of  culture  and  the  lack  of  sensitivity  or \nsufficient  clinical  information  afforded  by  serologic  and  antigenic \nmethods,  the  push  toward  nucleic  acid\u2013based  diagnostics  originated \nin pursuit of viruses and fastidious bacteria, becoming part of the stan-\ndard of care for select organisms in U.S. hospitals. Such tests, including \npolymerase chain reaction (PCR) and other nucleic acid amplification \ntests  (NAATs),  are  now  widely  used  for  many  viral  infections,  both \nchronic  (e.g.,  HIV  infection,  hepatitis  C)  and  acute  (e.g.,  influenza). \nNAATs provide essential information about both the initial diagnosis \nand  the  response  to  therapy  and  in  some  cases  genotypically  predict \ndrug  resistance.  Indeed,  progression  from  antigen  detection  to  PCR \ntransformed our understanding of the natural course of HIV infection, \nwith profound implications for treatment (Fig. 121-2A). In the early \nyears  of  the  AIDS  pandemic,  p24  antigenemia  was  detected  in  acute \nHIV  infection  but  then  disappeared  for  years  before  emerging  again \nwith  progression  to  AIDS  (Fig.  121-2B).  Without  a  marker  demon-\nstrating viremia, the role of treatment during HIV infection prior to the \ndevelopment of clinical AIDS was uncertain, and assessing treatment \n\nefficacy was challenging. With the emergence of PCR as a progressively \nmore sensitive test (now able to detect as few as 20 copies of virus per \nmilliliter of blood), viremia was recognized as a near-universal feature \nof HIV infection. Given the challenges of phenotypic assays, genotypic \nantiviral resistance testing was also adopted early for HIV and is now \nthe standard of care before the initiation of therapy in developed coun-\ntries. These developments have been transformative in guiding therapy \nin early disease and, together with the development of less toxic ther-\napies, have helped to shape policy that is moving toward ever-earlier \nintroduction of antiretroviral therapy in HIV infection. Reverse tran-\nscriptase PCR (RT-PCR) assays are the core method for detecting the \nnovel SARS-CoV-2 virus in the acute phase, forming a critical compo-\nnent of the clinical and public health response to COVID-19, just as \nthey were on a smaller scale for the related coronaviruses SARS-CoV \nand Middle East respiratory syndrome (MERS)-CoV. Tests for SARS-\nCoV-2 represent the largest implementation of a molecular infectious \ndisease assay to date and play a critical role in both clinical diagnostics \nand public health measures to contain the COVID-19 pandemic.\n\nA\nGENERAL\nMILESTONES\n\nHIV genome\nsequenced\n\nFirst AIDS case\nseries published\n\nNevirapine (NNRTI)\napproved\n\nFirst clinical\ncases worldwide\n\nHIV first\nisolated\n\nAZT (NRTI)\napproved\n\nSaquinavir (PI)\napproved\n\nFirst once-a-day\ncombination ARV\napproved\n\n1980\n\n1985\n\n1990\n\n1995\n\n2000\n\n2005\n\n2010\n\nHIV antibody test\napproved\n\nPhenotypic\nresistance\ntesting available\n\nHIV viral load\ntest approved\n\nHIV genotype\nrecommended\nbefore ARV start\n\nFrontier: clinical\nimpact of rare\nsequence variants\n\nB\nDIAGNOSTIC\nMILESTONES\n\np24 antigen test\napproved\n\nHIV genotypic\nresistance\ntesting approved\n\nAcute HIV\n\nChronic HIV\n\nAIDS\n\nl\n\ne\nv\ne\nl\n\ne\nv\ni\nt\na\nl\ne\nR\n\nVL\n\nAb\n\np24\n\nweeks\n\nyears\n\nTime\n\nFIGURE 121-2  A. Timeline of select milestones in HIV management. Genomic advances are shown in bold type. The approvals and recommendations indicated apply to the \nUnited States. ARV, antiretroviral; AZT, zidovudine; NRTI, nucleoside reverse transcriptase (RT) inhibitor; NNRTI, nonnucleoside RT inhibitor; PI, protease inhibitor. B. Viral \ndynamics in the natural history of HIV infection. Three diagnostic markers are shown: HIV antibody (Ab), p24 antigen (p24), and viral load (VL). Dashed gray line represents \nlimit of detection. (Adapted from data in HH Fiebig et al: Dynamics of HIV viremia and antibody seroconversion in plasma donors: Implications for diagnosis and staging of \nprimary HIV infection. AIDS 17:1871, 2003.)\n\nHPIM21e_Part05_p941-p1796.indd   962\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases \n"
  },
  {
    "page": "963",
    "text": "963\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n1\n\nM\ni\nc\nr\no\nb\ni\na\nl\n\nG\ne\nn\no\nm\ni\nc\ns\na\nn\nd\nI\nn\nf\ne\nc\nt\ni\no\nu\ns\nD\ni\ns\ne\na\ns\ne\n\nAs they are for viral testing, nucleic acid\u2013based tests have become \nthe  diagnostic  tests  of  choice  for  fastidious  bacteria,  including  the \ncommon  sexually  transmitted  bacterial  pathogens  Neisseria  gonor-\nrhoeae and Chlamydia trachomatis as well as the tick-borne Ehrlichia \nchaffeensis  and  Anaplasma  phagocytophilum.  More  recently,  nucleic \nacid  amplification\u2013based  detection  has  offered  improved  sensitivity \nfor diagnosis of the important nosocomial pathogen Clostridium diffi-\ncile, and NAATs have provided clinically relevant information on the \npresence of cytotoxins A and B as well as molecular markers of hyper-\nvirulence, such as the North American pulsotype 1 (NAP1) strain that \nis enriched in severe illness. The importance of genomics in selecting \nloci for diagnostic assays and in monitoring test sensitivity was high-\nlighted by the emergence in Sweden of a newly recognized variant of \nC. trachomatis with a deletion that includes the gene targeted by a set \nof  commercial  NAATs.  By  evading  detection  through  this  deletion \n(and thus avoiding treatment), this strain came to be highly prevalent \nin  some  areas  of  Sweden.  While  nucleic  acid\u2013based  tests  remain  the \ndiagnostic  approach  of  choice  for  fastidious  bacteria,  this  example \nserves as a reminder of the need for careful development and ongoing \nmonitoring of molecular diagnostics.\n\nIn contrast, for typical bacterial pathogens for which culture meth-\nods are well established, growth-based assays followed by biochemical \ntests still dominate in the clinical laboratory. Informed by decades of \nclinical  microbiology,  these  tests  have  served  clinicians  well,  yet  the \nlimitations of growth-based tests\u2014in particular, the delays associated \nwith  waiting  for  growth\u2014have  left  opportunities  for  improvements. \nDriven by this need, mass spectrometry\u2013based assays that offer highly \naccurate organism identification within a few hours of a positive blood \nculture  are  widely  adopted  in  clinical  microbiology  laboratories  in \nwell-resourced  areas.  Looking  ahead,  molecular  diagnostics,  greatly \ninformed  by  the  vast  quantity  of  microbial  genome  sequences  gen-\nerated in recent years, offers a way forward. First, sequencing studies \ncan readily identify key genes (or noncoding nucleic acids) that can be \ndeveloped  into  targets  for  clinical  assays  using  PCR  or  hybridization \nassay  platforms.  Second,  sequencing  itself  may  eventually  become \ncheap and rapid enough to be performed routinely on clinical speci-\nmens, with consequent unbiased detection of pathogens.\n\nOne of the biggest drivers for the implementation of novel molec-\nular  technologies  in  the  diagnosis  of  infectious  diseases  is  the  desire \nfor  more  rapid\u2014or  even  real-time\u2014pathogen  identification,  ideally \nwith antibiotic susceptibility information on those microbes for which \nresistance to the current anti-infective armamentarium is of concern. \nSuch  real-time  tests  have  the  potential  to  transform  infectious  dis-\nease management, impacting antibiotic stewardship in the outpatient \nsetting, mortality risk in the critically ill (i.e., patients in whom early \nadministration of effective antibiotics is the most significant factor in \ndecreasing mortality risk), hospital admission, and length of hospital \nstay;  the  extent  of  this  impact  will  depend  on  the  economic  forces \nthat  will  help  define  the  breadth  of  their  deployment.  On  the  public \nhealth  level,  such  tests  will  likely  play  a  role  in  improving  antibiotic \nstewardship,  thereby  influencing  the  rise  of  antibiotic  resistance  and \nenabling surveillance of outbreaks by local, national, and international \nnetworks. In the United States and the United Kingdom, for example, \npublic  health  agencies  have  shifted  from  pulsed-field  gel  electro-\nphoresis  to  genome  sequencing  to  track  food-borne  pathogens  and \nidentify outbreaks; in addition, these countries are rapidly expanding \nthe  routine  use  of  genomics  in  identifying  and  characterizing  other \npathogens, from mycobacteria (both M. tuberculosis and nontubercu-\nlous mycobacteria) to N. gonorrhoeae. Further, international efforts to \ntrack the spread of viral diseases, including recent work on Ebola and \nZika  outbreaks,  ongoing  work  on  seasonal  influenza,  and  efforts  to \ncontrol the SARS-CoV-2 pandemic offer opportunities for improving \ninterventions, surveillance, and prevention efforts, ranging from more \naccurate selection of the influenza virus strains to include in seasonal \nvaccine  development  to  improved  design  of  trials  to  evaluate  novel \nvaccines and therapies.\n\nTechnological  innovations  are  lowering  several  critical  barriers \nto  the  widespread  adoption  of  genomics  and  other  molecular  meth-\nods.  Specifically,  for  NAAT,  the  need  for  rapid  thermal  cycling  and \n\ncold-chain  storage  for  reagents  has  significantly  impeded  implemen-\ntation  in  resource-limited  settings.  Recent  efforts  aim  to  overcome \nthese challenges by developing isothermal amplification protocols and \nlyophilized reagents that do not require refrigeration or sophisticated \ninstrumentation.  For  clinical  sequencing,  (1)  the  cost  and  speed  of \nsequencing  and  analysis  methods  continue  to  fall  precipitously;  (2) \nautomation  and  miniaturization  of  the  preparation  of  a  sample  for \nsequencing promise to reduce cost and minimize the expertise needed; \nand  (3)  direct  sequencing  technologies  that  eliminate  the  complex \nmolecular  biology  required  to  prepare  clinical  samples  for  sequenc-\ning are improving in accuracy and robustness. Further barriers exist, \nincluding the need for standardized pipelines to process data and pres-\nent  clinicians  with  easily  interpretable  and  readily  actionable  results. \nHowever, as these advances give rise to rapid, accurate diagnostic tests, \nthe ultimate goal is to inform a clinician in real time whether antibi-\notics are indicated and, if so, which will be effective. Real-time diag-\nnostics will allow more efficient deployment of our precious antibiotic \narsenal,  thus  improving  both  societal  and  patient-specific  outcomes \nin much the same way that a rapid, sensitive troponin assay has trans-\nformed bedside management of chest pain.\n\n \u25a0 ORGANISM IDENTIFICATION\nIn order to adapt nucleic acid detection to diagnostic tests and thus to \nidentify pathogens on a wide scale, sequences must be found that are \nconserved enough within a species to identify the diversity of strains \nthat  may  be  encountered  in  various  clinical  settings,  but  divergent \nenough  to  distinguish  one  species  from  another.  Until  recently,  this \nproblem has been solved for bacteria by targeting the element of a bac-\nterial genome that is most highly conserved within a species, the 16S \nribosomal RNA (rRNA) subunit. Among many examples, this method \nhas now been used to confirm Mycobacterium chimaera infections in \nseveral patients after cardiothoracic surgery, leading ultimately to rec-\nognition of a widespread outbreak. At present, 16S PCR amplification \nfrom  tissue  specimens  can  be  performed  by  specialty  laboratories, \nthough its sensitivity and clinical utility to date have remained some-\nwhat  limited,  in  part  because  of  the  scarcity  and  relative  fragility  of \npathogen  nucleic  acid  in  the  sampled  tissue,  which  necessitates  reli-\nable, sensitive nucleic acid amplification. As such barriers are reduced \nthrough technological advances and as the causes of culture-negative \ninfection  are  clarified  (perhaps  in  part  through  sequencing  efforts), \nthese tests may become both more accessible and more helpful.\n\nWith  the  wealth  of  sequencing  data  now  available,  other  regions \nbeyond 16S rRNA can be targeted for bacterial species identification. \nThese other genomic loci can provide additional information about a \nclinical  isolate  that  is  relevant  to  patient  management.  For  instance, \ndetection of the presence, or potentially even the expression, of toxin \ngenes  such  as  C.  difficile  toxins  A  and  B  or  Shiga  toxin  can  provide \nclinicians with additional information that will help distinguish com-\nmensals or colonizing bacteria from pathogens and thus aid in prog-\nnostication and management as well as in diagnosis.\n\nBeyond  bacteria,  one  commonly  used  approach  to  PCR-based \npathogen detection is so-called \u201csyndromic panels\u201d of multiplexed PCR \nto identify common causes of clinical infection syndromes, including \nupper respiratory infection, gastroenteritis, and meningoencephalitis. \nThe most frequently deployed syndromic panel is the respiratory viral \npanel, which typically includes primer sets targeting a combination of \ninfluenza,  parainfluenza,  respiratory  syncytial  virus,  adenovirus,  rhi-\nnovirus, enterovirus, metapneumovirus, and common-cold coronavi-\nruses,  sometimes  in  conjunction  with  unculturable  bacteria  such  as \nMycoplasma, Chlamydophila, and Bordetella species. The goal of such \npanels is to capture common infectious causes of these syndromes in a \nsingle, standardized diagnostic test, ideally streamlining the diagnostic \nevaluation. The ready identification of a plausible etiologic agent may \noffer  diagnostic  clarity  if  judiciously  used  and  carefully  considered \nin  the  clinical  context  of  each  patient.  The  most  dramatic  recent \nchange  in  PCR-based  viral  diagnostics  has  been  the  wide  deploy-\nment of an RT-PCR assay for detection of the novel SARS-CoV-2. As \nnoted elsewhere in this chapter, this assay has played a crucial role in \npatient care, triage, infection control, and epidemiology in managing \n\nHPIM21e_Part05_p941-p1796.indd   963\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n"
  },
  {
    "page": "964",
    "text": "964\n\nthe  COVID-19  pandemic.  One  priority  for  the  coming  months  is  to \nintegrate SARS-CoV-2 detection into syndromic panels for detection \nalongside  other  common  respiratory  pathogens  to  enable  routine, \nstreamlined diagnostics when appropriate.\n\nOne challenge with PCR-based assays is the relative complexity of \nthe  molecular  biology  and  consequent  need  for  advanced  technol-\nogy  for  implementation,  including  instruments  and  reagents.  Several \nrecent  approaches  have  advanced  the  molecular  biology  of  nucleic \nacid detection with the aim of increasing deployability of NAATs for \nuse  in  resource-limited  or  even  field  settings.  These  methods  couple \nnucleic  acid  detection  to  an  enzymatic  readout,  enabling  catalytic \nsignal  amplification.  Several  such  approaches  build  on  the  intrinsic \nsensitivity,  specificity,  and  amplification  of  the  CRISPR  (clustered, \nregularly interspaced short palindromic repeats) effectors Cas12a and \nCas13a as nucleic acid sensors. Distinct from the famous gene editing \nCRISPR  effector  Cas9,  these  robust  and  versatile  enzymes  recognize \nshort  nucleic  acid  targets  with  high  specificity  and  transduce  this \nbinding event into \u201ccollateral cleavage\u201d of nearby nucleic acids that can \nbe engineered to create a signal using fluorescent reporter constructs. \nCrucially, all of this biotechnology can be made to work in conjunction \nwith  isothermal  enzymatic  preamplification  steps  to  achieve  remark-\nable  sensitivity,  all  robustly  enough  to  withstand  lyophilization  on \npaper before being reconstituted in the field. Such assays are still in the \nearly stages of development, but they have shown promise and could \nplay a critical role in global diagnostics and surveillance.\n\nWhile  amplification  tests  such  as  PCR  and  other  NAATs  exem-\nplify  one  approach  to  nucleic  acid  detection,  other  approaches  exist, \nincluding detection by hybridization. Although not currently used in \nthe  clinical  realm,  techniques  for  multiplexed  detection  and  identifi-\ncation of pathogens by hybridization to microarrays or in solution are \nbeing developed for other purposes. Of note, these different detection \ntechniques require different degrees of conservation. Highly sensitive \namplification  methods  require  a  high  degree  of  sequence  identity \nbetween PCR primer pairs and their short, specific target sequences; \neven a single base-pair mismatch (particularly near the 3\u2032 end of the \nprimer) may interfere with detection. In contrast, hybridization-based \ntests  are  more  tolerant  of  mismatch  and  thus  can  be  used  to  detect \nimportant regions that may be less precisely conserved within a spe-\ncies,  thus  potentially  allowing  detection  of  clinical  isolates  from  a \ngiven species with greater diversity between isolates. Such assays take \nadvantage of the predictable binding interactions of nucleic acids and \ndo not require enzymology, broadening the range of conditions under \nwhich such assays are feasible, including directly on primary clinical \nspecimens.  The  applicability  of  hybridization-based  methods  toward \neither DNA or RNA opens up the possibility of expression profiling, \nwhich can uncover phenotypic information from nucleic acid content.\nBoth PCR and hybridization methods target specific, known organ-\nisms. At the other extreme, as sequencing costs decline, metagenomic \nsequencing from patient samples is increasingly feasible. This shotgun \nsequencing approach is unbiased\u2014i.e., is able to detect any microbial \nsequence, however divergent or unexpected. In one recent example, a \nclinical  sample  of  cerebrospinal  fluid  from  an  immunocompromised \npatient  with  signs  and  symptoms  of  chronic  meningitis  was  found \nthrough  metagenomic  sequencing  and  analysis  to  contain  small \namounts of Leptospira DNA. In light of this information, retrospective \nPCR testing confirmed the diagnosis of neuroleptospirosis, which had \nbeen  missed  prior  to  the  sequencing  result.  The  patient  was  treated \nwith  penicillin  G  and  clinically  recovered.  Increasingly,  efforts  are \nunder way to bring whole genome sequencing to other clinical sam-\nples,  including  sputum  and  blood,  in  order  to  more  readily  identify \npathogens. One such assay was recently certified for clinical use in the \nUnited States\u2014a shotgun metagenomic sequencing approach applied \nto cell-free DNA circulating in the bloodstream that aims to identify \npathogens  both  in  blood  and  other  body  sites,  although  its  clinical \nniche  remains  to  be  defined.  This  new  approach  brings  its  own  set \nof  challenges,  however,  including  the  need  to  recognize  pathogenic \nsequences  against  a  background  of  expected  host  and  commensal \nsequences and to distinguish true pathogens from either colonizers or \nlaboratory contaminants. The burgeoning field of microbiome research \n\nis  driving  technology  development  for  sequencing  and  analyzing \ncomplex  microbial  communities.  Lessons  from  this  field  will  inform \ndiagnostic efforts.\n\n \u25a0 PATHOGEN DISCOVERY\nIn addition to clinical diagnostic applications, novel genomic technolo-\ngies, including whole genome sequencing, are being applied to clinical \nresearch specimens with a goal of identifying new pathogens in a vari-\nety of circumstances. The tremendous sensitivity and unbiased nature \nof sequencing is also ideal in searching clinical samples for unknown \nor unsuspected pathogens.\n\nCausal inference in infectious diseases has progressed since the time \nof Koch, whose historical postulates provided a rigorous framework for \nattributing a disease to a microorganism. To modernize Koch\u2019s postu-\nlates,  an  organism,  whether  it  can  be  cultured  or  not,  should  induce \ndisease upon introduction into a healthy host if it is to be implicated \nas a causative pathogen. Current sequencing technologies are ideal for \nadvancing this modern version of Koch\u2019s postulates because they can \nidentify candidate causal pathogens with unprecedented sensitivity and \nin an unbiased way, unencumbered by limitations such as culturability. \nYet, as direct sequencing on primary patient samples greatly expands \nour  ability  to  recognize  associations  between  microbes  and  disease \nstates, critical thinking and experimentation will remain vital in estab-\nlishing causality.\n\nVirus  discovery  in  particular  has  been  greatly  facilitated  by  new \nnucleic  acid  technology.  These  frontiers  were  first  notably  explored \nwith high-density microarrays containing spatially arrayed sequences \nfrom  a  phylogenetically  diverse  collection  of  viruses.  Despite  bias \ntoward those with homology to known viruses, novel viruses in clini-\ncal samples were successfully identified on the basis of their ability to \nhybridize to these prespecified sequences. This methodology famously \ncontributed  to  identification  of  the  coronavirus  causing  SARS.  Once \ndiscovered,  the  SARS  coronavirus  was  rapidly  sequenced:  the  full \ngenome was assembled in April 2003, <6 months after recognition of \nthe first case.\n\nWith the advent of next-generation sequencing, unbiased pathogen \ndiscovery is now addressed through a process known as metagenomic \nassembly  (Fig.  121-3),  largely  supplanting  other  methods.  Sequences \nof random nucleotide fragments can be generated from clinical speci-\nmens with no a priori knowledge of pathogen identity through a pro-\ncess called shotgun sequencing. This collection of sequences can then be \ncomputationally aligned to host (i.e., human) sequences, with aligned \nsequences  removed  and  remaining  sequences  compared  with  other \nknown  genomes  to  detect  the  presence  of  known  microorganisms. \nSequence fragments that remain unaligned suggest the presence of an \nadditional organism that cannot be matched to a known, characterized \ngenome;  these  reads  can  be  assembled  into  contiguous  nucleic  acid \nstretches that can be compared with known sequences to construct the \ngenome of a potentially novel organism. Assembled genomes (or parts \nof genomes) can then be compared with known genomes to infer the \nphylogeny of new organisms and identify related classes or traits. Thus, \nnot only can this process identify unanticipated pathogens, but it can \neven identify undiscovered organisms.\n\nThe  emergence  of  COVID-19  provides  a  dramatic  example  that \nillustrates advances in pathogen discovery technology in the interven-\ning 16 years since SARS-CoV was discovered: the causal coronavirus, \nSARS-CoV-2, was identified through metagenomic sequencing within \nabout 1 month of the first known case and just weeks after the outbreak \nwas first recognized. Sequencing and assembly were completed within \n5  days  of  the  discovery  of  the  new  virus,  and  a  NAAT  was  released \n1  day  later.  Given  the  ensuing  ravages  of  COVID-19  and  the  cost  of \ndelays of even a few weeks in implementing this new diagnostic test in \nsome locations, it is sobering to contemplate the added harm had this \noutbreak occurred even a decade earlier. This timeline illustrates the \nadvancing  power  and  speed  of  new  diagnostic  technologies,  but  also \nunderscores the pressing need for continued progress.\n\nOther early applications of sequencing on clinical samples have cen-\ntered around the discovery of novel viruses, including such emerging \npathogens as West Nile virus and MERS-CoV, as well as viral causes of \n\nHPIM21e_Part05_p941-p1796.indd   964\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "965",
    "text": "965\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n1\n\nM\ni\nc\nr\no\nb\ni\na\nl\n\nG\ne\nn\no\nm\ni\nc\ns\na\nn\nd\nI\nn\nf\ne\nc\nt\ni\no\nu\ns\nD\ni\ns\ne\na\ns\ne\n\nDNA extraction\n\nhost +/\u2013 microbial DNA\n\nhigh-throughput\nsequencing\n\nclinical specimen\n\ntaxonomic\nassignment\nof reads\n\nmixed reads\n\n+\n\n\u201cnovel\u201d\nmicrobe\n\nphylogenetic\ncomparison to\nknown genomes\n\n+\n\nde novo assembly\nof unmapped reads\n\ngenome fragments (\u201ccontigs\u201d)\n\nmicrobe 1\n\nmicrobe 2\n\nunmapped\n\nhost\n\naligned reads\n\nFIGURE 121-3  Workflow of metagenomic assembly for pathogen discovery. DNA is isolated from a specimen of interest (e.g., tissue, body fluid) containing a mixture of host \nDNA and nucleic acids from coexisting microbes, either commensal or pathogenic. All DNA (and RNA, if a reverse transcription step is added) is then sequenced, yielding \na mixture of DNA sequence fragments (\u201creads\u201d) from the organisms present. Except for reads that do not align (\u201cmap\u201d) to any known sequence, these reads are aligned \nto existing reference genomes for the host or any known microbes. The unmapped reads are computationally assembled de novo into the largest contiguous stretches \nof DNA possible (\u201ccontigs\u201d), representing fragments of previously unsequenced genomes. These genome fragments (contigs) are then mapped onto a phylogenetic tree \nbased on their sequence. Some may represent known but as-yet-unsequenced organisms, while others will represent novel species. (Figure prepared with valuable input \nfrom Dr. Ami S. Bhatt, personal communication.)\n\nmyriad other conditions, from tropical hemorrhagic fevers to diarrhea \nin newborns.\n\nAs metagenomic sequencing and assembly techniques become more \nrobust, this technology holds great promise for identifying microorgan-\nisms that are associated with clinical conditions of unknown etiology. \nConventional  methods  already  have  unexpectedly  linked  numerous \nconditions with specific agents of infection\u2014e.g., cervical and orophar-\nyngeal  cancers  with  human  papillomavirus  (HPV),  Kaposi\u2019s  sarcoma \nwith human herpesvirus 8, and certain lymphomas with Epstein-Barr \nvirus. Recently, Zika virus, first described in the 1940s, was found to be \nincreasing in incidence as a cause of febrile syndromes, particularly in \nCentral and South America. A concurrent increase in the incidence of \nmicrocephaly was noted that temporally and geographically matched \nthe Zika epidemics. Zika was suspected to be neurotropic because of \na previously recognized association with Guillain-Barr\u00e9 syndrome, but \nthe  strongest  link  between  Zika  virus  and  microcephaly  came  when \nthe virus itself was detected by both quantitative reverse transcription \nPCR (RT-qPCR) and whole genome sequencing in postmortem fetal \nbrain  tissue  from  microcephalic  infants.  An  argument  for  causality \nwas  built  on  the  foundation  of  epidemiologic  evidence  and  direct \nviral  detection,  both  of  which  were  built  on  nucleic  acid  detection \nand genome sequencing. Sequencing techniques offer unprecedented \nsensitivity and specificity for identifying foreign nucleic acid sequences \nthat  may  suggest  other  such  pathogen-associated  conditions\u2014from \nmalignancies  to  inflammatory  conditions  to  unexplained  fevers  or \nother clinical syndromes\u2014associated with organisms from viruses to \nbacteria to parasites. Caution is needed, though: in the absence of the \nability  to  fulfill  Koch\u2019s  postulates,  sequence-based  identification  of  a \nmicrobe from patient specimens is not, on its own, sufficient to identify \na novel pathogen. The increasing sensitivity of these methods warrants \ngreater rigor and care in defining what is \u201cnoise\u201d and what represents \na pathogen.\n\nAs sequencing-based discovery expands, microbes may be found to \nbe associated with conditions not classically thought of as infectious, \nsuch as the link between maternal Zika virus infection and fetal micro-\ncephaly. Studies of bowel flora in laboratory animals and even humans \nalready suggest correlations between microbe composition and various \naspects  of  metabolic  and  cardiovascular  health.  Improved  methods \nfor  pathogen  detection  will  continue  to  uncover  unexpected  correla-\ntions  between  microbes  and  disease  states,  but  the  mere  presence  of \na microbe does not establish causality. Fortunately, once the relatively \n\nlaborious  and  computationally  intensive  metagenomic  sequencing \nand  assembly  efforts  have  identified  a  pathogen,  further  detection \ncan  more  easily  be  undertaken  with  targeted  methods  such  as  PCR \nor hybridization, which may be more scalable and amenable to in situ \nconfirmation.  This  capacity  should  facilitate  the  additional  careful \ninvestigation that will be required to progress beyond correlation and \nto draw causal inference.\n\n \u25a0 ANTIBIOTIC RESISTANCE\nAt present, antibiotic resistance in bacteria and fungi is conventionally \ndetermined by isolating a single colony from a cultured clinical speci-\nmen and testing its growth in the presence of drug. The requirement \nfor multiple growth steps in these conventional assays has several con-\nsequences.  First,  only  culturable  pathogens  can  be  readily  processed. \nSecond, this process requires considerable infrastructure to support the \nsterile environment needed for culture-based testing of diverse organ-\nisms. Finally, and perhaps most significantly, even the fastest-growing \norganisms require 1\u20132 days of processing for identification and 2\u20133 days \nfor  determination  of  susceptibilities.  Some  slow-growing  organisms \ntake even longer; for instance, weeks must pass before drug-resistant \nM. tuberculosis can be identified by growth phenotype. Given the clin-\nical  imperative  in  serious  illness  to  begin  effective  therapy  early,  this \ninherent delay in susceptibility determination has obvious implications \nfor  empirical  antibiotic  use:  broad-spectrum  antibiotics  often  must \nbe chosen up front in situations where it is later shown that preferred \nnarrower-spectrum  drugs  would  have  been  effective  or  even  that  no \nantibiotics  were  appropriate  (i.e.,  in  viral  infections).  Even  with  this \nstrategy, as resistant organisms become more common, the empirical \nchoice  can  be  incorrect,  often  with  devastating  consequences.  Real-\ntime  identification  of  the  infecting  organism  and  information  on  its \nsusceptibility profile would guide initial therapy and support judicious \nantibiotic use, ideally improving patient outcomes while aiding in the \never-escalating fight against antibiotic resistance by reserving the use of \nbroad-spectrum agents for cases in which they are truly needed.\n\nMolecular  diagnostics  and  sequencing  offer  a  way  to  accelerate \ndetection  of  a  pathogen\u2019s  antibiotic  susceptibility  profile.  If  a  geno-\ntype  that  confers  resistance  can  be  identified,  this  genotype  can  be \ntargeted  for  molecular  detection.  In  infectious  disease,  this  approach \nhas  most  convincingly  come  to  fruition  for  HIV  (Fig.  121-2A).  (In  a \nconceptually parallel application of genomic analysis, molecular detec-\ntion  of  certain  resistance  determinants  in  cancers  informs  selection \n\nHPIM21e_Part05_p941-p1796.indd   965\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n"
  },
  {
    "page": "966",
    "text": "966\n\nof  targeted  chemotherapy.)  Extensive  sequencing  of  HIV  strains  and \ncorrelations drawn between viral genotypes and phenotypic resistance \nhave  delineated  the  majority  of  mutations  in  key  HIV  genes,  such  as \nreverse transcriptase, protease, and integrase, that confer resistance to  \nthe  antiretroviral  agents  that  target  these  proteins.  For  instance,  the \nsingle amino acid substitution K103N in the HIV reverse transcriptase \ngene  predicts  resistance  to  the  first-line  nonnucleoside  reverse  tran-\nscriptase  inhibitor  efavirenz,  and  its  detection  informs  a  clinician  to \nchoose a different agent. The effects of these common mutations on HIV \nsusceptibility to various drugs\u2014as well as on viral fitness\u2014are curated \nin publicly available databases. Thus, genotypes are now routinely used \nto predict drug resistance in HIV, as phenotypic resistance assays are far \nmore cumbersome than targeted sequencing. Indeed, the current rec-\nommendation in the United States is to sequence virus from a patient\u2019s \nblood  before  initiating  antiretroviral  therapy,  which  is  then  tailored \nto  the  predicted  resistance  phenotype.  As  new  targeted  therapies  are \nintroduced, this targeted sequencing\u2013based approach to drug resistance \nwill likely prove important in other viral infections, such as hepatitis C.\nThe  challenge  of  predicting  drug  susceptibility  from  genotype  is \nmore  daunting  for  bacteria  than  for  HIV,  yet  considerable  progress \nhas  been  made  toward  sequencing-based  determination  of  bacterial \nantibiotic  susceptibility.  Bacteria  have  far  more  complex  genomes \nthan  viruses,  with  thousands  of  genes  on  their  chromosomes  (many \nof which can functionally interact in ways that escape a priori predic-\ntion) and the capacity to acquire many more through horizontal gene \ntransfer of plasmids and mobile genetic elements within and between \nspecies. Thus, the task of comprehensively defining all possible genetic \nresistance mechanisms is orders of magnitude more complex in bac-\nteria than in viruses, which typically have far more limited genomes. \nDespite  these  challenges,  considerable  progress  has  been  made  in \nrecent  years.  In  select  cases  where  biological  factors  appear  to  have \nconstrained the genotypic basis for resistance to a small, well-defined \nset of mutations, genotypic assays for antibiotic resistance are already \nbeing introduced into clinical practice. One important example is the \ndetection  of  methicillin-resistant  Staphylococcus  aureus  (MRSA).  S. \naureus is one of the most common and serious bacterial pathogens of \nhumans, particularly in health care settings. Resistance to methicillin, \nthe  most  effective  class  of  antistaphylococcal  antibiotics,  has  become \nvery common, even in community-acquired strains. Vancomycin\u2014the \nalternative drug to methicillin\u2014is effective against MRSA but is mea-\nsurably inferior to methicillin against methicillin-susceptible S. aureus \n(MSSA).  Analysis  of  clinical  MRSA  isolates  has  demonstrated  that \nthe molecular basis for resistance to methicillin in essentially all cases \nstems from the expression of an alternative penicillin-binding protein \n(PBP2A) encoded by the gene mecA, which is found within a transfer-\nable genetic element called mec. This mobile cassette has spread rapidly \nthrough the S. aureus population via horizontal gene transfer and selec-\ntion from widespread antibiotic use. Because methicillin resistance is \nessentially always due to the presence of the mec cassette, MRSA is par-\nticularly amenable to molecular detection. In recent years, a PCR test \nfor the mec cassette, which saves hours to days compared with standard \nculture-based methods, has been approved by the U.S. Food and Drug \nAdministration (FDA). Similar to MRSA, vancomycin-resistant entero-\ncocci (VRE) harbor one of a limited number of van genes found to be \nresponsible  for  resistance  to  this  important  antibiotic,  which  occurs \nthrough  alteration  of  the  mechanism  for  cell  wall  cross-linking  that \nvancomycin inhibits. Detection of one of these genes by PCR indicates \nresistance.  More  recently,  identification  of  carbapenemase-encoding \nplasmids  responsible  for  a  significant  fraction  of  carbapenem  resis-\ntance (though not all instances) has led to multiplexed PCR assays to \ndetect this important resistance element to this crucial antibiotic class. \nFinally, a PCR assay targeting the highly conserved RNA polymerase \ngene serves not only to detect M. tuberculosis directly in sputum sam-\nples but also to detect resistance to rifampin, since the determinants of \nresistance to this RNA polymerase inhibitor map almost exclusively to \na short region of this gene. Since rifampin resistance is epidemiologi-\ncally associated with, though not causal for, multidrug resistance, this \nassay identifies strains at high risk for multidrug resistance, enhancing \nits value.\n\nAlthough  identification  and  rapid  detection  of  monogenic  resis-\ntance  determinants  have  improved,  bacteria  have  tended  to  evolve \nmultiple,  diverse  resistance  mechanisms  to  most  antibiotics;  there-\nfore, these tasks often require probing for and integration of multiple \ngenetic  lesions,  targets,  or  mechanisms.  For  instance,  at  least  five \ndistinct modes of resistance to fluoroquinolones are known: reduced \nimport, increased efflux, target site mutation, drug modification, and \nshielding  of  the  target  sites  by  expression  of  another  protein.  These \nmechanisms are typically present in combination in clinically resistant \nisolates;  thus,  the  problem  of  detecting  genetic  resistance  is  often  a \ncombinatorial  one.  In  another  clinically  important  example,  while \ncarbapenem resistance in Enterobacteriaceae is often explained by the \npresence of carbapenemases, resistance may also develop when other, \nless broad-spectrum \u03b2-lactamases are found in combination with porin \nmutations or efflux pumps. Thus, while multiplexed PCR assays for the \nmost common carbapenemases (e.g., those encoded by the KPC, NDM, \nOXA-48, IMP, and VIM genes) have become a valuable tool for rapid \nidentification of the subset of carbapenem-resistant Enterobacteriaceae \nin  which  resistance  is  caused  by  carbapenemases,  their  sensitivity  is \nlimited  by  their  inability  to  detect  other  mechanisms  of  carbapenem \nresistance.  Additionally,  plasmids  and  transposable  elements,  which \noften are enriched for antibiotic resistance determinants, may be more \ntechnically  and  analytically  challenging  to  sequence,  although  newer \nlong-read  sequencing  technologies  are  beginning  to  address  these \nchallenges. To further complicate genetic prediction, changes in gene \nexpression (which may be detectable through mutations in promoter \nregions or regulatory genes without coding mutations in known resis-\ntance  determinants)  and  even  gene  copy  number  (which  may  occur \nwithout changes in primary sequence) of resistance determinants play \ncritical roles in some cases of genetic resistance. Thus, while predicting \nresistance when determinants are found is rapidly becoming feasible, \nthe  more  clinically  relevant  task  of  predicting  susceptibility  when  no \nknown resistance determinants are found remains more difficult.\n\nTo  build  on  early  successes  with  the  goal  of  advancing  beyond \nbinary  detection  of  monogenic  resistance  determinants,  the  ultimate \nfrontier  for  genetic  prediction  of  bacterial  antibiotic  resistance  lies \nin  more  comprehensive  prediction  of  a  resistance  phenotype  from \nsequence information\u2014a task similar to HIV resistance prediction. Yet \nthere is no comprehensive compendium of genetic elements conferring \nresistance and their pairwise and higher-order interactions with each \nother  and  with  the  genetic  background  of  bacterial  pathogens.  Non-\nviral  genomes  are  much  larger  than  viral  ones,  and  their  abundance \nand diversity are such that thousands of genetic differences often exist \nbetween clinical isolates of the same species, of which perhaps only one \nor  a  few  may  contribute  to  resistance.  In  addition,  new  mechanisms \nmay  emerge  in  the  face  of  antibiotic  deployment  or  with  the  release \nof  new  drugs,  and  genetic  prediction  of  resistance  will  inevitably  lag \nbehind  the  emergence  of  unforeseen  mechanisms.  While  confident \nprediction of bacterial antibiotic resistance from sequencing determi-\nnants  may  therefore  seem  daunting,  the  vast  expansion  of  microbial \nsequencing  capacity  (Fig.  121-1),  combined  with  analytic  methods \nsuch as microbial genome-wide association studies and machine learn-\ning  algorithms,  offers  powerful  analytical  approaches  to  this  \u201cneedle \nin  a  haystack\u201d  problem  and  has  permitted  remarkable  advances  in \nthe  predictive  power  of  sequence  determinants  to  date.  Particularly \nin M. tuberculosis, where horizontal gene transfer is minimal and the \npathogen  is  essentially  restricted  to  human  hosts  so  as  to  facilitate \nmore representative sampling, a remarkably wide array of phenotypic \nresistance  can  be  explained  by  known  genetic  determinants.  Because \nof these biologic advantages, as well as the slow and laborious growth \nprocess that impedes traditional phenotypic assessment, whole genome \nsequencing has proven quite effective at predicting susceptibility pro-\nfiles in this organism, to the point that the United Kingdom now rou-\ntinely performs whole genome sequencing in parallel with phenotypic \nantibiotic susceptibility testing for M. tuberculosis in what some hope \nwill be a precursor to fully whole genome sequencing\u2013based antibiotic \nsusceptibility  testing.  Even  in  more  highly  variable  pathogens,  with \nsequencing  of  sufficient  numbers  of  susceptible  and  resistant  patho-\ngens, sequence-based prediction methods are improving in predictive \n\nHPIM21e_Part05_p941-p1796.indd   966\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "967",
    "text": "accuracy,  at  least  within  the  geographic  region  from  which  the  test \nsamples have been sequenced.\n\nIt  is  important  to  note  that  genotype-based  analytical  methods \nlargely identify correlates, not necessarily surrogates or determinants, \nof resistance. In HIV diagnostics, surrogates (i.e., causal determinants \nof  resistance)  were  found  to  be  more  reliable  predictors  than  mere \ncorrelates in expanding sequencing-based resistance prediction to the \ngeneral  population.  Without  a  mechanistic  understanding  of  genetic \nresistance,  a  correlative  relationship  may  be  lineage-specific  and  less \ngeneralizable.  Especially  with  multiple  possible  mechanisms  of  resis-\ntance to a given antibiotic and ongoing evolutionary pressure resulting \nin the development and acquisition of new modes of resistance, a geno-\ntypic  approach  to  diagnosing  antibiotic  resistance  is  likely  to  remain \nchallenging and to require ongoing vigilance in constantly correlating \ngenotypic  with  more  traditional  phenotypic  methods.  An  important \ncorollary  benefit  of  a  genomic  approach  to  resistance  prediction, \nanchored in phenotypic validation, could be the systematic identifica-\ntion of outliers with unexplained resistance. These strains can form the \nbasis for understanding newly emerging resistance mechanisms, which \ncan in turn inform new drug development endeavors. Understanding \nresistance  mechanisms  may  also  help  direct  infection  control  efforts. \nFor  instance,  the  first  identification  of  the  mcr-1  (mobilized  colistin \nresistance) gene on a plasmid, together with other antibiotic resistance \ndeterminants, heightened concern about colistin-resistant Enterobac-\nteriaceae identified first in China and later elsewhere because it implied \nrapid  transmissibility  of  multidrug  resistance.  Early  recognition  of \nthese potentially dangerous strains elucidated the immediate need for \nstrict containment protocols.\n\nIn  parallel  with  advancing  sequencing  technologies,  progress  in \ncomputational techniques, bioinformatics and statistics, and data stor-\nage as well as experimental confirmatory testing of hypotheses will be \nneeded to advance toward the ambitious goal of a comprehensive com-\npendium  of  global  antibiotic  resistance  determinants.  Open  sharing \nand careful curation of new sequence information will be of paramount \nimportance, as will iterative or even continuous comparison of predic-\ntions with ongoing phenotypic testing in order to assess performance \nand  allow  prediction  algorithms  to  keep  up  with  newly  evolving  or \nemerging resistance mechanisms.\n\nWe continuously observe the accumulation of new or unanticipated \nmodes of resistance from ongoing evolutionary pressure caused by the \nwidespread  clinical  use  of  antibiotics.  Even  with  MRSA,  perhaps  the \nbest-studied case of antibiotic resistance and a model of relative sim-\nplicity with a single known monogenic resistance determinant (mecA), \na  genotype-based  approach  to  resistance  detection  proved  imperfect. \nOne limitation was a recall of the initial commercial genotypic resis-\ntance assay that was deployed for the identification of MRSA. A clinical \nisolate of S. aureus that emerged in Belgium expressed a variant of the \nmec  cassette  not  detected  by  the  assay\u2019s  PCR  primers.  New  primers \nwere added to detect this new variant, and the assay was reapproved for \nuse. This example illustrates the need for ongoing monitoring of any \ngenotypic resistance assay. A second limitation is that a contradiction \ncan occur between genotypic and phenotypic evidence for resistance. \nUp to 5% of MSSA strains have been reported to carry a copy of the \nmecA  gene  that  is  either  nonfunctional  or  not  expressed.  Thus,  the \nerroneous identification of these strains as MRSA by genotypic detec-\ntion would lead to administration of the inferior antibiotic vancomycin \nrather than the preferred \u03b2-lactam therapy.\n\nThese  examples  illustrate  one  of  the  prime  challenges  of  moving \nbeyond  growth-based  assays:  genotype  is  merely  a  proxy  for  the \nresistance  phenotype  that  directly  informs  patient  care.  Alternative \napproaches  currently  under  development  attempt  to  circumvent  the \nlimitations of genotypic resistance testing by returning to phenotypic \nassays,  albeit  more  rapid  ones.  One  such  approach  is  informed  by \ngenomic  methods:  transcriptional  profiles  serve  as  a  rapid  pheno-\ntypic signature for antibiotic response. Conceptually, since dying cells  \nare  transcriptionally  distinct  from  cells  fated  to  survive,  susceptible \nbacteria  enact  different  transcriptional  profiles  after  antibiotic  expo-\nsure than resistant ones, independent of the mechanism of resistance. \nThese  differences  can  be  measured  and,  since  transcription  is  one \n\n967\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n1\n\nM\ni\nc\nr\no\nb\ni\na\nl\n\nG\ne\nn\no\nm\ni\nc\ns\na\nn\nd\nI\nn\nf\ne\nc\nt\ni\no\nu\ns\nD\ni\ns\ne\na\ns\ne\n\nof  the  most  rapid  responses  to  cell  stress  (minutes  to  hours),  can  be \nused  to  determine  whether  cells  are  resistant  or  susceptible  much \nmore rapidly than is possible if growth in the presence of antibiotics \nis  awaited  (days).  Like  DNA,  RNA  can  be  readily  detected  through \npredictable  rules  governing  base  pairing  via  either  amplification  or \nhybridization-based  methods.  Changes  in  a  carefully  selected  set  of \ntranscripts  form  an  expression  signature  that  can  represent  the  total \ncellular response to antibiotic without requiring full characterization of \nthe entire transcriptome. Preliminary proof-of-concept studies suggest \nthat  this  approach  may  identify  antibiotic  susceptibility  on  the  basis \nof transcriptional phenotype much more quickly than is possible with \ngrowth-based assays. Other rapid phenotype-based approaches to anti-\nbiotic susceptibility testing, including automated microscopy, ultrafine \nmeasurements of mass fluctuations, and others are under development \nas well, with the former approved for clinical use.\n\nBecause  of  its  sensitivity  in  detecting  even  very  rare  nucleic  acid \nfragments, sequencing provides an unprecedented depth of study into \ncomplex populations of cells and tissues. The strength of this depth and \nsensitivity  applies  not  only  to  the  detection  of  rare,  novel  pathogens \nin a sea of host signal, but also to the identification of heterogeneous \npathogen subpopulations in a single host that may differ, for example, \nin drug resistance profiles or pathogenesis determinants. For instance, \nrecent  studies  have  highlighted  the  diversification  of  pathogens  in \nchronic  bacterial  infections,  such  as  Pseudomonas  in  the  lungs  of \npatients  with  cystic  fibrosis  or  M.  tuberculosis  in  disseminated  infec-\ntion,  perhaps  allowing  for  niche  specialization  within  the  host.  Such \ndiversification has long been recognized in chronic viral populations, \nas exemplified by HIV. Future studies will be needed to elucidate the \nclinical  significance  of  these  variable  subpopulations,  even  as  deep \nsequencing  is  now  providing  unprecedented  levels  of  detail  about \nmajority and minority members of this population.\n\n \u25a0 HOST-BASED DIAGNOSTICS\nWhile pathogen-based diagnostics continue to be the mainstay for con-\nfirming infection, serologic testing and nonspecific biomarkers\u2014such \nas  erythrocyte  sedimentation  rate,  C-reactive  protein  level,  and  even \ntotal white blood cell and neutrophil counts\u2014have long been the basis \nof  a  strategy  for  measuring  host  responses  to  aid  in  the  diagnosis  of \ninfection. Even recently identified host biomarkers of bacterial infec-\ntion,  such  as  procalcitonin,  have  fallen  short  in  their  versatility,  with \npositive  and  negative  predictive  values  that  are  thus  far  adequate  for \nonly a few narrow applications but inadequate for generalized clinical \nuse.  Here,  too,  the  application  of  genomics  is  now  being  explored  to \nimprove upon this approach, given the previously described limitations \nof  serologic  testing  and  the  lack  of  specificity  of  protein  biomarkers \nidentified to date. Rather than using antibody responses as a retrospec-\ntive biomarker for infection, recent efforts have focused on transcrip-\ntomic analysis of the host response as a new direction with diagnostic \nimplications for human disease.\n\nFor  instance,  while  pathogen-based  diagnostic  tests  to  distinguish \nactive  from  latent  tuberculosis  infection  have  proven  elusive,  recent \nwork shows that the transcriptional profile of circulating white blood \ncells  exhibits  a  differential  pattern  of  expression  of  nearly  400  tran-\nscripts that distinguish active from latent tuberculosis; this expression \npattern is driven in part by changes in interferon-inducible genes in the \nmyeloid lineage. In a validation cohort, this transcriptional signature \nwas  able  to  distinguish  patients  with  active  versus  latent  disease,  to \ndistinguish tuberculosis infection from other pulmonary inflammatory \nstates or infections, and to track responses to treatment in as little as  \n2 weeks, with normalization of expression toward that of patients with-\nout active disease over 6 months of effective therapy. Such a test could \nplay  an  important  role  not  only  in  the  management  of  patients  but \nalso as a marker of efficacy in clinical trials of new therapeutic agents. \nMore recently, a distilled three-transcript signature has shown promise \nfor  distinguishing  active  from  latent  tuberculosis,  raising  hopes  of  a \ndeployable assay in the near term.\n\nSimilarly, considerable progress has been made toward identifying \nhost  transcriptional  signatures  in  circulating  blood  cells  that  distin-\nguish  viral  from  bacterial  causes  of  upper  respiratory  infection,  with \n\nHPIM21e_Part05_p941-p1796.indd   967\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n"
  },
  {
    "page": "968",
    "text": "968\n\nbetter  performance  characteristics  than  current  clinical  parameters \nor available protein biomarkers. Additional host signatures have been \nreported  that  distinguish  among  bacterial  infection,  viral  infection, \nand inflammatory states; identify Lyme disease; identify influenza; and \neven  distinguish  between  gram-positive  and  gram-negative  bacterial \ninfections. In some cases, results have been extended to different host \npopulations\u2014including  adults  and  children,  and  those  with  varying \nimmune  function\u2014which  obviously  will  be  critical  for  generalizing \nsuch  an  approach.  Thus,  profiling  of  host  transcriptional  dynamics \ncould  augment  the  information  obtained  from  studies  of  pathogens, \nboth  enhancing  diagnosis  and  monitoring  the  progression  of  illness \nand  the  response  to  therapy.  The  frontier  of  genomic  applications  to \nunderstand  host  response  to  infection,  with  the  potential  of  identi-\nfying  biomarkers  or  even  underlying  disease  biology,  continues  to \nrapidly advance, incorporating novel technological and computational \napproaches, such as single-cell host transcriptional profiling of infected \npatients, to understand complex processes such as sepsis.\n\nIn this era of genome-wide association studies and attempts to move \ntoward  personalized  medicine,  genomic  approaches  are  also  being \napplied to the identification of host genetic loci and factors that con-\ntribute to infection susceptibility. Such loci will have undergone strong \nselection among populations in which the disease is endemic. Through \nidentification of the beneficial genetic alleles among individuals who \nsurvive  in  such  settings,  markers  for  susceptibility  or  resistance  are \nbeing  discovered;  these  markers  can  be  translated  to  diagnostic  tests \nto identify susceptible individuals in order to implement preventive or \nprophylactic interventions. Further, such studies may offer mechanistic \ninsight  into  the  pathogenesis  of  infection  and  inform  new  methods \nof therapeutic intervention. Such beneficial genetic associations were \nrecognized  long  before  the  advent  of  genomics,  as  in  the  protective \neffects of the negative Duffy blood group or heterozygous hemoglobin \nabnormalities  against  Plasmodium  infection.  Genomic  approaches \nallow  more  systematic  and  widespread  application  of  this  principle \nto  identify  not  only  people  with  increased  susceptibility  to  prevalent \ndiseases (e.g., HIV infection, tuberculosis, and cholera) but also host \nfactors that contribute to and thus might predict the severity of disease, \nincluding studies currently under way for COVID-19, which displays \nmarkedly variable severity that is thus far poorly understood.\n\nTHERAPEUTICS\nGenomics has the potential to impact infectious disease therapeutics in \ntwo ways. By transforming the speed or type of diagnostic information \nthat  can  be  attained,  it  can  influence  therapeutic  decision-making. \nAlternatively,  by  opening  new  avenues  to  a  better  understanding  of \npathogenesis, providing new ways to disrupt infection, and delineating \nnew approaches to antibiotic discovery, it has the potential to facilitate \nthe development of new therapeutic agents.\n \u25a0 GENOMIC DIAGNOSTICS INFORMING \nTHERAPEUTICS\nEfforts  at  antibiotic  discovery  are  declining,  with  few  new  agents  in \nthe pipeline and even fewer new drugs (in particular, few agents with \nnew mechanisms of action) entering the market. This phenomenon is \ndue in part to the lack of economic incentives for the private sector; \nhowever,  it  is  also  attributable  in  part  to  the  enormous  challenges \ninvolved in the discovery and development of antibiotics. Most recent \nefforts have focused on broad-spectrum antibiotics; the development \nof  a  chemical  entity  that  works  across  an  extremely  diverse  set  of \norganisms (i.e., species more divergent from each other than a human \nis from an amoeba) is far more challenging than the development of \nan agent that is designed to target a single bacterial species. Neverthe-\nless,  the  concept  of  narrow-spectrum  antibiotics  has  heretofore  been \nrejected because of the lack of early diagnostic information that would \nguide the selection of such agents. Thus, rapid diagnostics providing \nantibiotic susceptibility information that can guide antibiotic selection \nin real time has the potential to alter and simplify antibiotic strategies \nby  allowing  a  paradigm  shift  away  from  broad-spectrum  drugs  and \ntoward narrow-spectrum agents. Such a paradigm shift clearly would \nhave additional implications for antibiotic resistance, helping to limit \n\nselective pressure applied to pathogens and commensal bacteria during \ntherapy.\n\nIn  yet  another  diagnostic  paradigm  with  the  potential  to  impact \ntherapeutic interventions, genomics is opening new avenues to a better \nunderstanding  not  only  of  different  host  susceptibilities  to  infection \nbut  also  of  different  host  responses  to  therapy.  For  example,  the  role \nof  glucocorticoids  in  tuberculous  meningitis  has  long  been  debated. \nRecently,  polymorphisms  in  the  human  genetic  locus  LTA4H,  which \nencodes  a  leukotriene-modifying  enzyme,  were  found  to  modulate \nthe inflammatory response to tuberculosis. Patients with tuberculous \nmeningitis  who  were  homozygous  for  the  proinflammatory  LTA4H \nallele were most helped by adjunctive glucocorticoid treatment, while \nthose who were homozygous for the anti-inflammatory allele were neg-\natively affected by steroid treatment. Steroids have become part of the \nstandard of care in tuberculous meningitis, but this study suggests that \nperhaps only a subset of patients benefit from this anti-inflammatory \nadjunct (while others may be harmed) and further suggests a genetic \nmeans of prospectively identifying this subset. Thus, genomic diagnos-\ntic  tests  may  eventually  approach  the  goal  of  personalized  medicine, \ninforming diagnosis, prognosis, and treatment decisions by revealing \nthe pathogenic potential of the microbe and by detecting individual-\nized host responses to both infection and therapy.\n\n \u25a0 GENOMICS IN DRUG AND VACCINE \nDEVELOPMENT\nGenomic  technologies  are  dramatically  changing  research  on  host\u2013\npathogen  interactions,  with  a  goal  of  increasingly  influencing  the \nprocess of therapeutic discovery and development. Sequencing offers \nseveral possible avenues into antimicrobial therapeutic discovery. First, \ngenome-scale molecular methods have paved the way for comprehen-\nsive identification of all essential genes encoded by a pathogen, thereby \nsystematically  identifying  critical  vulnerabilities  within  a  pathogen \nthat could be targeted therapeutically. Second, genome-scale method-\nologies offer rapid ways to address the mechanism of action of newly \nidentified  hits  from  compound  screens.  Whole  genome  sequencing \noffers  a  rapid,  unbiased  way  to  detect  mutations  arising  in  resistant \nmutants  during  selection.  Similarly,  transcriptional  profiling  can \nprovide  insights  into  mechanisms  of  action  of  new  candidate  drugs. \nFor instance, the transcriptional signature of cell wall disruptors (e.g., \n\u03b2-lactams) is distinct from that of DNA-damaging agents (e.g., fluoro-\nquinolones)  or  protein  synthesis  inhibitors  (e.g.,  aminoglycosides). \nEither approach can thus suggest a mechanism of action or flag com-\npounds for prioritization because of a potentially novel activity. In an \nalternative  genomic  strategy  for  determining  mechanisms  of  action, \nan  RNA  interference  approach  followed  by  targeted  sequencing  was \nused  to  identify  genes  required  for  antitrypanosomal  drug  efficacy. \nThis approach provided new insights into the mechanism of action of \ndrugs that have been in use for decades for human African trypano-\nsomiasis.  Third,  sequencing  can  readily  identify  the  most  conserved \nregions of a pathogen\u2019s genomes and corresponding gene products; this \ninformation is invaluable in narrowing antigen candidates in vaccine \ndevelopment. These surface proteins can be expressed recombinantly \nand tested for the ability to elicit a serologic response and protective \nimmunity. This process, termed reverse vaccinology, has proved partic-\nularly useful for pathogens that are difficult to culture or poorly immu-\nnogenic. More directly, mRNA vaccines targeting conserved regions of \nthe SARS-CoV-2 genome were developed at record speed in the face \nof unprecedented urgency, enabled by the rapid availability of genomic \nsequencing data. These vaccines proved remarkably effective in initial \nstudies  and  represent  a  dramatic  breakthrough  in  efforts  to  mitigate \nthe pandemic. Moreover, this novel vaccine platform offers a facile way \nto deliver new viral antigens if needed based on genomic surveillance.\nGenomics  has  been  employed  in  both  developing  vaccines  and \ndefining their impact on microbial epidemiology and ecology. Exam-\nples  include  recent  studies  of  influenza,  malaria,  S.  pneumoniae,  and \nHPV  following  vaccine  introduction.  Extensive  sequencing  of  influ-\nenza viruses has been valuable in understanding the modest efficacy \nof  seasonal  influenza  vaccination,  and  the  combination  of  genomics \nand antigenic cartography is proving helpful in the selection of strains \n\nHPIM21e_Part05_p941-p1796.indd   968\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "969",
    "text": "to include in subsequent influenza vaccines. The RTS,S/AS01 malaria \nvaccine was analyzed by targeted sequencing of parasites from vacci-\nnated and control populations during a phase 3 trial conducted at 11 \nsites in Africa; these analyses revealed reduced vaccine efficacy against \nparasites with amino acid mutations in the circumsporozoite protein \ntargeted by the vaccine. Similarly, studies of the more established pneu-\nmococcal vaccines (the 7- and 13-valent polysaccharide conjugate vac-\ncines, PCV-7 and PCV-13) documented serotype replacement: strains \ntargeted  by  the  vaccine  have  dramatically  decreased  in  prevalence \nfollowing widespread vaccination campaigns. Given that specific sero-\ntypes of HPV (e.g., types 16 and 18) clearly are more strongly associ-\nated than others with carcinogenesis, HPV vaccines have capitalized on \nserotype replacement, targeting vaccine strains to specifically prevent \ninfection with the more dangerous serotypes. Such a strategy, informed \nby  pathogen  genomics,  aims  to  protect  individuals  and  ideally  to \ndecrease the circulating burden of more virulent strains within society.\nLarge-scale  gene  content  analysis  from  sequencing  or  expression \nprofiling  enables  new  research  directions  that  provide  novel  insights \ninto  the  interplay  of  pathogen  and  host  during  infection  or  coloniza-\ntion. One important goal of such research is to suggest new therapeutic \napproaches to disrupt this interaction in favor of the host. Indeed, one \nof  the  most  immediate  applications  of  next-generation  sequencing \ntechnology has come from simply characterizing human pathogens and \nrelated commensal or environmental strains and then finding genomic \ncorrelates for pathogenicity. For instance, as Escherichia coli varies from \na  simple  nonpathogenic,  lab-adapted  strain  (K-12)  to  a  Shiga  toxin\u2013 \nproducing  enterohemorrhagic  gastrointestinal  pathogen  (O157:H7),  it \ndisplays up to a 25% difference in gene content, though it is classified \nas  the  same  species.  Similarly,  some  isolates  of  Enterococcus\u2014a  genus \nnotorious for its increasing incidence of resistance to common antibiot-\nics such as ampicillin, vancomycin, and aminoglycosides\u2014also contain \nrecently acquired genetic material comprising up to 25% of the genome \non  mobile  genetic  elements.  This  fact  suggests  that  horizontal  gene \ntransfer plays an important role in the organisms\u2019 adaptation as noso-\ncomial pathogens. On closer study, this genome expansion is associated \nwith loss of CRISPR elements, which protect the bacterial genome from \ninvasion by certain foreign genetic material, and may thus facilitate the \nacquisition of antibiotic resistance\u2013conferring genetic elements. While \nloss  of  this  regulation  appears  to  impose  a  competitive  disadvantage \nin  antibiotic-free  environments,  these  drug-resistant  strains  thrive  in \nthe  presence  of  even  some  of  the  best  antienterococcal  therapies.  In \naddition  to  insights  gained  from  genome  sequencing,  extension  of \nunbiased whole-transcriptome sequencing (RNA-Seq) efforts to bacte-\nria is beginning to identify unexpected regulatory, noncoding RNAs in \nmany diverse species. While the functional implications of these new \ntranscripts are as yet largely unknown, the presence of such features\u2014\nconserved across many bacterial species\u2014implies evolutionary impor-\ntance and suggests areas for future study and possible new therapeutic \navenues. Transcriptomic and proteomic profiling of pathogens under \nvarious  conditions  that  mimic  colonization  or  infection,  including \nexistence  as  biofilms  or  in  polymicrobial  communities,  intracellular \ninfection  models,  antibiotic  exposure,  and  nutrient  starvation,  has \nbegun to reveal novel biologic features that may be targeted by the next \ngeneration of therapies. At the cutting edge of the host\u2013pathogen inter-\nface, single-cell transcriptomic methodologies are rapidly increasing in \nfeasibility and extent, revealing previously unknown heterogeneity in \nthe potential outcomes of intracellular infection.\n\nThus, genomic studies are transforming our understanding of infec-\ntion,  offering  evidence  of  virulence  factors  or  toxins  and  providing \ninsight  into  ongoing  evolution  of  pathogenicity  and  drug  resistance. \nOne  goal  of  such  studies  is  to  identify  therapeutic  agents  that  can \ndisrupt the pathogenic process. There is currently much interest in the \ntheoretical concept of antivirulence drugs that inhibit virulence factors \nrather  than  killing  the  pathogen  outright  as  a  means  to  intervene  in \ninfection. Further, with sequencing ever more accessible and efficient, \nongoing  large-scale  studies  have  unprecedented  statistical  power  to \nassociate  clinical  outcomes  with  pathogen  and  host  genotypes  and \nthus to further reveal vulnerabilities in the infection process that can \nbe  targeted  for  disruption.  Although  this  is  just  the  beginning,  such \n\nstudies point to a tantalizing future in which the clinician is armed with \ngenomic predictors of infection outcome and therapeutic response to \nguide clinical decision-making.\n\n969\n\nEPIDEMIOLOGY OF INFECTIOUS DISEASES\nEpidemiologic  studies  of  infectious  diseases  have  several  main  goals: \nto  identify  and  characterize  outbreaks,  to  describe  the  pattern  and \ndynamics  of  an  infectious  disease  as  it  spreads  through  populations, \nand  to  identify  interventions  that  can  limit  or  reduce  the  burden  of \ndisease. One classic, paradigmatic example is John Snow\u2019s elucidation \nof the origin of the 1854 London cholera outbreak. Snow used careful \ngeographic mapping of cases to determine that the likely source of the \noutbreak  was  contaminated  water  from  the  Broad  Street  pump,  and \nby removing the pump handle, he aborted the outbreak. Whereas that \neffort  was  undertaken  without  knowledge  of  the  causative  agent  of \ncholera,  advances  in  microbiology  and  genomics  have  expanded  the \npurview of epidemiology to consider not just the disease but also the \npathogen, its virulence factors, and the complex relationships between \nmicrobial and host populations.\n\nThrough  use  of  genomic  tools  such  as  high-throughput  sequenc-\ning,  the  diversity  of  a  microbial  population  can  be  rapidly  described \nwith  unprecedented  resolution,  with  discrimination  between  isolates \nthat  have  single-nucleotide  differences  across  the  entire  genome  and \nadvancement  beyond  prior  approaches  that  relied  on  phenotypes \n(such as antibiotic susceptibility profiles) or genetic markers (such as \nmultilocus sequence typing). The development of statistical methods \ngrounded  in  molecular  genetics  and  evolutionary  theory  has  estab-\nlished  analytical  approaches  that  translate  descriptions  of  microbial \npopulation diversity and structure into descriptions of the origin and \nhistory  of  pathogen  spread.  By  linking  phylogenetic  reconstruction \nwith  epidemiologic  and  demographic  data,  genomic  epidemiology \npresents the opportunity to track transmission from person to person \nand  across  demographic  and  geographic  boundaries,  to  infer  trans-\nmission patterns of both pathogens and sequence elements that confer \nphenotypes of interest, and to estimate the transmission dynamics of \noutbreaks.\n\n \u25a0 TRANSMISSION NETWORKS\nWhole genome sequencing of pathogen genomes can be used to infer \ntransmission and identify point-source outbreaks. As reported in a sem-\ninal paper in 2010, a study of MRSA in a Thai hospital demonstrated the \nuse of whole genome sequencing in reconstructing the transmission of \na pathogen from patient to patient by integrating the analysis of accu-\nmulation of mutations over time with the dates and hospital locations \nof the infected individuals. Since then, multiple instances of the use of \nwhole genome sequencing to define and motivate interventions aimed \nat  interrupting  transmission  chains  have  been  reported.  In  another \nMRSA  outbreak  in  a  special-care  baby  unit  in  Cambridge,  United \nKingdom, whole genome sequencing extended the traditional infection \ncontrol  analysis,  which  relies  on  typing  organisms  by  their  antibiotic \nsusceptibilities,  to  sequencing  of  isolates  from  clinical  samples.  This \napproach identified an otherwise unrecognized outbreak of a specific \nMRSA  strain  that  was  occurring  against  a  background  of  the  usual \npattern of infection caused by a diverse circulating population of MRSA \nstrains. The analysis showed evidence of transmission among mothers \nwithin the special-care baby unit and in the community and demon-\nstrated the key role of MRSA carriage in a single health care provider in \nthe persistence of the outbreak. In yet another example, in response to \nthe observation of 18 cases of infection by carbapenemase-producing  \nKlebsiella pneumoniae over 6 months at the National Institutes of Health \nClinical Research Center, genome sequencing of the isolates was used \nto  discriminate  between  the  possibilities  that  these  cases  represented \nmultiple,  independent  introductions  into  the  health  care  system  or  a \nsingle introduction with subsequent transmission. On the basis of net-\nwork and phylogenetic analysis of genomic and epidemiologic data, the \nauthors reconstructed the likely relationships among the isolates from \npatient to patient, demonstrating that the spread of resistant Klebsiella \ninfection was in fact due to nosocomial transmission of a single strain. \nSimilar  approaches  have  elucidated  the  extent  to  which  presumed \n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n1\n\nM\ni\nc\nr\no\nb\ni\na\nl\n\nG\ne\nn\no\nm\ni\nc\ns\na\nn\nd\nI\nn\nf\ne\nc\nt\ni\no\nu\ns\nD\ni\ns\ne\na\ns\ne\n\nHPIM21e_Part05_p941-p1796.indd   969\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n"
  },
  {
    "page": "970",
    "text": "970 nosocomial C. difficile, VRE, and carbapenem-resistant Enterobacteri-\naceae represent within-hospital transmission rather than independent \nacquisitions. With these demonstrations of the potential contribution \nof genomics to hospital infection-control efforts, an important avenue \nof research seeks to develop statistical methods with which to ascertain \nwhen such tools are useful and their cost-effectiveness when compared \nwith that of current nongenomic approaches.\n\nGenome sequencing of clinical specimens of viruses has been used \nto understand their epidemiologic patterns of spread. As RNA viruses \nuse an error-prone RNA-dependent RNA polymerase, they accumulate \nmutations at a rapid rate, facilitating inferences about the dynamics and \npatterns of spread. These tools have been applied to the study of out-\nbreaks of well-known viruses, such as recent outbreaks of yellow fever \nin South America and mumps in the United States, as well as recent \nzoonotic pathogens, such as the coronaviruses MERS-CoV and SARS-\nCoV. The sequencing of SARS-CoV-2 in the context of the pandemic \nhas  offered  a  powerful  example  of  the  contributions  that  genomic \nepidemiology can make to, and its increasingly central role in, tracking \nthe spread of a pathogen both locally and globally and informing policy \nand public health decision-making.\n\nThe  uncovering  of  unexpected  transmission  events  by  genomic \nepidemiology studies is motivating investigations into pathogen ecol-\nogy  and  modes  of  transmission.  For  example,  the  rise  in  prevalence \nof  infections  with  nontuberculous  mycobacteria,  including  Myco-\nbacterium  abscessus,  among  patients  with  cystic  fibrosis  has  led  to \nspeculation  about  the  possible  role  of  patient-to-patient  transmis-\nsion  in  the  cystic  fibrosis  community;  however,  conventional  typing \napproaches have lacked the resolution to define pathogen population \nstructure  accurately,  a  critical  component  of  inferring  transmission. \nPast  infection-control  guidelines  discounted  the  possibility  of  acqui-\nsition  of  nontuberculous  mycobacteria  in  health  care  settings,  as  no \nstrong  evidence  for  such  transmission  had  been  described.  In  whole \ngenome sequencing studies of M. abscessus isolates from patients with \ncystic fibrosis, an analytical approach using genome sequencing, epi-\ndemiology, and Bayesian modeling revealed that, contrary to the prior \nbelief  that  infections  with  M.  abscessus  are  independently  acquired, \nthe  majority  of  infections  appear  to  be  transmitted.  Because  there \nare often no clear epidemiologic links that place the infected patients \nin the same place at the same time, this finding highlights a need to \nexplore preexisting notions of circumstances required for transmission, \nincluding the roles of fomites and aerosols, and a reconsideration of M. \nabscessus infection-control guidelines. In a clear example of the utility \nof  whole  genome  sequencing  for  revealing  unexpected  transmission \nnetworks,  isolates  of  M.  chimaera  causing  infections  after  cardiotho-\nracic  surgery  in  patients  in  different  locations  were  all  found  to  be \nclosely related. These isolates differed from one another by at most 38 \npairwise single nucleotide polymorphisms out of >5 million bases; in \ncontrast, they differed by >2900 single nucleotide polymorphisms from \nthe  nonclonally  related  reference  isolate.  Although  a  hospital  source \nwas  initially  suspected  when  the  first  of  these  cases  were  identified, \nthis  whole  genome  sequencing  analysis  strongly  supported  a  single \npoint-source for these geographically dispersed isolates. A subsequent \ninvestigation ultimately implicated M. chimaera contamination in the \nmanufacturing  chain  of  a  temperature-control  system  used  during \ncardiac bypass. Similar studies of other pathogens\u2014particularly those \nthat share human, other animal host, and environmental reservoirs\u2014\nwill  continue  to  advance  our  understanding  of  the  relative  roles  and \nprominence of sources of infection and the modes of spread through \npopulations, thereby establishing evidence-based strategies for preven-\ntion and intervention.\n\nAs  more  studies  aim  to  carefully  define  the  origins  and  spread  of \ninfectious  agents  using  the  high-resolution  lens  of  whole  genome \nsequencing, fundamental questions arise about the diversity of infect-\ning  and  colonizing  microbial  populations.  Traditional  microbiologic \nmethods include taking a single colony from a growth plate as repre-\nsentative of the population. However, the more diverse the colonizing \nor  infecting  pathogen  population,  the  less  representative  these  indi-\nvidual isolates are and the greater the possibility for introducing error \ninto  whole  genome  sequencing\u2013based  methods  while  reconstructing \n\ntransmission. Sequencing studies of multiple colonies of an S. aureus \nstrain colonizing a single individual showed a \u201ccloud\u201d of diversity. What \nis the clinical significance of this diversity? What are the processes that \ngenerate and limit it? What amount of diversity is transmitted under \ndifferent conditions and routes of transmission? How do the answers \nto these questions vary by infectious organism, type of infection, host, \nand response to treatment? More comprehensive descriptions of diver-\nsity, population dynamics, transmission bottlenecks, and the forces that \nshape  and  influence  the  growth  and  spread  of  microbial  populations \nwill be a critically important focus of future investigations.\n\n \u25a0 ORIGINS AND DYNAMICS OF PATHOGEN SPREAD\nIn  addition  to  reconstructing  the  transmission  chains  of  local  out-\nbreaks,  genomics-based  epidemiologic  methods  reveal  broad-scale \ngeographic  and  temporal  spread  of  pathogens.  Four  recent  examples \ninclude the origins of cholera in Haiti, the history of HIV-1 group M, \nthe spread of Ebola in West Africa, and the timing and nature of spread \nof the zoonotic COVID-19 pandemic. Cholera, a dehydrating diarrheal \nillness caused by infection with Vibrio cholerae, first spread worldwide \nfrom the Indian subcontinent in the 1800s and has since caused seven \npandemics; the seventh pandemic has been ongoing since the 1960s. \nAn investigation into the geographic patterns of cholera spread in the \nseventh pandemic used genome sequences from a global collection of \n154  V.  cholerae  strains  representing  isolates  from  1957  to  2010.  This \ninvestigation  revealed  that  the  seventh  pandemic  has  comprised  at \nleast three overlapping waves spreading out from the Indian subcon-\ntinent (Fig. 121-4A). Further, analysis of the genome of an isolate of \nV.  cholerae  from  the  2010  outbreak  of  cholera  in  Haiti  showed  it  to \nbe  more  closely  related  to  isolates  from  South  Asia  than  to  isolates \nfrom  neighboring  Latin  America,  supporting  the  hypothesis  that  the \noutbreak was derived from V. cholerae introduced into Haiti by human \ntravel (likely from Nepal) rather than by environmental or more geo-\ngraphically proximal sources. A subsequent study that dated the time \nto  the  most  recent  common  ancestor  of  a  population  of  V.  cholerae \nisolates from Haiti provided further support for a single point-source \nintroduction from Nepal. Application of similar methods that integrate \npathogen  genome  sequences,  mutation  rates,  geographic  locations, \nand phylogenetic inference to HIV-1 group M dated the origin of the \nvirus to the 1920s and the city of Kinshasa (then called Leopoldville), \nthe capital of the Democratic Republic of the Congo (then called the  \nBelgian  Congo).  This  work  established  an  understanding  of  how  a \nboom in industry and a city with extensive railroad connections pro-\nvide a scaffolding along which a virus can rapidly spread geographically.\nGenome  sequencing  has  proven  invaluable  in  understanding  the \ngeographic,  demographic,  climatic,  and  administrative  factors  that \ndrove, sustained, and limited the 2013\u20132016 Ebola outbreak that rav-\naged West Africa (Fig. 121-4B) as well as the factors and patterns of \ntransmission of Zika virus in the Americas and most recently the tim-\ning  and  origins  of  SARS-CoV-2  transmission  in  human  populations. \nWith the rapid availability of the SARS-CoV-2 viral genome sequence, \ndata from a set of cases from early in the pandemic enabled inference of \nthe time to the most recent common ancestor, supporting that SARS-\nCoV-2  entered  circulation  in  human  populations  in  Wuhan,  China, \nsometime in late November to early December of 2019. Subsequently, \nlarge, coordinated sequencing networks have been able to recreate its \npattern of early global spread.\n\nThese efforts illustrate the remarkable promise of genome sequenc-\ning in improving outbreak response strategies by elucidating previously \nhidden  origins  and  paths  of  disease  spread  and  details  of  the  forces \nthat shape epidemics. The combination of in-the-field sequencing with \nportable  sequencing  platforms,  rapid  data  sharing,  and  rapid  open \nanalysis through sites such as nextstrain.org offers a paradigm by which \nreal-time  genomic  epidemiology  may  contribute  to  \u201cweather  maps,\u201d \nenabling prediction of epidemic patterns and thus providing guidance \nfor public health interventions to slow or control their spread.\n\nIncreasing numbers of investigations into the spread of many patho-\ngens  are  contributing  to  a  growing  atlas  of  maps  describing  routes, \npatterns,  and  tempos  of  microbial  diversification  and  dissemination, \nnot  just  for  agents  of  emerging  infectious  diseases  but  for  common \n\nHPIM21e_Part05_p941-p1796.indd   970\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "971",
    "text": "971\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n1\n\nM\ni\nc\nr\no\nb\ni\na\nl\n\nG\ne\nn\no\nm\ni\nc\ns\na\nn\nd\nI\nn\nf\ne\nc\nt\ni\no\nu\ns\nD\ni\ns\ne\na\ns\ne\n\nA\n\nB\n\nFIGURE 121-4  A. Transmission events inferred from phylogenetic reconstruction of 154 Vibrio cholerae isolates from the seventh cholera pandemic. Date ranges represent \nestimated time to the most recent common ancestor for strains transmitted from source to destination locations, based on a Bayesian model of the phylogeny. (Reprinted by \npermission from the Nature Publishing Group, Nature 477:462. Evidence for several waves of global transmission in the seventh cholera pandemic, A Mutreja et al. \u00a9 2011.)  \nB. Inferred Ebola virus spread in West Africa (Liberia, red; Guinea, green; and Sierra Leone, blue) by phylogeographic methods using virus genome sequences, dates, and an \nevolutionary model. The lines reflect spread between population centroids of each administrative region, going from the thin end to the thick end and colored by a time scale. \n(Reprinted by permission from Nature Publishing Group, Nature 544:309. Virus genomes reveal factors that spread and sustained the Ebola epidemic, G Dudas et al. \u00a9 2017.)\n\npathogens as well. Such studies will create a vast amount of data that \ncan be used to investigate the diversity and microbiologic links within \ndistinct niches and the patterns of spread from one niche to another. \nThe  increasingly  broad  adoption  of  genome  sequencing  by  health \ncare and public health institutions ensures that the available catalog of \ngenome  sequences  and  associated  epidemiologic  data  will  grow  very \nrapidly.  For  example,  updating  from  the  pulsed-field  gel  electropho-\nresis  techniques  that  have  been  used  to  define  strains  of  food-borne \npathogens since the late 1980s, PulseNet\u2014the U.S. Centers for Disease \nControl and Prevention network for monitoring these pathogens\u2014has \ninstituted routine genome sequencing. The COVID-19 pandemic fur-\nther underscores the importance of building a new global public health \ninfrastructure in which sequencing plays a central role to facilitate early \ndisease discovery, rapid and close tracking of spread, and development \n\nof  diagnostics  and  targeted  effective  interventions.  With  higher- \nresolution  description  of  microbial  diversity  and  of  the  dynamics  of \nthat  diversity  over  time  and  across  epidemiologic  and  demographic \nboundaries and evolutionary niches, we will gain even greater insights \ninto the relationships of transmission routes and patterns of historical \nspread.\n\n \u25a0 EPIDEMIC POTENTIAL\nDefining pathogen transmissibility is a critical step in the development \nof  public  health  surveillance  and  intervention  strategies  because  this \ninformation can help to predict the epidemic potential of an outbreak. \nTransmissibility  can  be  estimated  by  a  variety  of  methods,  including \ninference  from  the  growth  rate  of  an  epidemic  and  the  generation \ntime of an infection (the mean interval between infection of an index \n\nHPIM21e_Part05_p941-p1796.indd   971\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n"
  },
  {
    "page": "972",
    "text": "972\n\ncase and infection of the people infected by that index case). Genome \nsequencing and analysis of a well-sampled population provide another \nmethod by which to derive similar fundamental epidemiologic param-\neters.  One  key  measure  of  transmissibility  is  the  basic  reproduction \nnumber, defined as the number of secondary infections generated from \na single primary infectious case. When the basic reproduction number \nis >1, an outbreak has epidemic potential; when it is <1, the outbreak \nwill  become  extinct.  On  the  basis  of  sequences  from  influenza  virus \nsamples obtained from infected patients very early in the 2009 H1N1 \ninfluenza  pandemic,  the  basic  reproduction  number  was  estimated \nthrough  a  population  genomic  analysis  at  1.2;  this  result  provided \ngreater  confidence  to  estimates  derived  by  traditional  epidemiologic \ndata, which ranged from 1.4 to 1.6. In addition, with the assumption \nof a molecular clock model, sequences of H1N1 samples together with \ninformation  about  when  and  where  the  samples  were  obtained  have \nbeen used to estimate the date and location of the pandemic\u2019s origin, \nproviding insight into disease origins and dynamics. No doubt, similar \nanalyses will be explored for SARS-CoV-2, as more data become avail-\nable. Integrating viral genomics with other types of data\u2014such as the \ntiming and nature of mitigation efforts and the impact of those efforts \non mobility\u2014will expand the toolkit with which to assess the impact of \npublic health interventions on slowing and controlling disease spread. \nThese  tools  may  also  be  applied  to  institutional  infection  control: \nwith the development of return-to-work protocols, sequencing offers \none  option  to  help  learn  the  extent  to  which  infections  arose  from \nwithin-institution spread. Because the magnitude and intensity of the \npublic health response are guided by the predicted size of an outbreak, \nthe ability of genomic methods to cast light on a pathogen\u2019s origin and \nepidemic potential adds an important dimension to the contributions \nof these methods to infectious disease epidemiology.\n\n \u25a0 PATHOGEN EVOLUTION\nBeyond  describing  transmission  and  dynamics,  pathogen  genomics \ncan provide insight into the evolution of pathogens and the interactions \nof selective pressures, the host, and pathogen populations, which can \nhave  implications  for  clinical  decision-making  and  the  development \nof vaccines and therapeutics. From a clinical perspective, this process \nis central to the acquisition of antibiotic resistance, the generation of \nincreasing  pathogenicity  or  new  virulence  traits,  the  evasion  of  host \nimmunity  and  clearance  (leading  to  chronic  infection),  and  vaccine \nefficacy.\n\nMicrobial genomes evolve through a variety of mechanisms, includ-\ning  mutation,  duplication,  insertion,  deletion,  recombination,  and \nhorizontal gene transfer. Segmented viruses (e.g., influenza virus) can \nreassort  gene  segments  within  multiply  infected  cells.  The  pandemic \n2009  H1N1  influenza  A  virus,  for  example,  appears  to  have  been \ngenerated  through  reassortment  of  several  avian,  swine,  and  human \ninfluenza strains. Such potential for the evolution of novel pandemic \nstrains has precipitated concern about the possible evolution to trans-\nmissibility  of  virulent  strains  that  have  been  associated  with  high \nmortality rates but have not yet exhibited efficient human infectivity. \nExperiments  with  H5N1  avian  influenza,  for  example,  have  defined \nfive  mutations  that  render  it  transmissible,  at  least  in  ferrets\u2014the \nanimal  model  system  for  human  influenza.  Studies  that  examine  the \ngenomes of pathogens collected longitudinally from individual infec-\ntions  have  similarly  demonstrated  the  evolution  of  bacteria  as  they \nadapt from colonization to invasion and to new host environments and \nnew immune and therapeutic pressures.\n\nThe continuous antigenic evolution of seasonal influenza offers an \nexample of how studies of pathogen evolution can impact surveillance \nand  vaccine  development.  Frequent  updates  to  the  annual  influenza \nvaccine are needed to ensure protection against the dominant strains. \nThese updates are based on anticipating which viral populations from \na pool of substantial locally and globally diverse circulating viruses will \npredominate  in  the  upcoming  season.  Toward  that  end,  sequencing- \nbased studies of influenza virus dynamics have shed light on the global \nspread of influenza, providing concrete data on patterns of spread and \nhelping  to  elucidate  the  origins,  emergence,  and  circulation  of  novel \n\nstrains.  Through  analysis  of  >1000  influenza  A  H3N2  virus  isolates \nover the 2002\u20132007 influenza seasons, Southeast Asia was identified as \nthe usual site from which diversity originates and spreads worldwide. \nFurther studies of global isolate collections have shed further light on \nthe diversity of circulating virus, showing that some strains persist and \ncirculate outside of Asia for multiple seasons.\n\nNot  only  do  genomic  epidemiology  studies  have  the  potential  to \nhelp guide vaccine selection and development, but they are also helping \nto  track  what  happens  to  pathogens  circulating  in  the  population  in \nresponse to vaccination. By describing pathogen evolution under the \nselective pressure of a vaccinated population, such studies can play a \nkey  role  in  surveillance  and  identification  of  virulence  determinants \nand perhaps may even help to predict the future evolution of escape \nfrom  vaccine  protection.  The  seven-valent  pneumococcal  conjugate \nvaccine (PCV-7) targeted the seven serotypes of S. pneumoniae respon-\nsible for the majority of invasive disease at the time of its introduction \nin 2000; since then, PCV-7 has dramatically reduced the incidence of \npneumococcal  disease  and  mortality.  However,  sequencing  of  >600 \nMassachusetts  pneumococcal  isolates  from  2001  to  2007  has  shown \nthat,  in  the  pneumococcal  population,  previously  rare  nonvaccine \nserotypes are replacing vaccine serotypes and that some vaccine strains \nhave persisted despite vaccination by recombining the vaccine-targeted \ncapsule  locus  with  a  cassette  of  capsule  genes  from  non-vaccine- \ntargeted serotypes.\n\nThe  large  collections  of  pathogen  genome  sequences  are  driving \ndevelopment of tools to decipher the genetic basis for antibiotic resis-\ntance, virulence, and infection risk. Some pathogens have distinct types \nof clinical manifestations, the basis for which we are just beginning to \nunravel with the aid of genomics. For example, Listeria is a food-borne \npathogen  that  can  cause  both  central  nervous  system  infections  and \nmaternal/neonatal infections. Although all Listeria isolates are treated \nthe  same  from  a  public  health  perspective,  variation  in  outcomes \nexists  and  appears  to  be  linked  to  the  strains\u2019  genomic  background. \nMolecular  analysis  of  a  national  reference  laboratory\u2019s  collections  of \nwell-characterized specimens, based on the fraction of immunocom-\npetent people in which they caused disease, revealed that some clonal \ncomplexes of Listeria appear to be more virulent than others. Linking \nepidemiology  and  comparative  genomics  then  enabled  enumeration \nof putative virulence factors that contribute to the clinical phenotypes \nas well as identification and confirmation of a novel gene cluster that \nmediates  central  nervous  system  tropism.  This  approach  illustrates \nprogress toward a future in which we can link pathogen identification \nwith risk, thereby informing resource use and allocation.\n\nGLOBAL CONSIDERATIONS\nWhile cutting-edge genomic technologies are largely implemented in \nthe  developed  world,  their  application  to  infectious  diseases  perhaps \noffers the biggest potential impact in less developed regions where the \nburden  of  these  infections  is  greatest.  This  globalization  of  genomic \ntechnology and its extensions has already begun in each of the areas \nof focus highlighted in this chapter; it has occurred both through the \napplication of advanced technologies to samples collected in the devel-\noping world and through the adaptation and importation of technolo-\ngies directly to the developing world for on-site implementation as they \nbecome more globally accessible.\n\nGenomic  characterization  of  the  pathogens  responsible  for  such \nimportant  global  illnesses  such  as  tuberculosis,  malaria,  trypanoso-\nmiasis,  cholera,  and  most  recently  COVID-19,  has  led  to  insights  in \ndiagnosis,  treatment,  and  infection  control.  For  instance,  with  the \nincreasing  burden  of  drug-resistant  tuberculosis  in  the  developing \nworld, a molecular diagnostic test has been developed to detect rifampin- \nresistant  tuberculosis.  The  genetic  basis  for  rifampin  resistance  has \nbeen well defined by targeted sequencing: characteristic mutations in \nthe molecular target of rifampin, RNA polymerase, account for the vast \nmajority of instances of rifampin resistance. At least in areas that can \nafford to implement it, a rapid, automated PCR assay that can detect \nboth M. tuberculosis and a rifampin-resistant allele of RNA polymerase \ndirectly  in  clinical  samples  has  been  implemented  in  parts  of  Africa \n\nHPIM21e_Part05_p941-p1796.indd   972\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "973",
    "text": "and  Asia,  transforming  the  recognition  and  management  of  incident \ntuberculosis and multidrug resistance where they are most prevalent. \nSince  rifampin  resistance  frequently  accompanies  resistance  to  other \nantibiotics, this test can suggest the presence of multidrug-resistant M. \ntuberculosis within hours instead of weeks, without the infrastructure \nrequired for culture.\n\nHigh-resolution genomic tracking of the spread of epidemics\u2014from \ncholera to Ebola to Zika to COVID-19\u2014has yielded insights into which \npublic  health  measures  may  prove  most  effective  in  controlling  local \nepidemics. Many genomic tracking efforts have involved close collabo-\nrations with local scientists and public health officials, and considerable \ninvestment  in  sequencing  infrastructure  in  sub-Saharan  Africa  has \nmade on-location epidemic tracking in the event of another such out-\nbreak feasible. Such investment can not only enable real-time outbreak \nrecognition and tracking but also provide the infrastructure needed to \ncapitalize on the many other benefits of high-throughput sequencing \nas they are developed. The early returns of such investments are exem-\nplified by the rapid reporting of genome sequences for SARS-CoV-2, \nwith 55,000 viral genome sequences reported within the first 6 months \nof the pandemic. Overall, sequencing efforts have become cheaper and \nhave  moved  closer  to  point-of-care  with  each  passing  year.  As  these \ntechnologies synergize with efforts to globalize information-technology \nresources,  global  implementation  of  genomic  methods  promises  to \nspread state-of-the-art methods for diagnosis, treatment, and epidemic \ntracking of infections to areas that need these capabilities the most.\n\nGENOMICS AND THE COVID-19 PANDEMIC\nThe COVID-19 pandemic, which began in 2019 and spread worldwide \nin  2020,  resulted  in  hundreds  of  millions  of  documented  infections \nand millions of deaths and serves as a prime example of the pandemic \npotential  of  infectious  pathogens.  It  also  demonstrated  the  central \nrole that genomic tools now play in response to infectious outbreaks, \nranging from enabling diagnostics and vaccines to tracking evolution, \nvirulence, and transmissibility of the pathogen. The rapid discovery of \nSARS-CoV-2 and sequencing of its genome was complete within weeks \nof the recognition of the clinical syndrome. The rapid public sharing \nof  this  genome  sequence  led  directly  to  two  key  interventions:  diag-\nnostic assay development via RT-qPCR and vaccine design. Crucially, \nvaccine development was informed by homology of the SARS-CoV-2 \nsequence  to  SARS  and  MERS  coronaviruses.  The  dominant  antigen \nof  those  viruses,  the  surface  protein  Spike,  was  well  characterized, \nenabling  the  design  of  the  first  SARS-CoV-2  vaccines  to  begin  the \nday after the genome sequence was shared. The progress of the most \nrapidly developed and validated vaccine in human history was unques-\ntionably  accelerated  by  genomic  technology.  WGS  has  also  played  a \nlarge  role  in  outbreak  tracking  and  confirmation  of  case  clusters  in \ninstitutional settings such as hospitals or congregate living facilities, in \nhelping  to  distinguish  reinfections  from  recrudescence  or  prolonged \nviral shedding, in monitoring spread through societies, and in tracking \npathogen evolution, including the emergence of new variants of con-\ncern with altered transmissibility, severity, and/or partial evasion of the \nimmune  response  generated  to  prior  versions  of  the  virus,  vaccines, \nor  monoclonal  antibody  therapeutics.  Finally,  cutting  edge  genomic \nmethods  including  single-cell  transcriptional  profiling  and  genome-\nwide association studies are contributing to our understanding of the \nwide variability in outcomes of SARS-CoV-2 infection, ranging from \nasymptomatic  carriage  to  death.  Overall,  just  as  the  global  response \nto  the  COVID-19  pandemic  underscores  the  indispensable  role  that \ngenomics methods have come to play in the clinical and public health \nmanagement of infectious diseases, the devastating impact of this pan-\ndemic reveals the urgent need for further development and implemen-\ntation of tools for disease surveillance and response.\n\nSUMMARY\nBy  illuminating  the  genetic  information  that  encodes  the  most  fun-\ndamental  processes  of  life,  genomic  technologies  are  transforming \nmany  aspects  of  medicine.  In  infectious  diseases,  methods  such  as \nnext-generation  sequencing  and  genome-scale  expression  analysis \n\noffer information of unprecedented depth about individual microbes \nas well as microbial communities. This information is expanding our \nunderstanding of the interactions of microorganisms with each other, \ntheir  human  hosts,  and  the  environment.  Despite  technological  and \nfinancial  barriers  that  continue  to  slow  the  widespread  adoption  of \nlarge-scale pathogen sequencing in clinical and public health settings, \ngenomic  methodologies  have  utterly  transformed  the  research  land-\nscape  in  infectious  disease  and  are  beginning  to  make  meaningful \ninroads  into  clinical  settings.  As  even  vaster  amounts  of  data  are \ngenerated, innovations in data storage, development of bioinformatics \ntools to manipulate the data, standardization of methods, and training \nof end-users in both the research and clinical realms will be required. \nThe cost-effectiveness and applicability of whole genome sequencing, \nparticularly  in  the  clinic,  remain  to  be  studied,  and  studies  of  the \nimpact of genome sequencing on patient outcomes will be needed to \nclarify the contexts in which these new methodologies can make the \ngreatest  contributions  to  patient  well-being.  The  ongoing  efforts  to \novercome limitations through collaboration, teaching, and reduction of \nfinancial obstacles should be applauded and expanded. With advances \nin  genomic  technologies  and  computational  analysis,  our  ability  to \ndetect,  characterize,  treat,  monitor,  prevent,  and  control  infections \nhas advanced rapidly in recent years and will continue to do so, with \nthe hope of heralding a new era where the clinician is better armed to \ncombat infection and promote human health.\n \u25a0 FURTHER READING\nBullman S et al: Emerging concepts and technologies for the discov-\nery of microorganisms involved in human disease. Annu Rev Pathol \n12:217, 2017.\n\nBurnham  CD  et  al:  Diagnosing  antimicrobial  resistance.  Nat  Rev \n\nMicrobiol 15:697, 2017.\n\nCroucher NJ et al: Population genomics of post-vaccine changes in \n\npneumococcal epidemiology. Nat Genet 45:656, 2013.\n\nCryptic  Consortium  et  al:  Prediction  of  susceptibility  to  first-line \ntuberculosis  drugs  by  DNA  sequencing.  N  Engl  J  Med  379:1403, \n2018.\n\nDudas G et al: Virus genomes reveal factors that spread and sustained \n\nthe Ebola epidemic. Nature 544:309, 2017.\n\nGardy JL, Loman NJ: Towards a genomics-informed, real-time, global \n\npathogen surveillance system. Nat Rev Genet 19:9, 2018.\n\nGrubaugh ND et al: Tracking virus outbreaks in the twenty-first cen-\n\ntury. Nat Microbiol 4:10, 2019.\n\nLoman NJ, Pallen MJ: Twenty years of bacterial genome sequencing. \n\nNat Rev Microbiol 13:787, 2015.\n\nMutreja A et al: Evidence for several waves of global transmission in \n\nthe seventh cholera pandemic. Nature 477:462, 2011.\n\nWu Z, McGoogan JM: Characteristics of and important lessons from \nthe coronavirus disease 2019 (COVID-19) outbreak in China. JAMA \n323:1239, 2020.\n\n973\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n2\n\nA\np\np\nr\no\na\nc\nh\nt\no\nt\nh\ne\nA\nc\nu\nt\ne\nl\ny\nI\nl\nl\n\nI\nn\nf\ne\nc\nt\ne\nd\nF\ne\nb\nr\ni\nl\ne\nP\na\nt\ni\ne\nn\nt\n\n122 Approach to the Acutely  \nIll Infected Febrile Patient\nTamar F. Barlam\n\nThe  physician  treating  the  acutely  ill  febrile  patient  must  be  able  to \nrecognize infections that require emergent attention. If such infections \nare  not  adequately  evaluated  and  treated  at  initial  presentation,  the \nopportunity to alter an adverse outcome may be lost. In this chapter, \nthe clinical presentations of and approach to patients with infectious \ndisease emergencies are discussed. These infectious processes and their \ntreatments are discussed in detail in other chapters.\n\nHPIM21e_Part05_p941-p1796.indd   973\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n \n \n \n \n"
  },
  {
    "page": "974",
    "text": "974\n\nAPPROACH TO THE PATIENT\nAcute Febrile Illness\n\nBefore the history is elicited and a physical examination is performed, \nan immediate assessment of the patient\u2019s general appearance can yield \nvaluable information. The perceptive physician\u2019s subjective sense that \na patient is septic or toxic often proves accurate. Visible agitation or \nanxiety in a febrile patient can be a harbinger of critical illness. \nHISTORY\nPresenting  symptoms  are  frequently  nonspecific.  Detailed  ques-\ntions  should  be  asked  about  the  onset  and  duration  of  symptoms \nand  about  changes  in  severity  or  rate  of  progression  over  time. \nHost  factors,  such  as  extremes  of  age,  and  comorbid  conditions \nmay  increase  the  risk  of  infection  with  certain  organisms  or  of  a \nmore fulminant course than is usually seen. Lack of splenic func-\ntion,  alcoholism  with  significant  liver  disease,  IV  drug  use,  HIV \ninfection,  diabetes,  malignancy,  morbid  obesity,  organ  transplan-\ntation, and chemotherapy all predispose to specific infections and \nfrequently to increased severity. The patient should be questioned \nabout factors that might help identify a nidus for invasive infection, \nsuch as recent upper respiratory tract infections, influenza, or vari-\ncella; prior trauma; disruption of cutaneous barriers due to lacera-\ntions, burns, surgery, body piercing, or decubiti; and the presence \nof  foreign  bodies  or  prosthetic  devices.  Travel,  presence  during  a \nnatural disaster such as a hurricane or tsunami, contact with pets \nor other animals, or activities that might result in tick or mosquito \nexposure can lead to diagnoses that would not otherwise be con-\nsidered.  Recent  dietary  intake,  medication  use,  social  or  occupa-\ntional contact with ill individuals, vaccination history, recent sexual \ncontacts, and menstrual history may be relevant. Pregnancy might \nincrease  the  risk  and  severity  of  some  illnesses,  such  as  influenza \nor COVID-19, or increase the risk of significant morbidity for the \nfetus, as in Listeria or Zika virus infection. A detailed review of sys-\ntems should include any neurologic signs or sensorium alterations, \nrashes or skin lesions, and focal pain or tenderness. \nPHYSICAL EXAMINATION\nA complete physical examination should be performed, with special \nattention  to  several  areas  that  are  sometimes  given  short  shrift  in \nroutine  examinations  such  as  assessment  of  the  patient\u2019s  general \nappearance and a detailed skin, soft tissue and neurologic evaluation.\nThe  patient  may  appear  either  anxious  and  agitated  or  lethargic \nand  apathetic.  Fever  is  usually  present,  although  elderly  patients  and \ncompromised  hosts  (e.g.,  patients  who  are  uremic  or  cirrhotic  and \nthose who are taking glucocorticoids or nonsteroidal anti-inflammatory \ndrugs) may be afebrile despite serious underlying infection. Criti-\ncally ill patients may be hypothermic, with a high risk of organ fail-\nure and mortality. Mortality at 30 days has been shown to decrease \nwith increasing temperature at presentation. Measurement of blood \npressure,  heart  rate,  and  respiratory  rate  and  oxygen  saturation \nhelps determine the degree of hemodynamic and metabolic com-\npromise. The patient\u2019s airway must be evaluated to rule out the risk \nof obstruction from an invasive oropharyngeal infection.\n\nThe  etiologic  diagnosis  may  become  evident  in  the  context \nof  a  thorough  skin  examination  (Chap.  19).  Petechial  rashes  are \ntypically seen with meningococcemia or Rocky Mountain spotted \nfever  (RMSF;  see  Fig.  A1-16);  erythroderma  is  associated  with \ntoxic shock syndrome (TSS). On soft tissue and muscle examina-\ntion, areas of erythema or duskiness, edema, and tenderness may \nindicate underlying necrotizing fasciitis, myositis, or myonecrosis. \nThe neurologic examination must include a careful assessment of \nmental status for signs of early encephalopathy. Evidence of nuchal \nrigidity or focal neurologic findings should be sought. \nDIAGNOSTIC WORKUP\nAfter  a  quick  clinical  assessment,  diagnostic  material  should  be \nobtained  rapidly  and  antibiotic  and  supportive  treatment  begun. \nBlood (for cultures; baseline complete blood count with differential; \n\nmeasurement  of  serum  electrolytes,  blood  urea  nitrogen,  serum \ncreatinine, and serum glucose; liver function tests and serum lac-\ntate;  C-reactive  protein,  lactate  dehydrogenase,  and  d-dimer)  can \nbe obtained at the time an IV line is placed and before antibiotics \nare administered. Three sets of blood cultures should be performed \nfor  patients  with  possible  acute  endocarditis.  Blood  smears  from \npatients at risk for severe parasitic disease, such as malaria or babe-\nsiosis (Chaps. 224, 225, and A2), must be examined for the diag-\nnosis  and  quantitation  of  parasitemia.  Blood  smears  may  also  be \ndiagnostic in ehrlichiosis and anaplasmosis. Testing of a nasophar-\nyngeal sample for possible COVID-19 may be indicated.\n\nPatients with possible meningitis should have cerebrospinal fluid \n(CSF) drawn before the initiation of antibiotic therapy. Focal findings, \ndepressed mental status, or papilledema should be evaluated by brain \nimaging prior to lumbar puncture, which, in this setting, could initiate \nherniation. Antibiotics should be administered before imaging but after \nblood for cultures has been drawn. If CSF cultures are negative, blood \ncultures will provide the diagnosis in 50\u201370% of cases. Molecular diag-\nnostic techniques (e.g., broad-range 16S rRNA gene polymerase chain \nreaction testing for bacterial meningitis pathogens) are of increasing \nimportance in the rapid diagnosis of life-threatening infections.\n\nFocal abscesses necessitate immediate CT or MRI as part of an \nevaluation  for  surgical  intervention.  Other  diagnostic  procedures, \nsuch as wound cultures, should not delay the initiation of treatment \nfor more than minutes. Once emergent evaluation, diagnostic pro-\ncedures, and (if appropriate) surgical consultation (see below) have \nbeen completed, other laboratory tests can be conducted. Appropri-\nate radiography, CT and/or MRI imaging, urinalysis, measurement \nof  the  erythrocyte  sedimentation  rate,  C-reactive  protein,  and/or \nprocalcitonin, and transthoracic or transesophageal echocardiogra-\nphy all may prove important.\n\nTREATMENT\nThe Acutely Ill Patient\n\nIn the acutely ill patient, empirical antibiotic therapy for presumed \nbacterial or fungal infection is critical and should be administered \nwithout undue delay in addition to fluid resuscitation and vasopres-\nsor support as needed. Increased prevalence of antibiotic resistance \nin community-acquired bacteria must be considered when antibi-\notics are selected. Table 122-1 lists first-line empirical regimens for \ninfections considered in this chapter. In addition to the rapid initia-\ntion of antibiotic therapy, several of these infections require urgent \nsurgical attention. Neurosurgical evaluation for subdural empyema, \notolaryngologic surgery for possible mucormycosis, and cardiotho-\nracic surgery for critically ill patients with acute endocarditis are as \nimportant as antibiotic therapy. For infections such as necrotizing \nfasciitis  and  clostridial  myonecrosis,  rapid  surgical  intervention \nsupersedes other diagnostic or therapeutic maneuvers.\n\nAdjunctive treatments may reduce morbidity and mortality rates \nand include dexamethasone for bacterial meningitis or IV immu-\nnoglobulin for TSS. Adjunctive therapies should usually be initiated \nwithin  the  first  hours  of  treatment;  however,  dexamethasone  for \nbacterial meningitis must be given before or at the time of the first \ndose of antibiotic. Glucocorticoids may also be harmful\u2014e.g., when \ngiven in the setting of cerebral malaria or viral hepatitis.\n\nSPECIFIC PRESENTATIONS\nThe  infections  considered  below  according  to  common  clinical  pre-\nsentation can have rapidly catastrophic outcomes, and their immediate \nrecognition and treatment can be life-saving. Recommended empirical \ntherapeutic regimens are presented in Table 122-1.\n \u25a0 SEPSIS WITHOUT AN OBVIOUS FOCUS OF \nPRIMARY INFECTION\nPatients  initially  have  a  brief  prodrome  of  nonspecific  symptoms \nand  signs  that  progresses  quickly  to  hemodynamic  instability  with \n\nHPIM21e_Part05_p941-p1796.indd   974\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "975",
    "text": "975\n\nSEE CHAP(S).\n\n147, 148, 161, \n164, 304\n\nTABLE 122-1  Empirical Treatment for Common Infectious Disease Emergenciesa\nPOSSIBLE ETIOLOGIES TREATMENT\nCLINICAL SYNDROME\n\nCOMMENTS\n\nSepsis without a Clear Focus\n\nSeptic shock\n\nOverwhelming post-\nsplenectomy sepsis\n\nBabesiosis\n\nPseudomonas spp., \ngram-negative enteric \nbacilli, Staphylococcus \nspp., Streptococcus \nspp.\n\nStreptococcus \npneumoniae, \nHaemophilus \ninfluenzae, Neisseria \nmeningitidis\nBabesia microti (U.S.), \nB. divergens (Europe)\n\nVancomycin (15 mg/kg q12h)b\nplus either\nPiperacillin/tazobactam (4.5 \ng) q8h via extended infusion \n(EI)c or cefepime (2 g) q8h \nvia EI\nCeftriaxone (2 g q12h) plus \nvancomycin  \n(15 mg/kg q12h)b\n\nAtovaquone (750 mg q12h) \nplus azithromycin (500 mg \nq24h)\n\nSepsis with Skin Findings\n\nMeningococcemia\n\nN. meningitidis\n\nCeftriaxone (2 g q12h) or \npenicillin (4 mU q4h)\n\nRickettsia rickettsii\n\nDoxycycline (100 mg bid)\n\nEmpirical therapy should be tailored to local resistance patterns. \nCarbapenem or aminoglycoside antibiotics should be considered \nfor empirical therapy when rates of multidrug-resistant gram-\nnegative organisms are high or for patients with risk factors for \nresistant organisms. Adjust treatment when culture data become \navailable.\n\nIf a \u03b2-lactam\u2013sensitive strain is identified, vancomycin can be \ndiscontinued and a narrower-spectrum agent, such as penicillin, \nconsidered based on susceptibility testing.\n\n304\n\nClindamycin (600 mg q8h) plus quinine (650 mg q8h) can be \nused in severe disease not responding to atovaquone and \nazithromycin. Treatment with doxycycline (100 mg bid) for \npotential co-infection with Borrelia burgdorferi or Anaplasma \nspp. may be prudent.\n\n222, 225\n\nRocky Mountain \nspotted fever (RMSF)\n\nPurpura fulminans\n\nErythroderma: toxic \nshock syndrome\n\nS. pneumoniae, \nH. influenzae, N. \nmeningitidis\nGroup A \nStreptococcus, \nStaphylococcus \naureus\n\nCeftriaxone eradicates nasopharyngeal carriage of the organism. \nClose contacts require chemoprophylaxis with rifampin (600 mg \nq12h for 2 days) or ciprofloxacin (a single dose, 500 mg).\nIf both meningococcemia and RMSF are being considered, use \nceftriaxone (2 g q12h) plus doxycycline (100 mg bid). If RMSF is \ndiagnosed, doxycycline is the proven superior agent.\nIf a \u03b2-lactam-sensitive strain is identified, vancomycin can be \ndiscontinued.\n\nCeftriaxone (2 g q12h) plus \nvancomycin (15 mg/kg q12h)b\n\nVancomycin (15 mg/kg q12h)b \nplus clindamycin (600 mg \nq8h)\n\nIf a penicillin- or oxacillin-sensitive strain is isolated, these \nagents are superior to vancomycin (penicillin, 2 mU q4h; or \noxacillin, 2 g IV q4h). The site of toxigenic bacteria should be \ndebrided; IV immunoglobulin can be used in severe cases.d\n\nSepsis with Soft Tissue Findings\n\nNecrotizing fasciitis\n\nGroup A \nStreptococcus, mixed \naerobic/anaerobic \nflora, CA-MRSAe\n\nClostridial myonecrosis Clostridium \nperfringens\n\nVancomycin (15 mg/kg q12h)b \nplus piperacillin/tazobactam \n(4.5 q q8h via EI)c plus \nclindamycin (600\u00a0mg q8h)\nPenicillin (2 mU q4h) plus \nclindamycin (600 mg q8h)\n\nUrgent surgical evaluation is critical. Empirical therapy should \nbe tailored to local resistance patterns. For mixed aerobic/\nanaerobic infections, clindamycin can be discontinued. Adjust \ntreatment when culture data become available.\nUrgent surgical evaluation is critical.\n\nNeurologic Infections\n\nBacterial meningitis\n\nS. pneumoniae, N. \nmeningitidis\n\nCeftriaxone (2 g q12h) plus \nvancomycin (15 mg/kg q12h)b\n\nBrain abscess, \nsuppurative intracranial \ninfections\n\nCerebral malaria\n\nStreptococcus spp., \nStaphylococcus spp., \nanaerobes, gram-\nnegative bacilli\nPlasmodium \nfalciparum\n\nVancomycin (15 mg/kg q12h)b \nplus metronidazole (500 mg \nq8h) plus ceftriaxone (2 g \nq12h)\nArtesunate (2.4 mg/kg IV at 0, \n12, and 24 h; then once daily)f\n\nSpinal epidural abscess Staphylococcus spp., \ngram-negative bacilli\n\nVancomycin (15 mg/kg q12h)b \nplus either\nPiperacillin/tazobactam (4.5 \ng q8h via EI) or cefepime (2 g \nq8h via EI)c\n\nIf a \u03b2-lactam\u2013sensitive strain is identified, vancomycin can be \ndiscontinued. If the patient is >50 years old or has comorbid \ndisease, add ampicillin (2 g q4h) for Listeria coverage. \nDexamethasone (10 mg q6h for 4 days) started before, or at the \ntime of, the first dose of antibiotic improves outcome in adults \nwith meningitis (especially pneumococcal).\nUrgent surgical evaluation is critical. If a penicillin- or oxacillin-\nsensitive strain is isolated, these agents are superior to \nvancomycin (penicillin, 4 mU q4h; or oxacillin, 2 g q4h).\n\nAvoid glucocorticoids. Until IV artesunate is available, treatment \ncan be initiated with oral artemether-lumefantrine. Atovaquone-\nproguanil, quinine, and mefloquine are other options.\nSurgical evaluation is essential. If a penicillin- or oxacillin-\nsensitive strain is isolated, these agents are superior to \nvancomycin (penicillin, 4 mU q4h; or oxacillin, 2 g q4h).\n\n222, 224\n\n442\n\n155\n\n187\n\n146, 155, 157, \n304\n\n147, 148\n\n129, 147, 148\n\n154\n\n138\n\n138\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n2\n\nA\np\np\nr\no\na\nc\nh\nt\no\nt\nh\ne\nA\nc\nu\nt\ne\nl\ny\nI\nl\nl\n\nI\nn\nf\ne\nc\nt\ne\nd\nF\ne\nb\nr\ni\nl\ne\nP\na\nt\ni\ne\nn\nt\n\nFocal Infections\n\nAcute bacterial \nendocarditis\n\nS. aureus, \u03b2-hemolytic \nstreptococci, HACEK \ngroup,g Neisseria spp., \nS. pneumoniae\n\nCeftriaxone (2 g q12h) or \ncefepime (2 g q8h via EI)c plus \nvancomycin (15 mg/kg q12h)b\n\nAdjust treatment when culture data become available. Surgical \nevaluation is essential.\n\n128\n\naThese empirical regimens include coverage for gram-positive pathogens that are resistant to \u03b2-lactam antibiotics. Local resistance patterns should be considered and \nmay alter the need for empirical vancomycin or for expanded coverage for antibiotic-resistant gram-negative pathogens. bA vancomycin loading dose of 20\u201325 mg/kg can \nbe considered in critically ill patients. Dosing must be adjusted based on pharmacokinetic/pharmacodynamic monitoring. cEI=extended infusion. \u03b2-Lactam antibiotics may \nexhibit unpredictable pharmacodynamics in sepsis. Prolonged or continuous infusions are often used. dThe optimal dose of IV immunoglobulin has not been determined, \nbut the median dose in observational studies is 2 g/kg (total dose administered for 1\u20135 days). eCommunity-acquired methicillin-resistant S. aureus. fIn the United States, \nartesunate must be obtained through the Centers for Disease Control and Prevention. gHaemophilus spp., Aggregatibacter spp., Cardiobacterium hominis, Eikenella \ncorrodens, and Kingella kingae.\n\nHPIM21e_Part05_p941-p1796.indd   975\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n \n \n \n \n"
  },
  {
    "page": "976",
    "text": "976\n\nhypotension,  tachycardia,  tachypnea,  respiratory  distress,  and  altered \nmental  status.  Disseminated  intravascular  coagulation  (DIC)  with \nclinical evidence of a hemorrhagic diathesis is a poor prognostic sign.\n\nSeptic  Shock  (See  also  Chap.  304)  Patients  with  bacteremia \nleading  to  septic  shock  may  have  a  primary  site  of  infection  (e.g., \npneumonia,  pyelonephritis,  or  cholangitis)  that  is  not  evident  ini-\ntially. Elderly patients who may have atypical presentations and often \nhave  comorbid  conditions,  hosts  compromised  by  malignancy  and \nneutropenia,  and  patients  who  have  recently  undergone  a  surgical \nprocedure or hospitalization are at increased risk for an adverse out-\ncome. Gram-negative bacteremia with organisms such as Pseudomonas \naeruginosa or Escherichia coli and gram-positive infection with organ-\nisms such as Staphylococcus aureus (including methicillin-resistant S. \naureus  [MRSA])  or  group  A  streptococci  can  present  as  intractable \nhypotension and multiorgan failure. Treatment can usually be initiated \nempirically on the basis of the presentation, host factors (Chap. 304), \nand  local  patterns  of  bacterial  resistance.  Outcomes  are  worse  when \nantimicrobial treatment is delayed or when the responsible pathogen \nultimately  proves  not  to  be  susceptible  to  the  initial  regimen.  The \nincreasing  prevalence  of  multidrug-resistant  organisms  makes  this \nespecially relevant. Broad-spectrum antimicrobial agents are therefore \nrecommended and should be instituted rapidly, preferably within the \nfirst hours after presentation. Pharmacodynamics are altered in sepsis \ndue  to  increased  volume  of  distribution  and  renal  clearance,  so  it  is \nimportant  to  adequately  dose  antimicrobials.  Risk  factors  for  fungal \ninfection should be assessed, as the incidence of fungal septic shock is \nincreasing. Nonbacterial causes of shock, such as dengue virus infec-\ntion, should be considered in endemic areas as mortality can increase if \nundiagnosed. Biomarkers such as C-reactive protein and procalcitonin \nhave  not  proved  reliable  diagnostically  or  prognostically  but,  when \nmeasured over time, can facilitate appropriate de-escalation of therapy \nwith  improved  outcomes.  Glucocorticoids  are  often  considered  for \npatients with severe sepsis who do not respond to fluid resuscitation \nand  vasopressor  therapy,  but  conclusive  evidence  for  efficacy  in  this \nsetting is lacking.\n\nOverwhelming  Infection  in  Asplenic  Patients  (See  also  \nChap. 304)  Patients without splenic function are at risk for over-\nwhelming  bacterial  sepsis.  Asplenic  adult  patients  succumb  to  sepsis \nat 58 times the rate of the general population. Most infections occur \nwithin the first l or 2 years, but the increased risk persists throughout \nlife. The median interval between splenectomy and sepsis is 4\u20136 years, \nwith  a  range  of  1\u201319  years.  In  asplenia,  encapsulated  bacteria  cause \nthe  majority  of  infections.  Adults,  who  are  more  likely  to  have  anti-\nbody to these organisms, are at lower risk than children. Streptococcus \npneumoniae is the most common isolate, causing 40\u201370% of cases. The \nrisk of infection with Haemophilus influenzae or Neisseria meningitidis \nalso is greater in patients without splenic function, but reported cases \nare declining. Severe clinical manifestations of infections due to other \norganisms,  such  as  E.  coli,  S.  aureus,  Bordetella  holmesii,  Capnocy-\ntophaga, Babesia, and Plasmodium species, have been described.\n\nBabesiosis  (See  also  Chap.  225)  A  history  of  recent  travel  to \nendemic areas raises the possibility of infection with Babesia. Between \n1 and 4 weeks after a tick bite, the patient experiences chills, fatigue, \nanorexia,  myalgia,  arthralgia,  shortness  of  breath,  nausea,  and  head-\nache; ecchymosis and/or petechiae are occasionally seen. The tick that \nmost  commonly  transmits  Babesia,  Ixodes  scapularis,  also  transmits \nBorrelia  burgdorferi  (the  agent  of  Lyme  disease)  and  Anaplasma;  co- \ninfection can occur, and may result in more severe disease. Infection \nwith  the  European  species  Babesia  divergens  is  more  frequently  ful-\nminant than that due to the U.S. species Babesia microti. B. divergens \ncauses a febrile syndrome with hemolysis, jaundice, hemoglobinemia, \nand renal failure and is associated with a mortality rate of >40%. Severe \nbabesiosis  is  especially  common  in  asplenic  hosts  but  does  occur  in \nhosts with normal splenic function, particularly those >60 years of age \nand those with underlying immunosuppressive conditions such as HIV \ninfection  or  malignancy.  Complications  include  renal  failure,  acute \nrespiratory failure, DIC, and splenic rupture.\n\nOther  Sepsis  Syndromes  Tularemia  (Chap.  170)  has  been \nreported  in  every  U.S.  state  except  Hawaii.  This  disease  is  associated \nwith  wild  rabbit,  tick,  horse-fly,  and  tabanid  fly  contact.  It  can  be \ntransmitted by arthropod bite, handling of infected animal carcasses, \nconsumption  of  contaminated  food  and  water,  or  inhalation.  The \ntyphoidal form can be associated with gram-negative septic shock and \na mortality rate of >30%, especially in patients with underlying comor-\nbid  or  immunosuppressive  conditions.  Plague  occurs  infrequently  in \nthe  United  States  (Chap.  171),  primarily  after  contact  with  ground \nsquirrels, prairie dogs, or chipmunks, but is endemic in other parts of \nthe world; >90% of all cases occur in Africa with Madagascar especially \naffected.  The  septic  form  is  particularly  rare  and  is  associated  with \nshock, multiorgan failure, and a 30% mortality rate. Pneumonic plague \nis  rapidly  progressive  and  fatal  without  treatment.  These  infections \nshould  be  considered  in  the  appropriate  epidemiologic  setting.  The \nCenters  for  Disease  Control  and  Prevention  (CDC)  lists  Francisella \ntularensis  and  Yersinia  pestis  (the  agents  of  tularemia  and  plague, \nrespectively)  along  with  Bacillus  anthracis  (the  agent  of  anthrax)  as \nimportant organisms that might be used for bioterrorism (Chap. S3).\n\n \u25a0 SEPSIS WITH SKIN MANIFESTATIONS  \n(SEE ALSO CHAP. 19)\nMaculopapular  rashes  may  reflect  early  meningococcal  or  rickettsial \ndisease but are usually associated with nonemergent infections. Exan-\nthems  are  usually  viral.  Primary  HIV  infection  commonly  presents \nwith a rash that is typically maculopapular and involves the upper part \nof the body but can spread to the palms and soles. The patient is usually \nfebrile  and  can  have  lymphadenopathy,  severe  headache,  dysphagia, \ndiarrhea,  myalgias,  and  arthralgias.  Recognition  of  this  syndrome \nprovides an opportunity to prevent transmission and to institute early \ntreatment.\n\nPetechial  rashes  caused  by  viruses  are  seldom  associated  with \nhypotension or a toxic appearance, although there can be exceptions \n(e.g.,  severe  measles  or  arboviral  infection).  Petechial  rashes  limited \nto the distribution of the superior vena cava are rarely associated with \nsevere disease. In other settings, petechial rashes require more urgent \nattention.\n\nMeningococcemia (See also Chap. 155)  Almost three-quarters \nof  patients  with  N.  meningitidis  bacteremia  have  a  rash.  Meningo-\ncoccemia  most  often  affects  young  children  (i.e.,  those  6  months  to  \n5 years old). In sub-Saharan Africa, the high prevalence of serogroup \nA meningococcal disease has been a threat to public health for more \nthan a century. Thousands of deaths occur annually in this area, which \nis  known  as  the  \u201cmeningitis  belt,\u201d  and  large  epidemic  waves  occur \napproximately  every  8\u201312  years.  Serogroups  W135,  X,  and  C  also \nare  important  emerging  pathogens  in  Africa.  For  example,  Nigeria \nexperienced a large serogroup C outbreak in 2016\u20132017 in the setting \nof  aggressive  vaccination  programs  against  serogroup  A.  Outside \nAfrica,  outbreaks  for  the  past  50  years  reported  in  the  United  States \nand Europe are caused primarily by serogroup C (approximately 60%) \nfollowed  by  serogroup  B  (29%).  In  the  United  States,  sporadic  cases \nand outbreaks occur in day-care centers, schools (grade school through \ncollege, particularly among college freshmen living in residential halls), \nand army barracks. Household contacts of index cases are at 400\u2013800 \ntimes  greater  risk  of  disease  than  the  general  population.  Patients \nmay  have  fever,  headache,  nausea,  vomiting,  myalgias,  changes  in \nmental  status,  and  meningismus.  However,  the  rapidly  progressive \nform of disease is not usually associated with meningitis. The rash is \ninitially pink, blanching, and maculopapular, appearing on the trunk \nand  extremities,  but  then  becomes  hemorrhagic,  forming  petechiae. \nPetechiae are first seen at the ankles, wrists, axillae, mucosal surfaces, \nand  palpebral  and  bulbar  conjunctiva,  with  subsequent  spread  on \nthe lower extremities and to the trunk. A cluster of petechiae may be \nseen  at  pressure  points\u2014e.g.,  where  a  blood  pressure  cuff  has  been \ninflated.  In  rapidly  progressive  meningococcemia  (10\u201320%  of  cases), \nthe  petechial  rash  quickly  becomes  purpuric  (see  Fig.  A1-41),  and \npatients develop DIC, multiorgan failure, and shock; 50\u201360% of these \npatients  die,  and  survivors  often  require  extensive  debridement  or \n\nHPIM21e_Part05_p941-p1796.indd   976\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "977",
    "text": "amputation of gangrenous extremities. Hypotension with petechiae for \n<12 h is associated with significant mortality. Cyanosis, coma, oliguria, \nmetabolic acidosis, and elevated partial thromboplastin time also are \nassociated with a fatal outcome. Antibiotics given in the office by the \nprimary care provider before hospital evaluation and admission may \nimprove  prognosis;  this  observation  suggests  that  early  initiation  of \ntreatment  may  be  life-saving.  Meningococcal  conjugate  vaccines  are \nprotective against serogroups A, C, Y and W135 and are recommended \nfor children 11\u201312 years of age with a booster dose at 16 years of age, \nand  for  other  high-risk  patients.  Vaccines  active  against  serogroup  B \nare recommended for high-risk individuals \u226510 years of age and may \nbe appropriate for teens and young adults (16 through 23 years of age).\n\nRocky  Mountain  Spotted  Fever  and  Other  Rickettsial  \nDiseases (See also Chap. 187)  RMSF is a tickborne disease that \noccurs  throughout  North  and  South  America.  It  is  caused  primarily \nby  Rickettsia  rickettsii  but  can  be  caused  by  other  rickettsiae  (e.g., \nR.\u00a0parkeri, R. akari). Up to 40% of patients do not report a history of \na  tick  bite,  but  a  history  of  travel  or  outdoor  activity  (e.g.,  camping \nin  tick-infested  areas)  can  often  be  ascertained.  For  the  first  3  days, \nheadache, fever, malaise, myalgias, nausea, vomiting, and anorexia are \ndocumented. By day 3, half of patients have skin findings. Blanching \nmacules develop initially on the wrists and ankles and then spread over \nthe legs and trunk. The lesions become hemorrhagic and are frequently \npetechial. The rash spreads to palms and soles later in the course. The \ncentripetal spread is a classic feature of RMSF but occurs in a minority \nof  patients.  Moreover,  10\u201315%  of  patients  with  RMSF  never  develop \na  rash.  The  patient  can  be  hypotensive  and  develop  noncardiogenic \npulmonary edema, confusion, lethargy, and encephalitis progressing to \ncoma. The CSF contains 10\u2013100 cells/\u03bcL, usually with a predominance \nof mononuclear cells. The CSF glucose level is often normal; the pro-\ntein concentration may be slightly elevated. Renal and hepatic injury \nas well as bleeding secondary to vascular damage are noted. Delayed \nrecognition and treatment are associated with a greater risk of death; \nmortality  rates  are  20\u201330%  if  untreated.  Native  Americans,  Alaskan \nnatives,  Pacific  Islanders,  children  5\u20139  years  of  age,  adults  >70  years \nold, and persons with underlying immunosuppression are at increased \nrisk of death as well.\n\nOther rickettsial diseases cause significant morbidity and mortality \nworldwide. Mediterranean spotted fever caused by Rickettsia conorii is \nfound  in  Africa,  southwestern  and  south-central  Asia,  and  southern \nEurope.  Patients  have  fever,  flu-like  symptoms,  and  an  inoculation \neschar at the site of the tick bite. A maculopapular rash develops within \n1\u20137  days,  involving  the  palms  and  soles  but  sparing  the  face.  Elderly \npatients  or  those  with  diabetes,  alcoholism,  uremia,  or  congestive \nheart failure are at risk for severe disease characterized by neurologic \ninvolvement, respiratory distress, and gangrene of the digits or purpura \nfulminans. Mortality rates associated with this severe form of disease \napproach  50%.  Epidemic  typhus,  caused  by  Rickettsia  prowazekii,  is \ntransmitted in louse-infested environments and emerges in conditions \nof extreme poverty, war, and natural disaster. Patients experience a sud-\nden onset of high fevers, severe headache, cough, myalgias, and abdom-\ninal pain. A maculopapular rash develops (primarily on the trunk) in \nmore than half of patients and can progress to petechiae and purpura. \nSerious  signs  include  delirium,  coma,  seizures,  noncardiogenic  pul-\nmonary  edema,  skin  necrosis,  and  peripheral  gangrene.  Mortality \nrates approached 60% in the preantibiotic era and continue to exceed \n10\u201315% in contemporary outbreaks. Scrub typhus, caused by Orientia \ntsutsugamushi (a separate genus in the family Rickettsiaceae), is trans-\nmitted  by  larval  mites  or  chiggers  and  is  one  of  the  most  common \ninfections in southeastern Asia and the western Pacific. The organism \nis  found  in  areas  of  heavy  scrub  vegetation  (e.g.,  along  riverbanks). \nPatients may have an inoculation eschar and may develop a maculopap-\nular  rash,  lymphadenopathy,  and  dyspnea.  Severe  cases  progress  to \npneumonia, meningoencephalitis, myocarditis, DIC, and renal failure. \nMortality rates range from 1% to 70% and vary by location, increasing \nage, myocarditis, delirium, pneumonitis, or signs of hemorrhage.\n\nIf recognized in a timely fashion, rickettsial disease is very respon-\nsive  to  treatment.  Doxycycline  (100  mg  twice  daily  for  3\u201314  days)  is \n\nthe treatment of choice for both adults and children. Mortality rates are \nhigher when tetracycline-based treatment is not given.\n\n977\n\nPurpura Fulminans (See also Chaps. 155 and 304)  Purpura \nfulminans is the cutaneous manifestation of DIC and presents as large \necchymotic areas and hemorrhagic bullae. Progression of petechiae to \npurpura, ecchymoses, and gangrene is associated with congestive heart \nfailure,  septic  shock,  acute  renal  failure,  acidosis,  hypoxia,  hypoten-\nsion, and death. Purpura fulminans has been associated primarily with \nN.\u00a0meningitidis but, in splenectomized patients, may be associated with \nS. pneumoniae, H. influenzae, and S. aureus.\n\nEcthyma  Gangrenosum  Septic  shock  caused  by  P.  aeruginosa \nor,  less  often,  Aeromonas  hydrophila  or  other  gram-negative  organ-\nisms,  can  be  associated  with  ecthyma  gangrenosum  (see  Figs.  164-1 \nand  A1-34):  hemorrhagic  vesicles  surrounded  by  a  rim  of  erythema \nwith central necrosis and ulceration. Echthyma gangrenosum is most \ncommon  among  patients  with  neutropenia,  extensive  burns,  and \nhypogammaglobulinemia.\n\nOther  Infections  Associated  with  Rash  Vibrio  vulnificus  and \nother noncholera Vibrio bacteremic infections (Chap. 168) can cause \nfocal skin lesions and overwhelming sepsis in hosts with chronic liver \ndisease,  heavy  alcohol  consumption,  iron  storage  disorders,  diabe-\ntes,  renal  insufficiency,  hematologic  disease,  or  malignancy  or  other \nimmunocompromising  conditions.  Over  95%  of  the  cases  are  in  the \nsubtropical Pacific and Atlantic Oceans coastal regions in the Northern \nHemisphere.  After  ingestion  of  contaminated  raw  shellfish  (typically \noysters from the Gulf Coast in U.S. cases), there is a sudden onset of \nmalaise, chills, fever, and hypotension. The patient develops bullous or \nhemorrhagic  skin  lesions,  usually  on  the  lower  extremities,  and  75% \nof patients have leg pain. The mortality rate can be as high as 35\u201360%, \nparticularly  when  the  patient  presents  with  hypotension.  Outcomes \nare  improved  when  patients  are  treated  with  fluoroquinolones  with \nor  without  cephalosporins  or  with  tetracycline-containing  regimens. \nOther infections, caused by agents such as Aeromonas, Klebsiella, and \nE. coli, can cause hemorrhagic bullae and death due to overwhelming \nsepsis in cirrhotic patients. Capnocytophaga canimorsus can cause sep-\ntic  shock  in  asplenic  or  cirrhotic  patients.  Infection  typically  follows \na  dog  bite.  Serovars  A\u2013C  appear  more  virulent,  constituting  92%  of \nhuman  infections  but  only  7.6%  of  canine  isolates.  Patients  present \nwith fever, chills, myalgia, vomiting, diarrhea, dyspnea, confusion, and \nheadache. Findings can include an exanthem or erythema multiforme \n(see Figs. 56-9 and A1-24), cyanotic mottling or peripheral cyanosis, \npetechiae, and ecchymosis. About 30% of patients with this fulminant \nform die of overwhelming sepsis and DIC, and survivors may require \namputation because of gangrene.\n\nErythroderma  TSS  (Chaps.  147  and  148)  is  usually  associated \nwith erythroderma. The patient presents with fever, malaise, myalgias, \nnausea,  vomiting,  diarrhea,  and  confusion.  There  is  a  sunburn-type \nrash that may be subtle and patchy but is usually diffuse and is found \non the face, trunk, and extremities. Erythroderma, which desquamates \nafter  1\u20132  weeks,  is  more  common  in  Staphylococcus-associated  than \nin Streptococcus-associated TSS. Hypotension develops rapidly\u2014often \nwithin hours\u2014after the onset of symptoms. Multiorgan failure occurs. \nEarly  renal  failure  may  precede  hypotension  and  distinguishes  this \nsyndrome from other septic shock syndromes. There may be no indi-\ncation  of  a  primary  focal  infection,  although  possible  cutaneous  or \nmucosal  portals  of  entry  for  the  organism  can  be  ascertained  when \na  careful  history  is  taken.  Colonization  rather  than  overt  infection \nof  the  vagina  or  a  postoperative  wound,  for  example,  is  typical  with \nstaphylococcal  TSS,  and  the  mucosal  areas  appear  hyperemic  but \nnot infected. Streptococcal TSS is more often associated with skin or \nsoft  tissue  infection  (including  necrotizing  fasciitis),  and  patients  are \nmore  likely  to  be  bacteremic.  TSS  caused  by  Clostridium  sordellii  is \nassociated  with  childbirth  or  with  skin  injection  of  black-tar  heroin. \nThe diagnosis of TSS is defined by the clinical criteria of fever, rash, \nhypotension, and multiorgan involvement. (Of note, fever is typically \nabsent when TSS is caused by C. sordellii.) The mortality rate is 5% for \n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n2\n\nA\np\np\nr\no\na\nc\nh\nt\no\nt\nh\ne\nA\nc\nu\nt\ne\nl\ny\nI\nl\nl\n\nI\nn\nf\ne\nc\nt\ne\nd\nF\ne\nb\nr\ni\nl\ne\nP\na\nt\ni\ne\nn\nt\n\nHPIM21e_Part05_p941-p1796.indd   977\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n \n \n \n \n"
  },
  {
    "page": "978",
    "text": "978 menstruation-associated TSS, 10\u201315% for nonmenstrual TSS, 30\u201370% \nfor streptococcal TSS, and up to 90% for obstetric C. sordellii TSS. Clin-\ndamycin improves outcomes when included in the treatment regimen. \nSome studies have shown that use of IV immunoglobulin is associated \nwith improved survival as well.\n\nViral Hemorrhagic Fevers  Viral hemorrhagic fevers (Chaps. 209 \nand 210) are zoonotic illnesses caused by viruses that reside in either \nanimal  reservoirs  or  arthropod  vectors.  These  diseases  occur  world-\nwide and are restricted to areas where the host species live. They are \ncaused by four major groups of viruses: Arenaviridae (e.g., Lassa fever \nin  Africa),  Bunyaviridae  (e.g.,  Rift  Valley  fever  in  Africa;  hantavirus \nhemorrhagic fever with renal syndrome in Asia; and Crimean-Congo \nhemorrhagic  fever,  which  has  an  extensive  geographic  distribution), \nFiloviridae  (e.g.,  Ebola  and  Marburg  virus  infections  in  Africa),  and \nFlaviviridae (e.g., yellow fever in Africa and South America and dengue \nin Asia, Africa, and the Americas). Lassa fever and Ebola and Marburg \nvirus infections are also transmitted from person to person. The vec-\ntors for most viral fevers are found in rural areas; dengue and yellow \nfever  are  important  exceptions.  After  a  prodrome  of  fever,  myalgias, \nand malaise, patients develop evidence of vascular damage, petechiae, \nand  local  hemorrhage.  Shock,  multifocal  hemorrhaging,  and  neuro-\nlogic signs (e.g., seizures or coma) predict a poor prognosis. Dengue \n(Chap. 209) is the most common arboviral disease worldwide. More \nthan half a million cases of dengue hemorrhagic fever occur each year, \nwith at least 12,000 deaths. Patients have a triad of symptoms: hemor-\nrhagic manifestations, evidence of plasma leakage, and platelet counts \nof <100,000/\u03bcL. Mortality rates are 10\u201320%. If dengue shock syndrome \ndevelops,  mortality  rates  can  reach  40%.  Ebola  infection  has  been \nassociated with outbreaks with high mortality rates. The 2014 outbreak \nin  West  Africa  had  a  mortality  rate  of  >50%.  Symptoms  can  appear  \n2\u201321 days after exposure, but most patients become ill within 9 days. \nThe  patient  first  presents  with  fatigue,  fever,  headache,  and  muscle \npains,  and  the  illness  can  progress  to  multiorgan  failure  and  hem-\norrhaging.  Careful  volume-replacement  therapy  to  maintain  blood \npressure and intravascular volume is key to survival in these infections. \nRibavirin also may be useful against Arenaviridae and Bunyaviridae.\n\nOther  viral  illnesses  with  rash,  such  as  measles,  can  be  associated \nwith significant mortality rates. Measles continues to be responsible for \nmore than 100,000 deaths per year, worldwide, and to cause outbreaks \nin populations with low vaccination rates.\n \u25a0 SEPSIS WITH A SOFT TISSUE/MUSCLE  \nPRIMARY FOCUS\nSee also Chap. 129.\n\nNecrotizing Fasciitis  This infection is characterized by extensive \nnecrosis of the subcutaneous tissue and fascia. It may arise at a site of \nminimal trauma or surgical incision and may also be associated with \nrecent varicella, childbirth, or muscle strain. The most common causes \nof necrotizing fasciitis are group A streptococci alone (Chap. 148) and \na  mixed  facultative  and  anaerobic  flora  (Chap.  129);  the  incidence \nof group A streptococcal necrotizing fasciitis has been increasing for \nthe past quarter-century. Diabetes mellitus, IV drug use, chronic liver \nor  renal  disease,  and  malignancy  are  associated  risk  factors.  Physical \nfindings are initially minimal compared with the severity of pain and \nthe degree of fever. The examination is often unremarkable except for \nsoft  tissue  edema  and  erythema.  The  infected  area  is  red,  hot,  shiny, \nswollen,  and  exquisitely  tender.  In  untreated  infection,  the  overlying \nskin develops blue-gray patches after 36 h, and cutaneous bullae and \nnecrosis  develop  after  3\u20135  days.  Necrotizing  fasciitis  due  to  a  mixed \nflora,  but  not  that  due  to  group  A  streptococci,  can  be  associated \nwith  gas  production.  Without  treatment,  pain  decreases  because  of \nthrombosis of the small blood vessels and destruction of the peripheral \nnerves\u2014an ominous sign. The mortality rate is 15\u201334% overall, >70% \nin  association  with  TSS,  and  nearly  100%  without  surgical  interven-\ntion. With surgery, outcomes are significantly better; e.g., mortality is \nreduced to <40% in the setting of TSS. Necrotizing fasciitis may also \nbe due to Clostridium perfringens (Chap. 154); in this condition, the \npatient is extremely toxic and the mortality rate is high. Within 48 h, \n\nrapid  tissue  invasion  and  systemic  toxicity  associated  with  hemolysis \nand  death  ensue.  The  distinction  between  this  entity  and  clostridial \nmyonecrosis is made by muscle biopsy. Necrotizing fasciitis caused by \ncommunity-acquired MRSA also has been reported.\n\nClostridial Myonecrosis (See also Chap. 154)  Myonecrosis is \noften associated with trauma or surgery but can develop spontaneously. \nThe incubation period is usually 12\u201324 h long, and massive necrotizing \ngangrene develops within hours of onset. Systemic toxicity, shock, and \ndeath can occur within 12 h. The patient\u2019s pain and toxic appearance \nare out of proportion to physical findings. On examination, the patient \nis  febrile,  apathetic,  tachycardic,  and  tachypneic  and  may  express  a \nfeeling  of  impending  doom.  Hypotension  and  renal  failure  develop \nlater,  and  hyperalertness  is  evident  preterminally.  The  skin  over  the \naffected area is bronze-brown, mottled, and edematous. Bullous lesions \nwith serosanguineous drainage and a mousy or sweet odor can develop. \nCrepitus can occur secondary to gas production in muscle tissue. The \nmortality  rate  is  >65%  for  spontaneous  myonecrosis,  which  is  often \nassociated  with  Clostridium  septicum  or  C.  tertium  and  underlying \nmalignancy. The mortality rates associated with trunk and limb infec-\ntion are 63% and 12%, respectively, and any delay in surgical treatment \nincreases the risk of death.\n\n \u25a0 NEUROLOGIC INFECTIONS WITH OR WITHOUT \nSEPTIC SHOCK\nBacterial Meningitis (See also Chap. 138)  Bacterial meningi-\ntis is one of the most common infectious disease emergencies involv-\ning  the  central  nervous  system.  Although  hosts  with  cell-mediated \nimmune deficiency (including transplant recipients, diabetic patients, \nelderly patients, and cancer patients receiving certain chemotherapeu-\ntic agents) are at particular risk for Listeria monocytogenes meningitis, \nmost cases in adults are due to S. pneumoniae (30\u201360%) and N. men-\ningitidis (10\u201335%). The classic presentation of fever, meningismus, and \naltered mental status is seen in only one-half to two-thirds of patients \nand less than one-half of adults. The elderly can present without fever \nor  meningeal  signs.  Cerebral  dysfunction  is  evidenced  by  confusion, \ndelirium, and lethargy that can progress to coma. In some cases, the \npresentation  is  fulminant,  with  sepsis  and  brain  edema;  papilledema \nat  presentation  is  unusual  and  suggests  another  diagnosis  (e.g.,  an \nintracranial lesion). Focal signs, including cranial nerve palsies (IV, VI, \nVII), can be seen in 10\u201320% of cases; 50\u201370% of patients have bacte-\nremia. A poor outcome is associated with coma, seizures, hypotension, \na  pneumococcal  etiology,  respiratory  distress,  a  CSF  glucose  level  of \n<0.6 mmol/L (<10 mg/dL), a CSF protein level of >2.5 g/L, a periph-\neral white blood cell count of <5000/\u03bcL, and a serum sodium level of \n<135 mmol/L. Rapid initiation of treatment is essential; the odds of an \nunfavorable outcome may increase by 30% for each hour that treatment \nis  delayed.  Dexamethasone  is  an  adjunctive  treatment  for  meningitis \nin adults, especially for infections caused by S. pneumoniae. It must be \ngiven before or with the first dose of antibiotics; otherwise, it is unlikely \nto improve outcomes.\n\nSuppurative  Intracranial  Infections  (See  also  Chap.  140)  \nSuppurative  intracranial  infections  present  along  with  sepsis  and \nhemodynamic  instability.  Rapid  recognition  of  the  toxic  patient  with \ncentral neurologic signs is crucial to improvement of the prognosis of \nthese entities. Patients with diabetes or hematologic disease may be at \nincreased risk for these infections. Subdural empyema arises from the \nparanasal sinus in 60\u201370% of cases. Microaerophilic streptococci and \nstaphylococci  are  the  predominant  etiologic  organisms.  The  patient \nis  toxic,  with  fever,  headache,  and  nuchal  rigidity.  Of  all  patients, \n75% have focal signs and 6\u201320% die. Despite improved survival rates, \n15\u201344% of patients are left with permanent neurologic deficits. Septic \ncavernous  sinus  thrombosis  follows  a  facial  or  sphenoid  sinus  infec-\ntion;  70%  of  cases  are  due  to  staphylococci  (including  MRSA),  and \nthe remainder are due primarily to aerobic or anaerobic streptococci. \nFungi have been common in some series. A unilateral or retro-orbital \nheadache  progresses  to  a  toxic  appearance  and  fever  within  days. \nThree-quarters  of  patients  have  unilateral  periorbital  edema  that \n\nHPIM21e_Part05_p941-p1796.indd   978\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "979",
    "text": "becomes bilateral and then progresses to ptosis, proptosis, ophthalmo-\nplegia, and papilledema. The mortality rate is as high as 30% in older \nstudies. Recent reports indicate improved survival as high as 90% with \nfewer sequelae. Septic thrombosis of the superior sagittal sinus spreads \nfrom the ethmoid or maxillary sinuses and is caused by S. pneumoniae, \nother streptococci, and staphylococci. The fulminant course is charac-\nterized by headache, nausea, vomiting, rapid progression to confusion \nand coma, nuchal rigidity, and brainstem signs. If the sinus is totally \nthrombosed, the mortality rate exceeds 80%. Broad-spectrum antibi-\notics  and  early  surgical  intervention  at  the  primary  site  of  infection \nmay  improve  outcomes.  Anticoagulation  or  steroids  are  of  uncertain \nbenefit.\n\nBrain Abscess (See also Chap. 140)  Brain abscess often occurs \nwithout  systemic  signs.  Almost  half  of  patients  are  afebrile,  and  pre-\nsentations  are  more  consistent  with  a  space-occupying  lesion  in  the \nbrain; 70% of patients have headache and/or altered mental status, 50% \nhave focal neurologic signs, and 25% have papilledema. Abscesses can \npresent  as  single  or  multiple  lesions  resulting  from  contiguous  foci \nor  hematogenous  infection,  such  as  endocarditis,  or  after  surgery  or \ntrauma. The infection progresses over several days from cerebritis to an \nabscess with a mature capsule. More than half of infections are polymi-\ncrobial, with an etiology consisting of aerobic bacteria (primarily strep-\ntococcal species) and anaerobes. Abscesses arising hematogenously are \nespecially apt to rupture into the ventricular space, causing a sudden \nand  severe  deterioration  in  clinical  status  and  a  high  mortality  rate. \nOtherwise,  mortality  is  low  (<20%)  but  morbidity  is  high  (30\u201355%). \nPatients presenting with stroke and a parameningeal infectious focus, \nsuch  as  sinusitis  or  otitis,  may  have  a  brain  abscess,  and  physicians \nmust maintain a high level of suspicion. Prognosis worsens in patients \nwith  a  fulminant  course,  delayed  diagnosis,  abscess  rupture  into  the \nventricles, multiple abscesses, or abnormal neurologic status at presen-\ntation. In one study, mortality at 1 year was 19%.\n\nCerebral  Malaria  (See  also  Chap.  224)  This  entity  should \nbe  urgently  considered  for  inhabitants  of,  or  recent  travelers  to, \nareas  endemic  for  malaria.  Patients  present  with  a  febrile  illness  and \nlethargy  or  other  neurologic  signs.  Fulminant  malaria  is  caused  by \nPlasmodium falciparum and is associated with temperatures of >40\u00b0C \n(>104\u00b0F), hypotension, jaundice, acute respiratory distress syndrome, \nand bleeding. By definition, any patient with a change in mental status \nor  repeated  seizure  in  the  setting  of  fulminant  malaria  has  cerebral \nmalaria.  In  adults,  this  nonspecific  febrile  illness  progresses  to  coma \nover  several  days;  occasionally,  coma  occurs  within  hours  and  death \nwithin  24  h.  Nuchal  rigidity  and  photophobia  are  rare.  On  physical \nexamination,  symmetric  encephalopathy  is  typical,  and  upper  motor \nneuron dysfunction with decorticate and decerebrate posturing can be \nseen in advanced disease. Unrecognized infection results in a 20\u201330% \nmortality rate. Children with neurologic deficit at hospital discharge, \nseizure recurrence during treatment, and/or ischemic neural injury on \nMRI are at particular risk for neurologic and mental health sequelae.\n\nIntracranial  and  Spinal  Epidural  Abscesses  (See  also  \nChap.  442)  Spinal  and  intracranial  epidural  abscesses  (SEAs  and \nICEAs) can result in permanent neurologic deficits, sepsis, and death. \nAt-risk  patients  include  those  with  diabetes  mellitus;  IV  drug  use; \nchronic alcohol abuse; recent spinal trauma, surgery, or epidural anes-\nthesia; and other comorbid conditions, such as HIV infection. Fungal \nepidural  abscess  and  meningitis  can  follow  epidural  or  paraspinal \nglucocorticoid  injections.  In  the  United  States  and  Canada,  where \nearly  treatment  of  otitis  and  sinusitis  is  typical,  ICEA  is  rare  but  the \nnumber of cases of SEA is on the rise. In areas with limited access to \nhealth care, SEAs and ICEAs cause significant morbidity and mortal-\nity. ICEAs typically present as fever, mental status changes, and neck \npain,  while  SEAs  often  present  as  fever,  localized  spinal  tenderness, \nand  back  pain.  ICEAs  are  typically  polymicrobial,  whereas  SEAs  are \nmost often due to hematogenous seeding, with staphylococci the most \ncommon  etiologic  agent.  Early  diagnosis  and  treatment,  which  may \ninclude surgical drainage, minimize rates of mortality and permanent \nneurologic  sequelae.  Outcomes  are  worse  for  SEA  due  to  MRSA,  for \n\ninfection at a higher vertebral-body level, for impaired neurologic sta-\ntus on presentation, and for dorsal rather than ventral abscess location. \nElderly patients and persons with renal failure, malignancy, and other \ncomorbidities also have less favorable outcomes.\n\n979\n\n \u25a0 OTHER FOCAL SYNDROMES WITH A  \nFULMINANT COURSE\nInfection  at  virtually  any  primary  focus  (e.g.,  osteomyelitis,  pneu-\nmonia,  pyelonephritis,  or  cholangitis)  can  result  in  bacteremia  and \nsepsis.  Lemierre\u2019s  syndrome\u2014jugular  septic  thrombophlebitis  caused \nby  Fusobacterium  necrophorum\u2014is  associated  with  metastatic  infec-\ntious  emboli  (primarily  to  the  lung  but  sometimes  to  the  liver  or \nother organs) and sepsis, with mortality rates of >15%. Fusobacterium \nbloodstream  infections  have  been  associated  with  occult  gastrointes-\ntinal or genitourinary malignancy. TSS has been associated with focal \ninfections  such  as  septic  arthritis,  peritonitis,  sinusitis,  and  wound \ninfection. Rapid clinical deterioration and death can be associated with \ndestruction of the primary site of infection, as is seen in endocarditis \nand in infections of the oropharynx (e.g., Ludwig\u2019s angina or epiglotti-\ntis, in which edema suddenly compromises the airway).\n\nRhinocerebral Mucormycosis (See also Chap. 218) \nIndividuals \nwith diabetes or immunocompromising conditions such as solid organ \ntransplants  or  hematologic  malignancies  are  at  risk  for  invasive  rhi-\nnocerebral mucormycosis. Patients present with low-grade fever, dull \nsinus pain, diplopia, decreased mental status, decreased ocular motion, \nchemosis,  proptosis,  dusky  or  necrotic  nasal  turbinates,  and  necrotic \nhard-palate lesions that respect the midline. Without rapid recognition, \nsurgical  intervention  and  antifungal  therapy,  the  process  continues \non  an  inexorable  invasive  course,  with  mortality  rates  of  50\u201385%  or \ngreater.  Uncontrolled  diabetes  and  increasing  age  are  negative  prog-\nnostic factors.\n\nAcute  Bacterial  Endocarditis  (See  also  Chap.  128)  This \nentity  presents  with  a  much  more  aggressive  course  than  subacute \nendocarditis.  Bacteria  such  as  S.  aureus,  S.  pneumoniae,  L.  monocy-\ntogenes,  Haemophilus  species,  and  streptococci  of  groups  A,  B,  and \nG  attack  native  valves.  Native-valve  endocarditis  caused  by  S.  aureus \n(including MRSA strains) is increasing. Mortality rates range from 10% \nto  40%.  The  host  may  have  comorbid  conditions  such  as  underlying \nmalignancy, diabetes mellitus, IV drug use, or alcoholism. The patient \npresents with fever, fatigue, and malaise <2 weeks after onset of infec-\ntion.  On  physical  examination,  a  changing  murmur  and  congestive \nheart  failure  may  be  noted.  Hemorrhagic  macules  on  palms  or  soles \n(Janeway lesions) sometimes develop. Petechiae, Roth\u2019s spots, splinter \nhemorrhages, and splenomegaly are unusual. Rapid valvular destruc-\ntion, particularly of the aortic valve, results in pulmonary edema and \nhypotension.  Myocardial  abscesses  can  form,  eroding  through  the \nseptum  or  into  the  conduction  system  and  causing  life-threatening \narrhythmias  or  high-degree  conduction  block.  Large  friable  vegeta-\ntions can result in major arterial emboli, metastatic infection, or tissue \ninfarction.  Older  patients  with  S.  aureus  endocarditis  are  especially \nlikely  to  present  with  nonspecific  symptoms\u2014a  circumstance  that \ndelays diagnosis and worsens prognosis. Rapid intervention is crucial \nfor a successful outcome.\n\nInhalational Anthrax (See also Chap. S3) \nInhalational anthrax, \nthe most severe form of disease caused by B. anthracis, had not been \nreported in the United States for more than 25 years until the use of this \norganism as an agent of bioterrorism in 2001. Patients presented with \nmalaise,  fever,  cough,  nausea,  drenching  sweats,  shortness  of  breath, \nand  headache.  Rhinorrhea  was  unusual.  All  patients  had  abnormal \nchest roentgenograms at presentation. Pulmonary infiltrates, medias-\ntinal widening, and pleural effusions were the most common findings. \nHemorrhagic  meningitis  was  documented  in  38%  of  these  patients. \nSurvival was more likely when antibiotics were given during the pro-\ndromal period and when multidrug regimens were used. In the absence \nof urgent intervention with antimicrobial agents and supportive care, \ninhalational anthrax progresses rapidly to hypotension, cyanosis, and \ndeath.\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n2\n\nA\np\np\nr\no\na\nc\nh\nt\no\nt\nh\ne\nA\nc\nu\nt\ne\nl\ny\nI\nl\nl\n\nI\nn\nf\ne\nc\nt\ne\nd\nF\ne\nb\nr\ni\nl\ne\nP\na\nt\ni\ne\nn\nt\n\nHPIM21e_Part05_p941-p1796.indd   979\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n \n \n \n \n"
  },
  {
    "page": "980",
    "text": "980 Viral  Respiratory  Tract  Illness  Viral  respiratory  tract  illnesses \ncan cause severe disease; several new syndromes have been described \nin the past decade. For patients who present with a respiratory illness \nand a relevant exposure and travel history, these viral illnesses must be \nconsidered  and  appropriate  infection  control  measures  instituted  in \naddition to supportive care.\n\nAvian  and  Swine  Influenza  (See  also  Chap.  200)  Human \ncases  of  avian  influenza  have  occurred  primarily  in  Southeast  Asia, \nparticularly  Vietnam  (H5N1)  and  China  (H7N9).  Avian  influenza \nshould be considered in patients with severe respiratory tract illness, \nparticularly if they have been exposed to poultry. Patients present with \nhigh fever, an influenza-like illness, and lower respiratory tract symp-\ntoms;  this  illness  can  progress  rapidly  to  bilateral  pneumonia,  acute \nrespiratory distress syndrome, multiorgan failure, and death. Younger \nage appears to be associated with a lower risk of complications. Early \nantiviral treatment with neuraminidase inhibitors should be initiated \nalong  with  aggressive  supportive  measures.  Unlike  avian  influenza, \nwhose  human-to-human  transmission  has  so  far  been  rare  and  has \nnot  been  sustained,  influenza  caused  by  a  novel  swine-associated  A/\nH1N1  virus  has  spread  rapidly  throughout  the  world;  by  2012,  214 \ncountries  had  diagnosed  cases  of  influenza  A/H1N1,  with  18,449 \ndeaths. Patients most at risk of severe disease are children <5 years of \nage, elderly persons, patients with underlying chronic conditions, and \npregnant women. Obesity also has been identified as a risk factor for \nsevere illness. Immunosuppression and co-infection with S. aureus at \npresentation are independent risk factors for increased mortality.\n\nSARS, COVID-19, and MERS (See Chap. 199)  Severe acute \nrespiratory  syndrome  (SARS)  caused  by  what  is  now  labeled  SARS-\nCoV-1  was  identified  in  2002  in  China  but  has  been  diagnosed  in \nseveral countries, primarily in Asia. Possible animal reservoirs include \nbats  and  civets.  SARS-CoV-1  is  characterized  by  efficient  human \ntransmission  but  relatively  low  mortality.  It  spreads  from  person  to \nperson  via  droplets;  \u201csuper-spreader\u201d  airborne  events  have  occurred. \nThe  potential  SARS-CoV-1  pandemic  was  controlled  through  iden-\ntification  and  isolation  of  infected  patients.  A  3-  to  7-day  prodrome \ncharacterized  by  fever,  malaise,  headache,  and  myalgia  can  progress \nto nonproductive cough, dyspnea, and respiratory failure. The risk of \ncontagion is low during the prodrome. Older patients and those with \ndiabetes mellitus, chronic hepatitis B, and other comorbidities can have \nless favorable outcomes.\n\nCOVID-19  caused  by  SARS-CoV-2  has  resulted  in  a  pandemic  of \nhistoric proportion. Originally linked to cases in China, SARS-CoV-2 \nhas  spread  internationally  at  a  rapid  pace.  As  of  June  2021,  cases \napproached 174 million worldwide with 3.7 million deaths; the United \nStates  had  more  than  33  million  cases  and  ~592,000  deaths.  Actual \ninfection may exceed reported cases by 8\u221210 fold. The primary mode \nof transmission of COVID-19 is through direct person-to-person con-\ntact via respiratory droplets; transmission by the airborne route or by \ncontact with contaminated surfaces has been documented but is much \nless  common.  Greatest  risk  of  infection  is  associated  with  close  and \nprolonged contact such as in household or congregate settings. Rates \nof secondary infection among household contacts are high, over 50% \nin recent studies.\n\nAlthough ~80% of patients with symptomatic COVID-19 infection \nhave mild disease such as cough, fever, gastrointestinal symptoms, and \ntaste and smell alterations, ~5% of patients develop respiratory failure, \nshock,  and  multiorgan  system  failure.  Patients  typically  have  been \nsymptomatic  for  5\u22127  days  before  progression  to  severe  pneumonia \nand hypoxemia. Shock, often in the setting of cardiac injury/myocarditis, \narrhythmias or cardiomyopathy, and thromboembolic complications such \nas pulmonary emboli or stroke, can occur. Other manifestations include \nacute  kidney  or  liver  injury.  Laboratory  findings  include  lymphopenia, \nelevated lactate dehydrogenase, and often evidence of a cytokine-release \ntype syndrome with elevated C-reactive protein, d-dimer, ferritin, and \nproinflammatory  cytokines  such  as  interleukin  6.  Chest  radiographs \nand  CT  imaging  reveal  diffuse  ground-glass  opacities.  Consolida-\ntion, predominantly in the lower lobes and peripherally, is a frequent \n\nfinding.  Mortality  rates  for  patients  requiring  intensive  care  and \nmechanical  ventilation  have  improved  significantly  over  the  course \nof the pandemic but remain \u226520%. Risk factors for severe disease and \npoor outcomes include age \u226565 years, morbid obesity, diabetes, cardio-\nvascular or cerebrovascular disease, hypertension, chronic obstructive \nlung disease, and chronic kidney disease. Severe disease and mortality \nis also increased in males and pregnant women.\n\nTreatment  currently  includes  supportive  care,  antiviral  therapy, \nanti-inflammatory  therapy,  and  anticoagulation  prophylaxis  or  treat-\nment. Patients with respiratory distress are hypoxemic but maintain pul-\nmonary mechanical functioning; thus, invasive mechanical ventilation \nshould be delayed until other interventions have been exhausted. In a \nmeta-analysis, dexamethasone or other glucocorticoids reduced 28-day \nmortality  in  patients  with  severe  COVID-19  disease  compared  with \nstandard care: 32% and 40%, respectively (odds ratio, 0.66; 95% confi-\ndence  interval,  0.53\u20130.82).  Secondary  bacterial  pneumonia  is  uncom-\nmon  in  COVID-19,  and  antibacterial  agents  should  not  be  routinely \nprescribed. With the availability of highly effective vaccines, prevention \nis the most important strategy to control COVID-19.\n\nMiddle  East  respiratory  syndrome  (MERS)  is  caused  by  a  novel \nbetacoronavirus  with  a  likely  bat  reservoir  and  was  first  recognized \nin  2012  in  Saudi  Arabia.  Human  cases  have  been  associated  with \ndirect and indirect contact with dromedary camels. Unlike SARS and \nCOVID-19, MERS exhibits inefficient human transmission but carries \na  high  mortality  rate.  As  of  2017,  >2000  cases  had  been  confirmed, \nmost in the Arabian Peninsula, with 35% mortality. MERS ranges from \nasymptomatic infection to acute respiratory distress syndrome, multi-\norgan failure, and death. Elderly men with comorbidities appear to be \nat highest risk for poor outcomes. Despite little documented human-\nto-human transmission in the community, nosocomial infection must \nbe prevented by adherence to strict infection control practices. MERS \nis currently a low-level public health threat and is likely to remain so \nunless the virus mutates and its transmissibility increases.\n\nHantavirus  Pulmonary  Syndrome  (See  also  Chap.  209)  \nHantavirus pulmonary syndrome has been documented in the United \nStates  since  1993  (primarily  the  southwestern  states,  west  of  the  \nMississippi  River),  Canada,  and  South  America.  Most  cases  occur  in \nrural areas and are associated with exposure to rodents. Patients pres-\nent with a nonspecific viral prodrome of fever, malaise, myalgias, nau-\nsea, vomiting, and dizziness that may progress to pulmonary edema, \nrespiratory failure, and death. Hantavirus pulmonary syndrome causes \nmyocardial depression and increased pulmonary vascular permeabil-\nity; therefore, careful fluid resuscitation and use of pressor agents are \ncrucial. Aggressive cardiopulmonary support during the first few hours \nof illness can be life-saving in this high-mortality syndrome. The early \nonset of thrombocytopenia may help distinguish this syndrome from \nother febrile illnesses in an appropriate epidemiologic setting.\n\nClostridioides  difficile  Infection  C.  difficile  infection  (CDI)  is  a \ntoxin-mediated  diarrheal  syndrome  that  is  strongly  associated  with \nprior antibiotic use. Proton-pump inhibitors also have been identified \nas a potential risk factor for the disease. Although most cases of CDI \nhave  occurred  in  the  health  care  setting,  community-onset  CDI  is \nincreasing. Overall, community-onset cases occur in younger patients \nthan  nosocomial  cases.  Patients  with  community-onset  CDI  are  less \nlikely  to  have  a  history  of  antibiotic  or  protein-pump  inhibitor  use. \nCDI is associated with significant morbidity and mortality, particularly \namong older patients. The CDC named C. difficile infection as one of \nthe top three health threats associated with antibiotic use.\n\nSUMMARY\nAcutely ill febrile patients with the syndromes discussed in this chapter \nrequire  close  observation,  aggressive  supportive  measures,  and\u2014in \nmost  cases\u2014admission  to  intensive  care  units.  The  most  important \ntask of the physician is to distinguish these patients from other infected \nfebrile  patients  whose  illness  will  not  progress  to  fulminant  disease. \nThe alert physician must recognize the acute infectious disease emer-\ngency and proceed with appropriate urgency.\n\nHPIM21e_Part05_p941-p1796.indd   980\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "981",
    "text": " \u25a0 FURTHER READING\nKrause PJ: Human babesiosis. Int J Parasitol 49:165, 2019.\nNelson  GE  et  al:  Epidemiology  of  invasive  Group  A  Streptococcal \ninfections  in  the  United  States,  2005\u20132012.  Clin  Infect  Dis  63:478, \n2016.\n\nRhee C et al: Incidence and trends of sepsis in US hospitals using clin-\n\nical vs claims data, 2009\u20132014. JAMA 318:1241, 2017.\n\nSterne JA et al: Association between administration of systemic corti-\ncosteroids and mortality among critically ill patients with COVID-19: \nA meta-analysis. JAMA 324:1330, 2020.\n\nTheilacker  C  et  al:  Overwhelming  postsplenectomy  infection:  A \nprospective multicenter cohort study. Clin Infect Dis 62:871, 2016.\nZunt  JR  et  al:  Global,  regional,  and  national  burden  of  meningitis, \n1990\u20132016: A systematic analysis for the Global Burden of Disease \nStudy 2016. Lancet Neurol 17:1061, 2018.\n\n123 Immunization Principles  \nand Vaccine Use\nSarah Mbaeyi, Amanda Cohn,  \nNancy Messonnier\n\nFew medical interventions of the past century can rival the effect that \nimmunization has had on longevity, economic savings, and quality of \nlife. Seventeen diseases are now preventable through vaccines routinely \nadministered to children and adults in the United States (Table 123-1), \nand most vaccine-preventable diseases of childhood are at historically \nlow levels (Table 123-2). Health care providers deliver the vast major-\nity of vaccines in the United States in the course of providing routine \n\nTARGET POPULATION(S) FOR ROUTINE USE\nChildren, adolescents, adults\nChildren, adolescents, adults\nChildren, adolescents, adults\nChildren\nChildren\nChildren\nChildren\n\nTABLE 123-1  Diseases Preventable with Vaccines Routinely \nAdministered in the United States to Children and/or Adults\nCONDITION\nPertussis\nDiphtheria\nTetanus\nPoliomyelitis\nMeasles\nMumps\nRubella, congenital rubella \nsyndrome\nHepatitis B\nHaemophilus influenzae type b \ninfection\nHepatitis A\nInfluenza\nVaricella\nPneumococcal disease\nSerogroups A, C, W, Y \nmeningococcal disease\nSerogroup B meningococcal \ndisease\nRotavirus infection\nHuman papillomavirus infection, \ncervical and anogenital cancers\nZoster\n\nChildren and high-risk adults\nChildren, adolescents, adults\nChildren\nChildren, older adults, and high-risk adultsa\nAdolescents and high-risk children and \nadults\nHigh-risk children and adultsa\n\nInfants\nAdolescents and young adultsa\n\nChildren and high-risk adults\nChildren and high-risk adults\n\nOlder adults\n\naOthers in certain age groups may be vaccinated based on shared clinical \ndecision-making.\n\nTABLE 123-2  Decline in Vaccine-Preventable Diseases in the United \nStates Following Widespread Implementation of National Vaccine \nRecommendations\n\n981\n\nANNUAL NO. \nOF PREVACCINE \nCASES (AVERAGE)\n29,005\n\nNO. OF CASES \nREPORTED IN \n2019a\n0\n\nREDUCTION (%) \nIN CASES AFTER \nWIDESPREAD \nVACCINATION\n100\n\n21,053\n530,217\n162,344\n200,752\n16,316\n47,745\n152\n\n580\n20,000\n\n117,333\n66,232\n63,067\n\n2\n1287\n3509\n15,662\n0\n3\n0\n\n19\n14b\n\n4000c\n20,900c\n30,400d\n\n62,500\n\n30,625e\n\n4,085,120\n\n102,128f\n\n>99\n>99\n98\n92\n100\n>99\n100\n\n97\n>99\n\n97\n68\n52\n\n51\n\n98\n\nCONDITION\nSmallpox\n\nDiphtheria\nMeasles\nMumps\nPertussis\nPolio (paralytic)\nRubella\nCongenital rubella \nsyndrome\nTetanus\nHaemophilus \ninfluenzae type b \ninfection\nHepatitis A\nHepatitis B (acute)\nInvasive \npneumococcal \ninfection: all ages\nRotavirus \nhospitalizations  \n(<3 years old)\nVaricella\n\na2019 reported cases unless otherwise specified (provisional as of January 2020). \nbAn additional 12 type b infections are estimated to have occurred among 243 \nreports of H. influenzae infection caused by unknown types among children  \n<5 years of age. cData from the CDC\u2019s Viral Hepatitis Surveillance, 2016. dUnpublished \ndata from the CDC\u2019s Active Bacterial Core Surveillance, 2016. eUnpublished data \nfrom the CDC\u2019s New Vaccine Surveillance Network, 2017, U.S. rotavirus disease \nnow has biennial pattern. fUnpublished data from CDC\u2019s varicella program, 2017.\nSource: Adapted from SW Roush et al: JAMA 298:2155, 2007 and Morb Mortal Wkly \nRep 65:924, 2017.\n\nhealth services and therefore play an integral role in the nation\u2019s public \nhealth system.\n\nThe  COVID-19  pandemic  has  highlighted  the  importance  of  a \nstrong  immunization  program:  robust  surveillance  systems  to  detect \nemerging  infectious  disease  threats;  public-private  partnerships  to \naccelerate the development of novel vaccines; and systems in place to \nrapidly implement a vaccination program and monitor vaccine safety \nand effectiveness. As of July 2021, multiple COVID-19 vaccines have \nreceived authorization for emergency use in the United States and over \ntwo-thirds of adults have received at least one dose. Furthermore, the \nlargest and most comprehensive vaccine safety program in U.S. history \nhas  demonstrated  the  safety  of  COVID-19  vaccines,  and  evaluations \nhave  demonstrated  the  vaccines  to  be  effective  against  SARS-CoV-2 \ninfections, including against severe disease that would otherwise result \nin hospitalization and death.\n \u25a0 VACCINE IMPACT\nDirect and Indirect Effects \nImmunizations against specific infec-\ntious diseases protect individuals against infection and thereby prevent \nsymptomatic illnesses. In addition, specific vaccines may also blunt the \nseverity of clinical illness (e.g., rotavirus vaccines and severe gastroen-\nteritis) or reduce complications (e.g., zoster vaccines and postherpetic \nneuralgia). Some immunizations also reduce transmission of infectious \ndisease  agents  from  immunized  people  to  others,  thereby  reducing \nthe impact of infection spread. This indirect impact is known as herd \nimmunity. The level of immunization in a population that is required to \nachieve indirect protection of unimmunized people varies substantially \nwith the specific vaccine and disease.\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n3\n\nI\n\nm\nm\nu\nn\ni\nz\na\nt\ni\no\nn\nP\nr\ni\nn\nc\ni\np\nl\ne\ns\na\nn\nd\nV\na\nc\nc\ni\nn\ne\nU\ns\ne\n\nHPIM21e_Part05_p941-p1796.indd   981\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n \n"
  },
  {
    "page": "982",
    "text": "982\n\nSince  childhood  vaccines  have  become  widely  available  in  the \nUnited States, major declines in rates of vaccine-preventable diseases \namong  both  children  and  adults  have  become  evident  (Table  123-2). \nFor example, vaccination of children <5 years of age against Streptococcus  \npneumoniae has led to not only a >90% overall reduction in invasive \ndisease  caused  by  the  types  covered  by  pneumococcal  conjugate \nvaccines,  but  also  substantial  reductions  in  incidence  among  adults \nthrough  herd  immunity.  Among  children  born  during  1994\u20132013,  a \nseries of childhood vaccines targeting 13 vaccine-preventable diseases \nwill prevent 322 million illnesses and 732,000 deaths over the course of \ntheir lifetimes and save $1.38 trillion (U.S.).\n\nControl, Elimination, and Eradication of Vaccine-Preventable \nDiseases \nImmunization  programs  are  associated  with  the  goals \nof  controlling,  eliminating,  or  eradicating  a  disease.  Control  of  a \nvaccine-preventable  disease  reduces  poor  illness  outcomes  and  often \nlimits  the  disruptive  impacts  associated  with  outbreaks  of  disease  in \ncommunities,  schools,  and  institutions.  Control  programs  can  also \nreduce absences from work for ill persons and for parents caring for \nsick  children,  decrease  absences  from  school,  and  limit  health  care \nutilization associated with treatment visits.\n\nElimination  of  a  disease  is  a  more  demanding  goal  than  control, \nusually requiring the reduction to zero of cases in a defined geographic \narea but sometimes defined as reduction in the indigenous sustained \ntransmission  of  an  infection  in  a  geographic  area.  As  of  2019,  the \nUnited  States  had  eliminated  indigenous  transmission  of  measles, \nrubella,  poliomyelitis,  and  diphtheria,  although  measles  elimination \nstatus was threatened in 2019 due to imported cases into undervacci-\nnated communities, resulting in prolonged outbreaks. Importation of \npathogens from other parts of the world continues to be important, and \npublic health efforts are intended to respond promptly to such cases in \norder to limit forward spread of the infectious agent.\n\nEradication  of  a  disease  is  achieved  when  its  elimination  can  be \nsustained  without  the  need  to  continue  interventions.  The  only  \nvaccine-preventable  disease  of  humans  that  has  been  globally  erad-\nicated  thus  far  is  smallpox.  Although  smallpox  vaccine  is  no  longer \ngiven  routinely,  the  disease  has  not  reemerged  naturally  because  all \nchains of human transmission were interrupted through earlier vacci-\nnation efforts and humans were the only natural reservoir of the virus. \nCurrently, a major health initiative is targeting the global eradication of \npolio. Two of the three wild poliovirus types (types 2 and 3) have been \neradicated globally. However, type 1 continues to circulate in Afghanistan \nand Pakistan and outbreaks of type 2 circulating vaccine-derived polio-\nvirus have been reported in recent years. Detection of a case of disease \nthat  has  been  targeted  for  eradication  or  elimination  is  considered  a \nsentinel event that could permit the infectious agent to become rees-\ntablished in the community or region. Therefore, such episodes must \nbe promptly reported to public health authorities.\n\nOutbreak  Detection  and  Control  Clusters  of  cases  of  a  vaccine- \npreventable disease detected in an institution, a medical practice, or a \ncommunity  may  signal  important  changes  in  the  pathogen,  vaccine, \nor environment. Several factors can give rise to increases in vaccine- \npreventable disease, including (1) low rates of immunization that result \nin  an  accumulation  of  susceptible  people  (e.g.,  measles  resurgence \namong vaccination abstainers); (2) changes in the infectious agent that \npermit it to escape vaccine-induced protection (e.g., non-vaccine-type \npneumococci);  (3)  waning  of  vaccine-induced  immunity  (e.g.,  per-\ntussis  among  adolescents  and  adults  vaccinated  in  early  childhood); \nand  (4)  point-source  introductions  of  large  inocula  (e.g.,  food-borne \nexposure to hepatitis A virus). Reporting episodes of outbreak-prone \ndiseases to public health authorities can facilitate recognition of clus-\nters that require further interventions.\nPUBLIC  HEALTH  REPORTING  Recognition  of  suspected  cases  of \ndiseases  targeted  for  elimination  or  eradication\u2014along  with  other \ndiseases that require urgent public health interventions, such as con-\ntact  tracing,  administration  of  chemo-  or  immunoprophylaxis,  or \nepidemiologic  investigation  for  common-source  exposure\u2014is  typi-\ncally  associated  with  special  reporting  requirements.  Many  diseases \n\nagainst which vaccines are routinely used, including measles, pertussis,  \nHaemophilus  influenzae  type  b  invasive  disease,  and  varicella,  are \nnationally  notifiable.  Clinicians  and  laboratory  staff  have  a  responsi-\nbility to report some vaccine-preventable disease occurrences to local \nor state public health authorities according to specific case-definition \ncriteria. All providers should be aware of state or city disease-reporting \nrequirements and the best ways to contact public health authorities. A \nprompt response to vaccine-preventable disease outbreaks can greatly \nenhance the effectiveness of control measures.\n\nGLOBAL  CONSIDERATIONS  Several  international  health  initiatives \ncurrently  focus  on  reducing  vaccine-preventable  diseases  in  regions \nthroughout  the  world.  The  American  Red  Cross,  the  World  Health \nOrganization  (WHO),  the  United  Nations  Foundation,  the  United \nNations Children\u2019s Fund (UNICEF), and the Centers for Disease Con-\ntrol  and  Prevention  (CDC)  are  partners  in  the  Measles  and  Rubella \nInitiative, which targets reduction of worldwide measles deaths. Dur-\ning  2000\u20132018,  global  measles  mortality  rates  declined  by  73%\u2014i.e., \nfrom an estimated 535,600 deaths in 2000 to 142,300 deaths in 2018. \nRotary  International,  UNICEF,  the  CDC,  and  the  WHO  are  leading \npartners in the global eradication of polio, an endeavor that reduced \nthe  annual  number  of  paralytic  polio  cases  from  350,000  in  1988  to \n33 in 2018. Furthermore, there are increased global efforts to develop \nand introduce vaccines for emerging infectious diseases, such as Ebola \nvirus  disease  and  COVID-19.  The  GAVI  Alliance  and  the  Bill  and \nMelinda  Gates  Foundation  have  brought  substantial  momentum  to \nglobal  efforts  to  reduce  vaccine-preventable  diseases,  expanding  on \nearlier efforts by the WHO, UNICEF, and governments in developed \nand developing countries.\n\nEnhancing  Immunization  in  Adults  Although  immunization \nhas  become  a  centerpiece  of  routine  pediatric  medical  visits,  it  has \nnot  been  as  well  integrated  into  routine  health  care  visits  for  adults. \nThis  chapter  focuses  on  immunization  principles  and  vaccine  use  in \nadults.  Accumulating  evidence  suggests  that  immunization  coverage \ncan  be  increased  through  efforts  directed  at  consumer-,  provider-, \ninstitution-,  and  system-level  factors.  The  literature  suggests  that  the \napplication of multiple strategies is more effective at raising coverage \nrates than is the use of any single strategy.\n\nRECOMMENDATIONS FOR ADULT IMMUNIZATIONS  The CDC\u2019s Advi-\nsory Committee on Immunization Practices (ACIP) is the main source \nof  recommendations  for  administration  of  vaccines  approved  by  the \nU.S.  Food  and  Drug  Administration  (FDA)  for  use  in  children  and \nadults  in  the  U.S.  civilian  population.  The  ACIP  is  a  federal  advi-\nsory committee that consists of 15 voting members (experts in fields \nassociated  with  immunization)  appointed  by  the  Secretary  of  the \nU.S. Department of Health and Human Services, as well as ex officio \nmembers representing federal agencies and nonvoting representatives \nof various liaison organizations, including major medical societies and \nmanaged-care organizations. The ACIP recommendations, which are \navailable at www.cdc.gov/vaccines/hcp/acip-recs/, are harmonized to the \ngreatest extent possible with vaccine recommendations made by other \norganizations, including the American College of Physicians, American \nAcademy of Family Physicians, American College of Obstetricians and \nGynecologists, and American College of Nurse-Midwives.\n\nACIP makes several types of recommendations. Routine, catch-up, \nand  risk-based  recommendations  are  those  in  which  everyone  in  a \nparticular age or risk group are recommended to receive vaccination. \nExamples  include  recombinant  zoster  vaccination  for  adults  aged  \n\u226550 years and hepatitis B vaccination in people living with HIV. Shared \nclinical  decision-making  recommendations  are  individually  based  \nand informed by a decision process between the health care provider \nand patient. With shared clinical decision-making recommendations, \nthe  decision  to  vaccinate  is  informed  by  the  best  available  evidence \non who may benefit from vaccination; the individual\u2019s characteristics, \nvalues,  and  preferences;  the  health  care  provider\u2019s  clinical  discretion; \nand  the  characteristics  of  the  vaccine  being  considered.  Examples \nof  shared  clinical  decision-making  recommendations  include  pneu-\nmococcal  conjugate  vaccination  of  immunocompetent  adults  aged  \n\nHPIM21e_Part05_p941-p1796.indd   982\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "983",
    "text": "\u226565  years,  human  papillomavirus  vaccination  of  adults  aged  27\u201345 \nyears, and serogroup B meningococcal vaccination of adolescents and \nyoung adults aged 16\u201323 years.\nADULT  IMMUNIZATION  SCHEDULES  Immunization  schedules  for \nadults  in  the  United  States  are  updated  annually  and  can  be  found \nonline  (www.cdc.gov/vaccines/schedules/hcp/adult.html).  In  February, \nthe  schedules  are  published  in  American  Family  Physician,  Annals  of \nInternal Medicine, and Morbidity and Mortality Weekly Report (www.\ncdc.gov/mmwr). The adult immunization schedules for 2020 are sum-\nmarized in Fig. 123-1. Additional information and specifications are \ncontained  in  the  footnotes  to  these  schedules.  In  the  time  between \nannual publications, additions and changes to schedules are published \nin Morbidity and Mortality Weekly Report.\n\n \u25a0 IMMUNIZATION PRACTICE STANDARDS\nAdministering  immunizations  to  adults  involves  a  number  of  pro-\ncesses, such as deciding whom to vaccinate, assessing vaccine contrain-\ndications  and  precautions,  providing  vaccine  information  statements \n(VISs), ensuring appropriate storage and handling of vaccines, admin-\nistering  vaccines,  and  maintaining  vaccine  records.  In  addition,  pro-\nvider reporting of adverse events that follow vaccination is an essential \ncomponent  of  the  vaccine  safety  monitoring  system.  In  2014,  the \nStandards for Adult Immunization Practice were revised to help pro-\nviders  take  steps  to  ensure  that  their  patients  are  fully  immunized, \nincluding assessing the immunization status of patients at every clinical \nencounter, strongly recommending vaccines that patients need, admin-\nistering vaccines or referring the patient to a vaccination provider, and \ndocumenting vaccines received by the patient.\n\nDeciding  Whom  to  Vaccinate  Every  effort  should  be  made  to \nensure that adults receive all indicated vaccines as expeditiously as pos-\nsible. When adults present for care, their immunization history should \nbe  assessed  and  recorded,  and  this  information  should  be  used  to \nidentify needed vaccinations according to the most current version of \nthe adult immunization schedule. Decision-support tools incorporated \ninto electronic health records can provide prompts for needed vacci-\nnations. Standing orders, which are often used for routinely indicated \nvaccines (e.g., influenza and zoster vaccines), permit a nurse or another \napproved  licensed  practitioner  to  administer  vaccines  without  a  spe-\ncific physician order, thus lowering barriers to adult immunization.\n\nAssessing  Contraindications  and  Precautions  Before  vac-\ncination,  all  patients  should  be  screened  for  contraindications  and \nprecautions.  A  contraindication  is  a  condition  that  increases  the  risk \nof a serious adverse reaction to vaccination. A vaccine should not be \nadministered when a contraindication is documented. For example, a \nhistory of an anaphylactic reaction to a dose of vaccine or to a vaccine \ncomponent  is  a  contraindication  for  further  doses.  A  precaution  is  a \ncondition that may increase the risk of an adverse event or that may \ncompromise the ability of the vaccine to evoke immunity (e.g., admin-\nistering measles vaccine to a person who has recently received a blood \ntransfusion and may consequently have transient passive immunity to \nmeasles  virus).  Normally,  a  vaccine  is  not  administered  when  a  pre-\ncaution  is  noted.  However,  situations  may  arise  when  the  benefits  of \nvaccination outweigh the estimated risk of an adverse event, and the \nprovider may decide to vaccinate the patient despite the precaution.\n\nIn  some  cases,  contraindications  and  precautions  are  temporary \nand  may  lead  to  mere  deferral  of  vaccination  until  a  later  time.  For \nexample,  moderate  or  severe  acute  illness  with  or  without  fever  is \ngenerally considered a transient precaution to vaccination and results \nin postponement of vaccine administration until the acute phase has \nresolved; thus, the superimposition of adverse effects of vaccination on \nthe underlying illness and the mistaken attribution of a manifestation \nof the underlying illness to the vaccine are avoided. Contraindications \nand  precautions  to  vaccines  licensed  in  the  United  States  for  use  in \nadults  are  summarized  in  Table  123-3.  It  is  important  to  recognize \nconditions that are not contraindications in order not to miss oppor-\ntunities for vaccination. For example, in most cases, mild acute illness \n(with or without fever), a history of a mild to moderate local reaction \n\nto a previous dose of the vaccine, and breast-feeding are not contrain-\ndications to vaccination.\n\n983\n\nHistory  of  Immediate  Hypersensitivity  to  a  Vaccine  \nComponent  A  severe  allergic  reaction  (e.g.,  anaphylaxis)  to  a \nprevious dose of a vaccine or to one of its components is a contrain-\ndication to vaccination. While most vaccines have many components, \nsubstances  to  which  individuals  are  most  likely  to  have  had  a  severe \nallergic  reaction  include  egg  protein,  gelatin,  and  yeast.  In  addition, \nalthough natural rubber (latex) is not a vaccine component, some vac-\ncines are supplied in vials or syringes that contain natural rubber latex. \nThese vaccines can be identified by the product insert and should not \nbe administered to persons who report a severe (anaphylactic) allergy \nto latex unless the benefit of vaccination clearly outweighs the risk for \na potential allergic reaction. The much more common local or contact \nhypersensitivity  to  latex,  such  as  to  medical  gloves  (which  contain \nsynthetic latex that is not linked to allergic reactions), is not a contrain-\ndication to administration of a vaccine supplied in a vial or syringe that \ncontains natural rubber latex.\n\nPregnancy  Two  inactivated  vaccines,  tetanus  and  diphtheria  tox-\noids  and  acellular  pertussis  (Tdap)  vaccine  and  inactivated  influenza \nvaccine,  are  routinely  recommended  for  pregnant  women  in  the \nUnited States. Tdap vaccine is recommended during each pregnancy, \nregardless of prior vaccination status, in order to prevent pertussis in \nneonates.  Annual  influenza  vaccination  is  recommended  for  all  per-\nsons 6 months of age and older, including among pregnant women to \nprotect  both  mother  and  infant.  Pregnancy  is  not  a  contraindication \nto administration of most other inactivated vaccines when otherwise \nindicated or when the benefits of vaccination are judged to outweigh \npotential  risks  (e.g.,  serogroup  B  meningococcal  vaccination);  it  is \nrecommended  that  human  papillomavirus  (HPV)  and  recombinant \nzoster vaccination is delayed until after pregnancy. Live-virus vaccines \n(e.g., measles, mumps, and rubella [MMR], varicella) are contraindi-\ncated during pregnancy because of the hypothetical risk that vaccine \nvirus replication will cause congenital infection or have other adverse \neffects on the fetus. Most live-virus vaccines are not secreted in breast \nmilk; therefore, breast-feeding is not a contraindication for live-virus \nor other vaccines.\n\nImmunosuppression  Live-virus  vaccines  elicit  an  immune \nresponse  due  to  replication  of  the  attenuated  (weakened)  vaccine \nvirus  that  is  contained  by  the  recipient\u2019s  immune  system.  In  persons \nwith compromised immune function, enhanced replication of vaccine \nviruses  is  possible  and  could  lead  to  disseminated  infection  with  the \nvaccine virus. For this reason, live-virus vaccines are contraindicated \nfor  persons  with  severe  immunosuppression,  the  definition  of  which \nmay vary with the vaccine. Severe immunosuppression may be caused \nby  many  disease  conditions,  including  hematologic  or  other  malig-\nnancy.  In  some  of  these  conditions,  all  affected  persons  are  severely \nimmunocompromised.  In  others  (e.g.,  HIV  infection),  the  degree  to \nwhich  the  immune  system  is  compromised  depends  on  the  severity \nof  the  condition,  which  in  turn  depends  on  the  stage  of  disease  or \ntreatment. For example, MMR vaccine may be given to HIV-infected \npersons  who  are  not  severely  immunocompromised.  Severe  immu-\nnosuppression  may  also  be  due  to  therapy  with  immunosuppressive \nagents, including high-dose glucocorticoids. In this situation, the dose, \nduration,  and  route  of  administration  may  influence  the  degree  of \nimmunosuppression.\n\n \u25a0 VACCINE INFORMATION STATEMENTS\nA  VIS  is  a  one-page  (two-sided)  information  sheet  produced  by  the \nCDC that informs vaccine recipients (or their parents or legal repre-\nsentatives) about the benefits and risks of a vaccine. VISs are mandated \nby the National Childhood Vaccine Injury Act (NCVIA) of 1986 and\u2014\nwhether the vaccine recipient is a child or an adult\u2014must be provided \nfor any vaccine covered by the Vaccine Injury Compensation Program. \nAs  of  April  2020,  vaccines  that  are  covered  by  the  NCVIA  and  that \nare  licensed  for  use  in  adults  include  tetanus  and  diphtheria  toxoids \n(Td),  Tdap,  hepatitis  A,  hepatitis  B,  HPV,  inactivated  influenza,  live \n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n3\n\nI\n\nm\nm\nu\nn\ni\nz\na\nt\ni\no\nn\nP\nr\ni\nn\nc\ni\np\nl\ne\ns\na\nn\nd\nV\na\nc\nc\ni\nn\ne\nU\ns\ne\n\nHPIM21e_Part05_p941-p1796.indd   983\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n \n"
  },
  {
    "page": "984",
    "text": "984\n\nVaccine\n\n19-26 years\n\n27-49 years\n\n50-64 years\n\n\u226565 years\n\nInfluenza inactivated (IIV) or\nInfluenza recombinant (RIV)\n\nInfluenza live, attenuated\n(LAIV)\n\nTetanus, diphtheria, pertussis\n(Tdap or Td)\n\nMeasles, mumps, rubella\n(MMR)\n\nVaricella\n(VAR)\n\nZoster recombinant\n(RZV) (preferred)\n\nZoster live\n(ZVL)\n\nor\n\nor\n\n1 dose annually\n\nor\n\n1 dose annually\n\n1 dose Tdap, then Td or Tdap booster every 10 years\n\n1 or 2 doses depending on indication\n(if born in 1957 or later)\n\n2 doses (if born in 1980 or later)\n\n2 doses\n\n2 doses\n\nor\n\n1 dose\n\nHuman papillomavirus (HPV)\n\n2 or 3 doses depending on age at\ninitial vaccination or condition\n\n27 through 45 years\n\nPneumococcal conjugate\n(PCV13)\n\nPneumococcal polysaccharide\n(PPSV23)\n\nHepatitis A\n(HepA)\n\nHepatitis B\n(HepB)\n\nMeningococcal A, C, W, Y\n(MenACWY)\n\nMeningococcal B\n(MenB)\n\nHaemophilus influenzae type b\n(Hib)\n\n1 dose\n\n1 or 2 doses depending on indication\n\n65 years and older\n\n1 dose\n\n2 or 3 doses depending on vaccine\n\n2 or 3 doses depending on vaccine\n\n1 or 2 doses depending on indication, see notes for booster recommendations\n\n19 through 23 years\n\n2 or 3 doses depending on vaccine and indication, see notes for booster recommendations\n\n1 or 3 doses depending on indication\n\nRecommended vaccination for adults who meet age requirement,\nlack documentation of vaccination, or lack evidence of past infection\n\nRecommended vaccination for adults with an\nadditional risk factors or another indication\n\nRecommended vaccination based\non shared clinical decision-making\n\nNo recommendation/\nNot applicable\n\nVaccine\n\nPregnancy\n\nImmuno-\ncompromised\n(excluding HIV\ninfection)\n\nHIV infection\nCD4 count\n\n<200\n\n\u2265200\n\nAsplenla,\ncomplement\ndeficlencles\n\nEnd-stage\nrenal\ndisease; or on\nhemodialysis\n\nHeart or\nlung disease,\nalcoholism\n\nChronic liver\ndisease\n\nDiabetes\n\nHealth care\npersonnel2\n\nMen who have\nsex with men\n\nIIV or RIV\n\nor\nLAIV\n\nNOT RECOMMENDED\n\nPRECAUTION\n\nor\n1 dose annually\n\n1 dose annually\n\nTdap or Td\n\n1 dose Tdap\neach pregnancy\n\nMMR\n\nVAR\n\nNOT RECOMMENDED\n\nNOT RECOMMENDED\n\nRZV\n(preferred)\n\nDELAY\n\nNOT RECOMMENDED\n\n1 dose Tdap, then Td or Tdap booster every 10 years\n\n1 or 2 doses depending on indication\n\n2 doses\n\n2 doses at age \u2265 50 years\n\nor\n1 dose at age \u2265 60 years\n\nDELAY\n\n3 doses through age 26 years\n\n2 or 3 doses through age 26 years\n\n1 dose\n\n1, 2, or 3 doses depending on age and indication\n\n2 or 3 doses depending on vaccine\n\n2 or 3 doses depending on vaccine\n\nor\nZVL\n\nHPV\n\nPCV13\n\nPPSV23\n\nHepA\n\nHepB\n\nMenACWY\n\n1 or 2 doses depending on indication, see notes for booster recommendations\n\nMenB\n\nPRECAUTION\n\n2 or 3 doses depending on vaccine and indication, see notes for booster recommendations\n\nHib\n\n3 doses HSCT3\nreciplents only\n\n1 dose\n\nRecommended vaccination\nfor adults who meet\nage requirement, lack\ndocumentation of\nvaccination, or lack\nevidence of past infection\n\nRecommended\nvaccination for adults\nwith an additional risk\nfactor or another\nindication\n\nPrecaution\u2014vaccination\nmight be indicated if\nbenefit of protection \noutweighs risk of\nadverse reaction\n\nDelay vaccination\nuntil after pregnancy\nif vaccine is indicated\n\nNot recommended/\ncontraindicated\u2014\nvaccine\nshould not be\nadministered\n\nNo recommendation/\nNot applicable\n\nFIGURE  123-1  Recommended  adult  immunization  schedules,  United  States,  2020.  Additional  information,  including  footnotes  for  each  vaccine,  contraindications,  and \nprecautions,  can  be  found  at  https://www.cdc.gov/vaccines/schedules/hcp/imz/adult.html.  The  recommendations  in  this  schedule  were  approved  by  the  Centers  for \nDisease Control and Prevention (CDC) Advisory Committee on Immunization Practices (ACIP), the American Academy of Family Physicians (AAFP), the American College of \nPhysicians (ACP), the American College of Obstetricians and Gynecologists (ACOG), and the American College of Nurse-Midwives (ACNM). For complete statements by the \nACIP, visit www.cdc.gov/vaccines/hcp/acip-recs/.\n\nHPIM21e_Part05_p941-p1796.indd   984\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "985",
    "text": "985\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n3\n\nI\n\nm\nm\nu\nn\ni\nz\na\nt\ni\no\nn\nP\nr\ni\nn\nc\ni\np\nl\ne\ns\na\nn\nd\nV\na\nc\nc\ni\nn\ne\nU\ns\ne\n\nTd\n\nHPV\n\nTdap\n\nMMR\n\nTABLE 123-3  Contraindications and Precautions for Commonly Used Vaccines in Adults\nVACCINE FORMULATION CONTRAINDICATIONS AND PRECAUTIONS\nContraindication\nAll vaccines\nSevere allergic reaction (e.g., anaphylaxis) after a previous vaccine dose or to a vaccine component\nPrecaution\nModerate or severe acute illness with or without fever. Defer vaccination until illness resolves.\nPrecautions\nGBS within 6 weeks after a previous dose of TT-containing vaccine\nHistory of Arthus-type hypersensitivity reactions after a previous dose of TD- or DT-containing vaccines (including MenACWY). Defer \nvaccination until at least 10 years have elapsed since the last dose.\nHistory of severe allergic reaction to dry natural rubber (latex) (certain formulations; syringe; see text)\nContraindication\nHistory of encephalopathy (e.g., coma or prolonged seizures) not attributable to another identifiable cause within 7 days of administration of \na vaccine with pertussis components, such as DTaP or Tdap\nPrecautions\nGBS within 6 weeks after a previous dose of TT-containing vaccine\nProgressive or unstable neurologic disorder, uncontrolled seizures, or progressive encephalopathy. Defer vaccination until a treatment \nregimen has been established and the condition has stabilized.\nHistory of Arthus-type hypersensitivity reactions after a previous dose of TT- or DT-containing vaccines (including MenACWY). Defer \nvaccination until at least 10 years have elapsed since the last dose.\nHistory of severe allergic reaction to dry natural rubber (latex) (syringe; see text)\nContraindications\nHistory of immediate hypersensitivity to yeast\nPrecaution\nPregnancy (If a woman is found to be pregnant after initiation of the vaccination series, the remainder of the series should be delayed \nuntil after completion of the pregnancy. If a vaccine dose has been administered during pregnancy, no intervention is needed. Exposure to \nGardasil during pregnancy should be reported to Merck at 877-888-4231.\nContraindications\nHistory of immediate hypersensitivity reaction to gelatina or neomycin\nPregnancy\nKnown severe immunodeficiency (e.g., hematologic and solid tumors; chemotherapy; congenital immunodeficiency; long-term \nimmunosuppressive therapy; severe immunocompromise due to HIV infection)\nPrecautions\nRecent receipt (within 11 months) of antibody-containing blood product\nHistory of thrombocytopenia or thrombocytopenic purpura\nContraindications\nPregnancy\nKnown severe immunodeficiency\nHistory of immediate hypersensitivity reaction to gelatina or neomycin\nPrecaution\nRecent receipt (within 11 months) of antibody-containing blood product\nPrecautionb\nHistory of GBS within 6 weeks after a previous influenza vaccine dose\nContraindicationsb\nPregnancy\nImmunosuppression, including that caused by medications or by HIV infection; known severe immunodeficiency (e.g., hematologic and \nsolid tumors; chemotherapy; congenital immunodeficiency; long-term immunosuppressive therapy; severe immunocompromise due to HIV \ninfection)\nClose contact with severely immunosuppressed persons who require a protected environment, such as isolation in a bone marrow \ntransplantation unit\nClose contact with persons with lesser degrees of immunosuppression (e.g., persons receiving chemotherapy or radiation therapy who are \nnot being cared for in a protective environment; persons with HIV infection) is not a contraindication or a precaution. Health care personnel \nin neonatal intensive care units or oncology clinics may receive live attenuated influenza vaccine.\nReceipt of influenza antiviral medication within 48 h before vaccination\nPrecautions\nHistory of GBS within 6 weeks of a previous influenza vaccine dose\nCertain chronic medical conditions, such as diabetes mellitus; chronic pulmonary disease (including asthma); chronic cardiovascular \ndisease (except hypertension); renal, hepatic, neurologic/neuromuscular, hematologic, or metabolic disorders\nNone, other than those listed for all vaccines\n\nInfluenza, live attenuated \nnasal spray\n\nInfluenza, inactivated, \ninjectable\n\nVaricella\n\nPneumococcal \npolysaccharide\nPneumococcal conjugate None, other than those listed for all vaccines\nContraindication\nHepatitis A\nHistory of severe allergic reaction to dry natural rubber (latex) (syringe; see text)\n\nHPIM21e_Part05_p941-p1796.indd   985\n\n20/01/22   6:34 PM\n\n(Continued)\n\n \n \n \n \n \n \n"
  },
  {
    "page": "986",
    "text": "986\n\nTABLE 123-3  Contraindications and Precautions for Commonly Used Vaccines in Adults\nVACCINE FORMULATION CONTRAINDICATIONS AND PRECAUTIONS\n\n(Continued)\n\nHepatitis B\n\nMeningococcal \nconjugate\nSerogroup B \nmeningococcal\n\nZoster\n\nContraindications\nHistory of immediate hypersensitivity to yeast (for Engerix-B\u00ae and Recombivax-HB\u00ae)\nHistory of severe allergic reaction to dry natural rubber (latex) (certain formulations; syringe; see text)\nNone, other than those listed for all vaccines\n\nContraindication\nHistory of severe allergic reaction to dry natural rubber (latex) (certain formulations; syringe; see text)\nPrecaution\nPregnancy (vaccination may be indicated if benefits of protection outweigh risks of adverse reaction)\nContraindications\nPregnancy\nKnown severe immunodeficiency (for Zostavax\u00ae)\nHistory of immediate hypersensitivity reaction to gelatina or neomycin (for Zostavax\u00ae)\nPrecaution\nReceipt of specific antiviral agents (i.e., acyclovir, famciclovir, or valacyclovir) within 24 h before vaccination (for Zostavax\u00ae)\n\naExtreme caution must be exercised in administering MMR, varicella, or live zoster vaccine to persons with a history of anaphylactic reaction to gelatin or gelatin-containing \nproducts. Before administration, skin testing for sensitivity to gelatin can be considered. However, no specific protocols for this purpose have been published. bHistory of \nsevere allergic reaction (e.g., anaphylaxis) to egg is a labeled contraindication to the use of inactivated influenza vaccine and live attenuated influenza vaccine. However, \nCDC\u2019s Advisory Committee on Immunization Practices recommends that any licensed, recommended, and appropriate inactivated influenza vaccine or recombinant \ninfluenza vaccine may be administered to persons with egg allergy of any severity (www.cdc.gov/vaccines/hcp/acip-recs/vacc-specific/flu.html).\nAbbreviations: DT, diphtheria toxoid; DTaP, diphtheria, tetanus, and pertussis; GBS, Guillain-Barr\u00e9 syndrome; HPV, human papillomavirus; MenACWY, quadrivalent \nmeningococcal conjugate vaccine; MMR, measles, mumps, and rubella; Td, tetanus and diphtheria toxoids; Tdap, tetanus and diphtheria toxoids and acellular pertussis;  \nTT, tetanus toxoid.\n\nintranasal influenza, MMR, pneumococcal conjugate, meningococcal \nconjugate,  serogroup  B  meningococcal,  polio,  and  varicella  vaccines. \nWhen  combination  vaccines  for  which  no  separate  VIS  exists  are \nadministered (e.g., hepatitis A and B combination vaccine), all relevant \nVISs  should  be  provided.  VISs  also  exist  for  some  vaccines  not  cov-\nered  by  the  NCVIA,  such  as  pneumococcal  polysaccharide,  Japanese \nencephalitis, rabies, herpes zoster, typhoid, anthrax, and yellow fever \nvaccines. The use of these VISs is encouraged but is not mandated.\n\nAll  current  VISs  are  available  on  the  Internet  at  two  websites:  the \nCDC\u2019s  Vaccines  and  Immunizations  site  (www.cdc.gov/vaccines/hcp/\nvis/) and the Immunization Action Coalition\u2019s site (www.immunize.org/\nvis/). (The latter site also includes translations of the VISs.) VISs from \nthese sites can be downloaded and printed.\n \u25a0 STORAGE AND HANDLING\nInjectable vaccines are packaged in multidose vials, single-dose vials, or \nmanufacturer-filled single-dose syringes. The live attenuated nasal-spray \ninfluenza vaccine is packaged in single-dose sprayers. Oral typhoid vac-\ncine is packaged in capsules. Some vaccines, such as MMR and varicella, \ncome as lyophilized (freeze-dried) powders that must be reconstituted \n(i.e.,  mixed  with  a  liquid  diluent)  before  use.  The  lyophilized  powder \nand the diluent come in separate vials. Diluents are not interchangeable \nbut rather are specifically formulated for each type of vaccine; only the \nspecific diluent provided by the manufacturer for each type of vaccine \nshould be used. Once lyophilized vaccines have been reconstituted, their \nshelf-life is limited and they must be stored under appropriate tempera-\nture and light conditions. For example, varicella must be protected from \nlight  and  administered  within  30  min  of  reconstitution;  recombinant \nzoster and MMR vaccines likewise must be protected from light but can \nbe used up to 6 and 8 h after reconstitution, respectively.\n\nVaccines are stored either at refrigerator temperature (2\u20138\u00b0C) or at \nfreezer temperature (\u201315\u00b0C or colder). In general, inactivated vaccines \n(e.g., inactivated influenza, pneumococcal polysaccharide, and menin-\ngococcal  conjugate  vaccines)  are  stored  at  refrigerator  temperature, \nwhile vials of lyophilized-powder live-virus vaccines (e.g., varicella, live \nzoster, and MMR vaccines) are stored at freezer temperature. Diluents \nfor lyophilized vaccines may be stored at refrigerator or room tempera-\nture. Live attenuated influenza vaccine\u2014a live-virus liquid formulation \nadministered by nasal spray\u2014is stored at refrigerator temperature.\n\nVaccine storage and handling errors can result in the loss of vaccines \nworth millions of dollars, and administration of improperly stored vac-\ncines may elicit inadequate immune responses in patients. To improve \n\nthe standard of vaccine storage and handling practices, the CDC has \npublished  detailed  guidance  (available  at  www.cdc.gov/vaccines/hcp/\nadmin/storage/toolkit/storage-handling-toolkit.pdf). For vaccine storage, \nthe CDC recommends stand-alone units\u2014i.e., self-contained units that \neither refrigerate or freeze but do not do both\u2014as these units maintain \nthe required temperatures better than combination refrigerator/freezer \nunits.  Dormitory-style  combined  refrigerator/freezer  units  should \nnever be used for vaccine storage.\n\nThe  temperature  of  refrigerators  and  freezers  used  for  vaccine \nstorage must be monitored and recorded at least twice each workday. \nIdeally, continuous thermometers that measure and record temperature \nall day and all night are used, and minimal and maximal temperatures \nare read and documented each workday. The CDC recommends the use \nof calibrated digital thermometers with a probe in thermal-buffered \nmaterial; more detailed information on specifications of storage units \nand temperature-monitoring devices is provided at the link given above.\n \u25a0 ADMINISTRATION OF VACCINES\nMost parenteral vaccines recommended for routine administration to \nadults in the United States are given by either the IM or the SC route; \none influenza vaccine formulation approved for use in persons 2\u201349 years \nof age is given intranasally. Some live-virus vaccines such as varicella \nand MMR are given SC. Most inactivated vaccines are given IM. The \n23-valent  pneumococcal  polysaccharide  vaccine  may  be  given  either \nIM or SC, but IM administration is preferred because it is associated \nwith a lower risk of injection-site reactions.\n\nVaccines  given  to  adults  by  the  SC  route  are  administered  with  a \n5/8-inch  needle  into  the  upper  outer-triceps  area.  Vaccines  adminis-\ntered to adults by the IM route are injected into the deltoid muscle  \n(Fig. 123-2) with a needle whose length should be selected on the basis \nof the recipient\u2019s sex and weight to ensure adequate penetration into the \nmuscle. Current guidelines indicate that, for men and women weighing \n<152  lb  (<70  kg),  a  1-inch  needle  is  sufficient;  for  women  weighing \n152\u2013200 lb (70\u201390 kg) and men weighing 152\u2013260 lb (70\u2013118 kg), a \n1- to 1.5-inch needle is needed; and for women weighing >200 lb (>90 kg) \nand  men  weighing  >260  lb  (>118  kg),  a  1.5-inch  needle  is  required. \nAdditional  illustrations  of  vaccine  injection  locations  and  techniques \nmay be found at www.immunize.org/catg.d/p2020.pdf.\n\nAspiration, the process of pulling back on the plunger of the syringe \nafter skin penetration but prior to injection, is not necessary because \nno large blood vessels are present at the recommended vaccine injec-\ntion sites.\n\nHPIM21e_Part05_p941-p1796.indd   986\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "987",
    "text": "Site of intramuscular\ninjection: deltoid\n\nDermis\n\nFatty tissue\n(subcutaneous)\n\nMuscle\ntissue\n\nFIGURE 123-2  Technique for IM administration of vaccine.\n\nMultiple  vaccines  can  be  administered  at  the  same  visit;  indeed, \nadministration of all needed vaccines at one visit is encouraged. Studies \nhave shown that vaccines are as effective when administered simulta-\nneously  as  they  are  individually,  and  simultaneous  administration  of \nmultiple  vaccines  is  not  associated  with  an  increased  risk  of  adverse \neffects.  If  more  than  one  vaccine  must  be  administered  in  the  same \nlimb, the injection sites should be separated by 1\u20132 inches so that any \nlocal reactions can be differentiated. If a vaccine and an immune glob-\nulin preparation are administered simultaneously (e.g., Td vaccine and \ntetanus immune globulin), a separate anatomic site should be used for \neach injection.\n\nFor  certain  vaccines  (e.g.,  HPV  vaccine  and  hepatitis  B  vaccine), \nmultiple  doses  are  required  for  an  adequate  and  persistent  antibody \nresponse. The recommended vaccination schedule specifies the inter-\nval between doses. Many adults who receive the first dose in a multi-\nple-dose  vaccine  series  do  not  complete  the  series  or  do  not  receive \nsubsequent doses within the recommended interval; this lack of adher-\nence to protocol compromises vaccine efficacy and/or the duration of \nprotection. Providers should implement recall systems that will prompt \npatients  to  return  for  subsequent  doses  in  a  vaccination  series  at  the \nappropriate intervals. With the exception of oral typhoid vaccination, \nan interruption in the schedule does not require restarting of the entire \nseries or the addition of extra doses.\n\nSyncope  may  follow  vaccination,  especially  in  adolescents  and \nyoung  adults.  Serious  injuries,  including  skull  fracture  and  cerebral \nhemorrhage, have occurred. Adolescents and adults should be seated \nor  lying  down  during  vaccination.  The  majority  of  reported  syncope \nepisodes  after  vaccination  occur  within  15  min.  The  ACIP  recom-\nmends  that  vaccine  providers  strongly  consider  observing  patients, \nparticularly adolescents, with patients seated or lying down for 15 min \nafter  vaccination.  If  syncope  develops,  patients  should  be  observed \nuntil the symptoms resolve.\n\nAnaphylaxis  is  a  rare  complication  of  vaccination.  All  facilities \nproviding  immunizations  should  have  an  emergency  kit  containing \naqueous  epinephrine  for  administration  in  the  event  of  a  systemic \nanaphylactic reaction.\n\n \u25a0 MAINTENANCE OF VACCINE RECORDS\nAll vaccines administered should be fully documented in the patient\u2019s \npermanent  medical  record.  Documentation  should  include  the  date \n\nIntramuscular\nneedle insertion\n\n90\u00ba\n\n987\n\nof  administration,  the  name  or  com-\nmon  abbreviation  of  the  vaccine,  the \nvaccine  lot  number  and  manufacturer, \nthe  administration  site,  the  VIS  edi-\ntion,  the  date  the  VIS  was  provided, \nand the name, address, and title of the \nperson  who  administered  the  vaccine. \nIncreasing  use  of  two-dimensional  bar \ncodes  on  vaccine  vials  and  syringes \nthat can be scanned for data entry into \ncompatible  electronic  medical  records \nand immunization information systems \nmay facilitate more complete and accu-\nrate recording of required information.\n \u25a0 VACCINE SAFETY \nMONITORING AND ADVERSE \nEVENT REPORTING\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n3\n\nI\n\nm\nm\nu\nn\ni\nz\na\nt\ni\no\nn\nP\nr\ni\nn\nc\ni\np\nl\ne\ns\na\nn\nd\nV\na\nc\nc\ni\nn\ne\nU\ns\ne\n\nPrelicensure  Evaluations  of  Vac-\ncine  Safety  Before  vaccines  are \nlicensed by the FDA, they are evaluated \nin clinical trials with volunteers. These \ntrials are conducted in three progressive \nphases. Phase 1 trials are small, usually \ninvolving  <100  volunteers.  Their  pur-\nposes  are  to  provide  a  basic  evalua-\ntion  of  safety  and  to  identify  common \nadverse events. Phase 2 trials, which are \nlarger and may involve several hundred participants, collect additional \ninformation on safety and are usually designed to evaluate immunoge-\nnicity as well. Data gained from phase 2 trials can be used to determine \nthe composition of the vaccine, the number of doses required, and a \nprofile of common adverse events. Vaccines that appear promising are \nevaluated in phase 3 trials, which typically involve several hundred to \nseveral thousand volunteers and are generally designed to demonstrate \nvaccine efficacy and provide additional information on vaccine safety.\n\nPostlicensure  Monitoring  of  Vaccine  Safety  After  licensure, \na  vaccine\u2019s  safety  is  assessed  by  several  mechanisms.  The  NCVIA \nof  1986  requires  health  care  providers  to  report  certain  adverse \nevents  that  follow  vaccination.  As  a  mechanism  for  that  reporting, \nthe  Vaccine  Adverse  Event  Reporting  System  (VAERS)  was  estab-\nlished  in  1990  and  is  jointly  managed  by  the  CDC  and  the  FDA. \nThis  safety  surveillance  system  collects  reports  of  adverse  events \nassociated  with  vaccines  currently  licensed  in  the  United  States. \nAdverse  events  are  defined  as  untoward  events  that  occur  after \nimmunization  and  that  might  be  caused  by  the  vaccine  product  or  \nvaccination process. While the VAERS was established in response to the \nNCVIA, any adverse event following vaccination\u2014whether in a child or \nan adult, and whether or not it is believed to have actually been caused by  \nvaccination\u2014may  be  reported  through  the  VAERS.  The  adverse \nevents  that  health  care  providers  are  required  to  report  are  listed  in \nthe  reportable-events  table  on  the  VAERS  website  at  vaers.hhs.gov/\nreportable.htm. Approximately 30,000 VAERS reports are filed annually, \nwith ~10\u221215% reporting serious events resulting in hospitalization, life- \nthreatening illness, disability, or death.\n\nAnyone  can  file  a  VAERS  report,  including  health  care  providers, \nmanufacturers,  and  vaccine  recipients  or  their  parents  or  guard-\nians.  VAERS  reports  may  be  submitted  online  or  in  paper  form \n(https://vaers.hhs.gov/reportevent.html);  additional  information  can \nbe obtained by email (info@vaers.org) or phone (800-822-7967). The \nVAERS  form  asks  for  the  following  information:  the  type  of  vaccine \nreceived;  the  timing  of  vaccination;  the  time  of  onset  of  the  adverse \nevent;  and  the  recipient\u2019s  current  illnesses  or  medications,  history  of \nadverse events following vaccination, and demographic characteristics \n(e.g.,  age  and  sex).  This  information  is  entered  into  a  database.  The \nindividual who reported the adverse event then receives a confirmation \nletter by mail with a VAERS identification number that can be used if \nadditional information is submitted later. In selected cases of serious \n\nHPIM21e_Part05_p941-p1796.indd   987\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n \n"
  },
  {
    "page": "988",
    "text": "988\n\nadverse reaction, the patient\u2019s recovery status may be followed up at 60 \ndays and 1 year after vaccination. The FDA and the CDC have access \nto VAERS data and use this information to monitor vaccine safety and \nconduct  research  studies.  VAERS  data  (minus  personal  information) \nare also available to the public.\n\nWhile  the  VAERS  provides  useful  information  on  vaccine  safety, \nthis  passive  reporting  system  has  important  limitations.  One  is  that \nevents  following  vaccination  are  merely  reported;  the  system  cannot \nassess whether a given type of event occurs more often than expected \nafter vaccination. A second is that event reporting is incomplete and \nis biased toward events that are believed to be more likely to be due to \nvaccination and that occur relatively soon after vaccination. To obtain \nmore systematic information on adverse events occurring in both vac-\ncinated and unvaccinated persons, the Vaccine Safety Datalink project \nwas  initiated  in  1991.  Directed  by  the  CDC,  this  project  includes \nnine managed-care organizations in the United States; member data-\nbases  include  information  on  immunizations,  medical  conditions, \ndemographics,  laboratory  results,  and  medication  prescriptions.  The \nDepartment of Defense oversees a similar system monitoring the safety \nof immunizations among active-duty military personnel. In addition, \npostlicensure  evaluations  of  vaccine  safety  may  be  conducted  by  the \nvaccine manufacturer. In fact, such evaluations are often required by \nthe FDA as a condition of vaccine licensure.\n \u25a0 CONSUMER ACCESS TO AND DEMAND FOR \nIMMUNIZATION\nBy removing barriers to the consumer or patient, providers and health \ncare institutions can improve vaccine use. Financial barriers have tra-\nditionally  been  important  constraints,  particularly  among  uninsured \nadults.  Even  for  insured  adults,  out-of-pocket  costs  associated  with \nnewer, more expensive adult vaccines (e.g., zoster vaccine) are an obsta-\ncle to be overcome. After influenza vaccine was included by Medicare \nfor all beneficiaries in 1993, coverage among persons \u226565 years of age \ndoubled (from ~30% in 1989 to >60% in 1997). Other strategies that \nenhance patients\u2019 access to vaccination include extended office hours \n(e.g.,  evening  and  weekend  hours)  and  scheduled  vaccination-only \nclinics where waiting times are reduced. Provision of vaccines outside \nthe  \u201cmedical  home\u201d  (e.g.,  through  occupational  clinics,  universities, \npharmacies, and retail settings) can expand access for adults who do \nnot make medical visits frequently. Increasing proportions of adults are \nbeing vaccinated in these settings.\n\nHealth  promotion  efforts  aimed  at  increasing  the  demand  for \nimmunization are common. Direct-to-consumer advertising by phar-\nmaceutical companies has been used for some newer adolescent and \nadult  vaccines.  Efforts  to  raise  consumer  demand  for  vaccines  have \nnot increased immunization rates unless implemented in conjunction \nwith other strategies that target strengthening of provider practices or \nreduction of consumer barriers. Attitudes and beliefs related to vacci-\nnation can be considerable impediments to consumer demand. Many \nadults view vaccines as important for children but are less familiar with \nvaccinations  targeting  disease  prevention  in  adults.  Several  vaccines \nare  recommended  for  adults  with  certain  medical  risk  factors,  but \nself-identification as a high-risk individual is relatively rare. Commu-\nnication research suggests that adults are motivated to get vaccines to \nprotect  their  own  health  and  many  would  get  vaccinated  to  protect \nloved ones. Adults with chronic conditions are more likely to be aware \nthat they need to protect their own health. Some vaccines are explicitly \nrecommended  for  persons  at  relatively  low  risk  of  serious  complica-\ntions, with the goal of reducing the risk of transmission to higher-risk \ncontacts. For example, for protection of newborns, vaccinations against \ninfluenza and pertussis are recommended for pregnant women.\n \u25a0 STRATEGIES FOR PROVIDERS AND HEALTH CARE \nFACILITIES\nRecommendation  from  the  Provider  Health  care  providers \ncan have great influence on patients with regard to immunization. A \nrecommendation  from  a  doctor  or  nurse  carries  more  weight  than \ndo recommendations from professional societies or endorsements by \ncelebrities.  Strong  provider  recommendations  using  a  presumptive \n\napproach  (e.g.,  \u201cYou\u2019re  due  for  the  flu  shot  today\u201d  vs  \u201cWhat  do  you \nwant to do about the flu shot?\u201d) have been shown to improve vaccine \nacceptance (Chap. 3). Providers should be well informed about vaccine \nrisks and benefits so that they can address patients\u2019 common concerns. \nThe  CDC,  the  American  College  of  Physicians,  and  the  American \nAcademy  of  Family  Physicians  review  and  update  the  schedule  for \nadult  immunization  on  an  annual  basis  and  have  developed  educa-\ntional materials to facilitate provider\u2013patient discussions about vacci-\nnation (www.cdc.gov/vaccines/hcp.htm).\n\nSystem  Supports  Medical  offices  can  incorporate  a  variety  of \nmethods  to  ensure  that  providers  consistently  offer  specific  immuni-\nzations to patients with indications for specific vaccines. Decision-support \ntools have been incorporated into some electronic health records to alert \nthe provider when specific vaccines are indicated. Manual or automated \nreminders  and  standing  orders  have  been  discussed  (see  \u201cDeciding \nWhom to Vaccinate,\u201d above) and have consistently improved vaccina-\ntion coverage in both office and hospital settings. Most clinicians\u2019 esti-\nmates of their own performance diverge from objective measurements \nof  their  patients\u2019  immunization  coverage;  quantitative  assessment  and \nfeedback  have  been  shown  in  pediatric  and  adolescent  practices  to \nincrease  immunization  performance  significantly.  Some  health  plans \nhave instituted incentives for providers with high rates of immunization \ncoverage.  Specialty  providers,  including  obstetrician\u2013gynecologists, \nmay be the only providers serving some high-risk patients with indi-\ncations  for  selected  vaccines  (e.g.,  Tdap,  influenza,  or  pneumococcal \npolysaccharide vaccine).\n\nImmunization Requirements  Vaccination against selected com-\nmunicable diseases is required for attendance at many universities and \ncolleges as well as for service in the U.S. military or in some occupa-\ntional settings (e.g., child care, laboratory, veterinary, and health care). \nImmunizations are recommended and sometimes required for travel to \ncertain countries (Chap. 124).\n\nVaccination  of  Health  Care  Staff  A  particular  area  of  focus \nfor  medical  settings  is  vaccination  of  health  care  workers,  including \nthose  with  and  without  direct  patient-care  responsibilities.  The  Joint \nCommission  (which  accredits  health  care  organizations),  the  CDC\u2019s \nHealthcare Infection Control Practices Advisory Committee, and the \nACIP all recommend influenza vaccination of all health care personnel; \nrecommendations also focus on requiring documentation of declina-\ntion for providers who do not accept annual influenza vaccination. As \npart of their participation in the Centers for Medicare and Medicaid \nServices\u2019  Hospital  Inpatient  Quality  Reporting  program,  acute-care \nhospitals are required to report the proportion of their health care per-\nsonnel who have received seasonal influenza vaccine. Some institutions \nand  jurisdictions  have  added  mandates  on  influenza  vaccination  of \nhealth care workers and have expanded on earlier requirements related \nto vaccination or proof of immunity for hepatitis B, measles, mumps, \nrubella, and varicella.\n\n \u25a0 VACCINATION IN NONMEDICAL SETTINGS\nReceipt of vaccination in medical offices is most frequent among young \nchildren and adults \u226565 years of age. Patients in these age groups make \nmore  office  visits  and  are  more  likely  to  receive  care  in  a  consistent \n\u201cmedical  home\u201d  than  are  older  children,  adolescents,  and  nonelderly \nadults.  Vaccination  outside  the  medical  home  can  expand  access  to \nthose  whose  health  care  visits  are  limited  and  reduce  the  burden  on \nbusy clinical practices. In some locations, financial constraints related \nto inventory and storage requirements have led providers to stock few \nor no vaccines. Outside private office and hospital settings, vaccination \nmay  also  occur  at  health  department  venues,  workplaces,  retail  sites \n(including pharmacies and supermarkets), and schools or colleges.\n\nWhen vaccines are given in nonmedical settings, it remains impor-\ntant for standards of immunization practice to be followed. Consum-\ners  should  be  provided  with  information  on  the  vaccine  and  how  to \nreport  adverse  events  (e.g.,  via  provision  of  a  VIS),  and  procedures \nshould  ensure  that  documentation  of  vaccine  administration  is  for-\nwarded to the primary care provider and the state or city public health \n\nHPIM21e_Part05_p941-p1796.indd   988\n\n20/01/22   6:34 PM\n\nPART 5Infectious Diseases"
  },
  {
    "page": "989",
    "text": "immunization  registry.  Detailed  documentation  may  be  required \nfor  employment,  school  attendance,  and  travel.  Personalized  health \nrecords  can  help  consumers  keep  track  of  their  immunizations,  and \nsome occupational health clinics have incorporated automated immu-\nnization  reports  that  help  employees  stay  up-to-date  with  recom-\nmended vaccinations. Some pharmacy chain establishments are using \nautomated systems to report immunization information to the state or \nlocal immunization information system.\n \u25a0 PERFORMANCE MONITORING\nTracking  of  immunization  coverage  at  national,  state,  institution, \nand  practice  levels  can  yield  feedback  to  practitioners  and  programs \nand facilitate quality improvement. Healthcare Effectiveness Data and \nInformation  Set  (HEDIS)  measures  related  to  adult  immunization \nfacilitate  comparison  of  health  plans.  The  CDC\u2019s  National  Health \nInterview  Survey  provides  selected  information  on  immunization \ncoverage  among  adults  and  tracks  progress  toward  achievement  of \nHealthy  People  2020  targets  for  immunization  coverage.  Vaccination \ncoverage among adults remains suboptimal, and state-specific immu-\nnization  coverage  with  pneumococcal  polysaccharide  and  influenza \nvaccines (as measured through the CDC\u2019s Behavioral Risk Factor Sur-\nveillance System) reveals substantial geographic variation in coverage. \nThere are persistent disparities in adult immunization coverage rates \nbetween whites and racial and ethnic minorities. In contrast, racial and \neconomic  disparities  in  immunization  of  young  children  have  been \ndramatically reduced during the past 20 years. Much of this progress \nis attributed to the Vaccines for Children Program, which since 1994 \nhas  entitled  eligible  children,  including  those  who  are  uninsured  or \nunderinsured, to receive free vaccines.\n \u25a0 FUTURE TRENDS\nAlthough  most  vaccines  developed  in  the  twentieth  century  targeted \ncommon acute infectious diseases of childhood, more recently devel-\noped  vaccines  prevent  chronic  conditions  prevalent  among  adults. \nHepatitis B vaccine prevents hepatitis B\u2013related cirrhosis and hepato-\ncellular carcinoma, and HPV vaccine prevents some types of cervical \ncancer,  genital  warts,  and  anogenital  cancers  and  may  also  prevent \nsome oropharyngeal cancers. A new herpes zoster subunit vaccine that \nwas  licensed  in  2017  should  substantially  improve  protection  against \nzoster and postherpetic neuralgia. New targets of vaccine development \nand research may further broaden the definition of vaccine-preventable \ndisease. Research is ongoing on vaccines to prevent insulin-dependent \ndiabetes mellitus, nicotine addiction, and Alzheimer\u2019s disease. Expand-\ning  strategies  for  vaccine  development  are  incorporating  molecular \napproaches  such  as  RNA,  DNA,  vector,  and  peptide  vaccines.  New \ntechnologies,  such  as  the  use  of  transdermal  and  other  needle-less \nroutes of administration, are being applied to vaccine delivery.\n\nAcknowledgment\nThe authors thank Anne Schuchat, MD, and Lisa A. Jackson, MD, for \ntheir significant contributions to this chapter in the previous editions.\n \u25a0 FURTHER READING\nCenters for Disease Control and Prevention: Epidemiology and \nPrevention of Vaccine-Preventable Diseases, 13th ed. J Hamborsky et al \n(eds). Washington DC, Public Health Foundation, 2015.\n\nEzeanolue E et al: General best practice guidelines for immunization. \nBest practices guidance of the Advisory Committee on Immunization \nPractices  (ACIP).  Available  at  www.cdc.gov/vaccines/hcp/acip-recs/\ngeneral-recs/downloads/general-recs.pdf. Accessed December 14, 2020.\nMcneil  MM  et  al:  The  vaccine  safety  datalink:  Successes  and  chal-\n\nlenges monitoring vaccine safety. Vaccine 32:5390, 2014.\n\nNational Vaccine Advisory Committee: Recommendations from \nthe  National  Vaccine  Advisory  Committee:  Standards  for  Adult \nImmunization Practice. Public Health Rep 129:115, 2014.\n\nPlotkin SA et al (eds): Plotkin\u2019s Vaccines, 7th ed. Philadelphia, Elsevier, \n\n2017.\n\nWhitney CW et al: Benefits from immunization during the Vaccines \nfor  Children  Program  era\u2014United  States,  1994\u20132013.  MMWR \n63:352, 2014.\n\n989\n\nC\nH\nA\nP\nT\nE\nR\n1\n2\n4\n\nH\ne\na\nl\nt\nh\nR\ne\nc\no\nm\nm\ne\nn\nd\na\nt\ni\no\nn\ns\nf\no\nr\n\nI\nn\nt\ne\nr\nn\na\nt\ni\no\nn\na\nl\n\nT\nr\na\nv\ne\nl\n\n124 Health Recommendations  \nfor International Travel\nJesse Waggoner, Henry M. Wu\n\nIn recent decades, international travel has increased dramatically with \nglobalization  and  greater  access  to  international  flights.  According  to \nthe United Nations World Tourism Organization, international tourist \narrivals increased 47.4% from 2010 to 2018; arrivals exceeded 1.4 billion \nin 2018, with the highest rate of growth in arrivals to destinations in \nAsia  and  the  Pacific.  In  2018,  according  to  the  United  Nations  Con-\nference on Trade and Development, total global merchandise exports \nreached a record 19.5 trillion USD, a nearly threefold increase over the \nprevious two decades. Although travel in 2020 dropped drastically dur-\ning the COVID-19 pandemic, there has been limited recovery in 2021 \nand eventual resumption of pre-pandemic growth trends appears likely.\nInternational travel has brought social, economic, and cultural benefits \nto the world; however, travel also widens the range of infections to which \nan individual may be exposed. The speed of air travel has been a major \nfactor in the ease with which emerging infectious diseases have quickly \nspread worldwide in recent years. In the nineteenth century, interconti-\nnental travel took long enough that travelers often recovered or perished \nfrom  acute  infections  before  arrival  at  their  destinations.  However,  in \nthe jet age, the time required to circumnavigate the globe has decreased \nto <24 hours. This duration is shorter than the incubation periods for \nalmost  all  infections,  increasing  the  likelihood  that  infected  travelers \ncan arrive at their destinations prior to symptom onset. Epidemics can \nresult;  examples  include  severe  acute  respiratory  syndrome  (SARS)  in \n2003, Ebola virus disease in 2014, and the COVID-19 pandemic in 2020. \nFurthermore, introduction of pathogens into vulnerable regions can sub-\nsequently lead to infections becoming endemic, as was observed with the \nreintroduction of dengue throughout much of the Americas beginning in \nthe 1970s and the global spread of HIV infections in the 1980s.\n\nAdditional  challenges  include  the  increasing  diversity  of  travelers. \nWhile tourism, business travel, and mission work continue to be pop-\nular,  recent  decades  have  seen  increasing  numbers  of  other  types  of \ntravelers,  including  students,  migrants,  medical  tourists,  and  persons \nvisiting their countries of origin (\u201cvisiting friends and relatives\u201d [VFR] \ntravelers).  Furthermore,  an  increasing  range  of  individuals  with  risk \nfactors  for  illness  or  injury  are  traveling  internationally,  including \nelderly  persons,  infants,  pregnant  women,  and  persons  with  chronic \nmedical conditions (e.g., immunocompromising conditions). Whether \npracticing travel medicine, primary care, or other specialties, providers \nwill encounter patients who travel internationally. This chapter outlines \nkey considerations and preventive measures for international travelers, \nparticularly those traveling to low- and middle-income countries.\n\nEPIDEMIOLOGY OF TRAVEL-RELATED \nCONDITIONS\nUnanticipated medical problems during travel are common. Although \nreported rates of travel-related morbidity and mortality vary widely by \ndestination, traveler type, and study methodology, as many as 43\u201379% \nof travelers report developing a travel-related illness. Most illnesses are \nminor,  with  diarrhea  often  the  most  commonly  reported  and  fewer \nthan  1\u20133%  of  travelers  reporting  hospitalization.  Among  vaccine- \npreventable infections in travelers to lower-income countries, influenza \nis  by  far  the  most  commonly  reported.  Typhoid  and  hepatitis  A  are \nreported much less often, but typically they are still reported more fre-\nquently than other infections commonly discussed in travel medicine \nbut not frequently diagnosed, including cholera, Japanese encephalitis, \nmeningococcal disease, rabies, poliomyelitis, and yellow fever. However, \noutbreaks such as the emergence of yellow fever in coastal southeastern \nBrazil in 2017\u20132018 or cholera associated with the 2010 earthquake in \nHaiti  can  result  in  an  increased  incidence  of  travel-associated  cases. \nAmong causes of death in travelers, studies suggest that cardiovascular \n\nHPIM21e_Part05_p941-p1796.indd   989\n\n20/01/22   6:34 PM\n\n \n \n \n \n \n \n"
  }
]
